{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Identifying host and pathogen information\n",
    "\n",
    "In this notebook I will be going thorugh how to identify your host and pathogen, crawl through NCBI using Biopython, and extract the full genbank record, create a list of associated publications, and download host and pathogen sequences. There are a few section in the Biopython cookbook I reference frequently and will outline here.\n",
    "\n",
    "* [Overview of Biopython](http://biopython.org/DIST/docs/tutorial/Tutorial.html#sec7)\n",
    "* [Accessing NCBI Entrez database](http://biopython.org/DIST/docs/tutorial/Tutorial.html#sec139)\n",
    "    + I would pay particular attention to section 9.15 and 9.16\n",
    "* [Full genbank record](http://biopython.org/DIST/docs/tutorial/Tutorial.html#sec145)\n",
    "* [Retrieving publications](http://biopython.org/DIST/docs/tutorial/Tutorial.html#sec158)\n",
    "* [Downloading Sequences](http://biopython.org/DIST/docs/tutorial/Tutorial.html#sec163)\n",
    "\n",
    "For this tutorial we will be using the human immunodeficiency virus 1 (HIV) and Homo sapeins as our models. These are two good models as humans have selfishly put many genetic records of themselves into NCBI along with their pathogens."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Starting with the Entrez database\n",
    "\n",
    "Entrez is a powerful but complex database. It allows nearly all of NCBI to be accessed using Biopython. Being complex and highly sought after comes with some caveats.\n",
    "\n",
    "1. You need to make an NCBI account. this will allow you to create an API key and have an email on file\n",
    "2. They limit traffic during certain parts of the week. Be diligent about not overloading their servers\n",
    "3. Did I mention it was complex?\n",
    "\n",
    "**We will always enter our apikey and email.**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['pubmed', 'protein', 'nuccore', 'ipg', 'nucleotide', 'structure', 'sparcle', 'protfam', 'genome', 'annotinfo', 'assembly', 'bioproject', 'biosample', 'blastdbinfo', 'books', 'cdd', 'clinvar', 'gap', 'gapplus', 'grasp', 'dbvar', 'gene', 'gds', 'geoprofiles', 'homologene', 'medgen', 'mesh', 'ncbisearch', 'nlmcatalog', 'omim', 'orgtrack', 'pmc', 'popset', 'proteinclusters', 'pcassay', 'biosystems', 'pccompound', 'pcsubstance', 'seqannot', 'snp', 'sra', 'taxonomy', 'biocollections', 'gtr']"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Showing the depth of Entrez\n",
    "# Import statement\n",
    "from Bio import Entrez\n",
    "# Enter your own api key. To find this go to your NCBI account, click your profile name (top right), go to API Key management\n",
    "Entrez.api_key = \"92471a1f25f1f658f6711d0c7bc31ff80708\"\n",
    "# Always add an email so the NCBI admin can contact you\n",
    "Entrez.email = \"gree9242@uri.edu\"\n",
    "# You will find these handle and record variables used often. I will specificy when I can.\n",
    "handle = Entrez.einfo()\n",
    "record = Entrez.read(handle)\n",
    "record.keys()\n",
    "# Print out database list\n",
    "record[\"DbList\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### See I told you\n",
    "\n",
    "To be able to manipulate these database we need to know what we can search for. To do this we need to veiw information about the individual databases.\n",
    "\n",
    "**If you are interested you can swap out the nucleotide database in the (db=\"nucleotide\") line for another database and view the description, entries, updates, and fields within each one**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is a description of the database Core Nucleotide db\n",
      "This is a count of all the entries in the database 434797561\n",
      "This is when the last update was done 2020/12/06 23:16\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.api_key = \"92471a1f25f1f658f6711d0c7bc31ff80708\"\n",
    "Entrez.email = \"gree9242@uri.edu\"\n",
    "handle = Entrez.einfo(db=\"nucleotide\")\n",
    "record = Entrez.read(handle)\n",
    "print(\"This is a description of the database \" + record[\"DbInfo\"][\"Description\"])\n",
    "print(\"This is a count of all the entries in the database \" + record[\"DbInfo\"][\"Count\"])\n",
    "print(\"This is when the last update was done \" + record[\"DbInfo\"][\"LastUpdate\"]) "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### The Esearch record can also help to parse out some of the information inside of the selected database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ALL, All Fields, All terms from all searchable fields\n",
      "UID, UID, Unique number assigned to each sequence\n",
      "FILT, Filter, Limits the records\n",
      "WORD, Text Word, Free text associated with record\n",
      "TITL, Title, Words in definition line\n",
      "KYWD, Keyword, Nonstandardized terms provided by submitter\n",
      "AUTH, Author, Author(s) of publication\n",
      "JOUR, Journal, Journal abbreviation of publication\n",
      "VOL, Volume, Volume number of publication\n",
      "ISS, Issue, Issue number of publication\n",
      "PAGE, Page Number, Page number(s) of publication\n",
      "ORGN, Organism, Scientific and common names of organism, and all higher levels of taxonomy\n",
      "ACCN, Accession, Accession number of sequence\n",
      "PACC, Primary Accession, Does not include retired secondary accessions\n",
      "GENE, Gene Name, Name of gene associated with sequence\n",
      "PROT, Protein Name, Name of protein associated with sequence\n",
      "ECNO, EC/RN Number, EC number for enzyme or CAS registry number\n",
      "PDAT, Publication Date, Date sequence added to GenBank\n",
      "MDAT, Modification Date, Date of last update\n",
      "SUBS, Substance Name, CAS chemical name or MEDLINE Substance Name\n",
      "PROP, Properties, Classification by source qualifiers and molecule type\n",
      "SQID, SeqID String, String identifier for sequence\n",
      "GPRJ, BioProject, BioProject\n",
      "SLEN, Sequence Length, Length of sequence\n",
      "FKEY, Feature key, Feature annotated on sequence\n",
      "PORG, Primary Organism, Scientific and common names of primary organism, and all higher levels of taxonomy\n",
      "COMP, Component Accession, Component accessions for an assembly\n",
      "ASSM, Assembly, Assembly\n",
      "DIV, Division, Division\n",
      "STRN, Strain, Strain\n",
      "ISOL, Isolate, Isolate\n",
      "CULT, Cultivar, Cultivar\n",
      "BRD, Breed, Breed\n",
      "BIOS, BioSample, BioSample\n"
     ]
    }
   ],
   "source": [
    "for field in record[\"DbInfo\"][\"FieldList\"]:\n",
    "    print(\"%(Name)s, %(FullName)s, %(Description)s\" % field)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**We need to set our host and pathogen names as variables**\n",
    "\n",
    "**This way we won't have to enter in this information everytime we move onto a new section**\n",
    "\n",
    "**It is helpful to be descriptive in this naming, such as providing genus and species**\n",
    "\n",
    "**Searching your host/pathogen in NCBI might help to refine the organism name**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Enter host organism name and gene\n",
    "host = 'Homo sapiens[Orgn]'\n",
    "\n",
    "#Enter pathogen organism name and gene\n",
    "pathogen = 'Human immunodeficiency virus 1[Orgn]'\n",
    "\n",
    "# Enter host and pathogen combined\n",
    "\n",
    "hp = 'Homo sapiens[Orgn] AND Human immunodeficiency virus 1[Orgn]'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Finding Records\n",
    "\n",
    "Now we will move onto finding records for our host and pathogen. This section will print out how many record we found and their accession ids. Accession ids are unique number applied to specific record of genetic information. Since there might be multiple instance of assembled human genomes, each verified and complete entry has its own unique identifier. We will be accessing the nucleotide database for host and pathogen information as this provides most specific results when using general names. We can use the assembly database to find ids, but this database is limited when using the efetch function in the next section."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "27704223 number of host ids found\n",
      "\n",
      "['NM_001389244.1', 'NM_001386188.2', 'NR_138058.2', 'NM_001324459.2', 'NM_001165037.2', 'NM_001265609.2', 'NM_001286378.2', 'NM_001329906.2', 'NM_001164720.3', 'NM_004194.5'] host id list\n",
      "\n",
      "929947 pathogen ids found\n",
      "\n",
      "['6XH3_D', '6XH2_D', '6XH1_D', '6XH0_D', '7JU1_X', '7K1Z_A', '6WB2_C', '6WB1_C', '6WB0_C', '6WAZ_C'] pathogen id list\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.api_key = \"92471a1f25f1f658f6711d0c7bc31ff80708\"\n",
    "Entrez.email = \"gree9242@uri.edu\"\n",
    "# Searchs assembly database for 10 inputs of host information and retrieves the accession numbers\n",
    "handle1 = Entrez.esearch(db='nucleotide',term=host,idtype=\"acc\",retmax='10')\n",
    "# Searchs nucleotide database for 10 inputs of pathogen information and retrieves the accession numbers\n",
    "handle2 = Entrez.esearch(db='nucleotide',term=pathogen,idtype=\"acc\",retmax='10')\n",
    "record1 = Entrez.read(handle1)\n",
    "record2 = Entrez.read(handle2)\n",
    "print('{} number of host ids found'.format(record1[\"Count\"]))\n",
    "print()\n",
    "print('{} host id list'.format(record1[\"IdList\"]))\n",
    "print()\n",
    "print('{} pathogen ids found'.format(record2[\"Count\"]))\n",
    "print()\n",
    "print('{} pathogen id list'.format(record2[\"IdList\"]))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### We can see here that there are many more instances of pathogen ids than host ids. This is why we limit our returns to 10 for the time being. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Accessing and saving full records\n",
    "\n",
    "Having full genbank records provides a large amount of information related to your study system. Best of all it can provide you with the publication and project associated with the id."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       NM_001389244            1831 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens keratin 40 (KRT40), transcript variant 4, mRNA.\n",
      "ACCESSION   NM_001389244 XM_017024189\n",
      "VERSION     NM_001389244.1\n",
      "KEYWORDS    RefSeq; RefSeq Select.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 1831)\n",
      "  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\n",
      "            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\n",
      "            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\n",
      "            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\n",
      "            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\n",
      "            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\n",
      "            Colabella C, Coppin G, D'Amata C, De Ridder D, De Rouck S,\n",
      "            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\n",
      "            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\n",
      "            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\n",
      "            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\n",
      "            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\n",
      "            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\n",
      "            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\n",
      "            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\n",
      "            MA.\n",
      "  TITLE     A reference map of the human binary protein interactome\n",
      "  JOURNAL   Nature 580 (7803), 402-408 (2020)\n",
      "   PUBMED   32296183\n",
      "REFERENCE   2  (bases 1 to 1831)\n",
      "  AUTHORS   Fragoza R, Das J, Wierbowski SD, Liang J, Tran TN, Liang S, Beltran\n",
      "            JF, Rivera-Erick CA, Ye K, Wang TY, Yao L, Mort M, Stenson PD,\n",
      "            Cooper DN, Wei X, Keinan A, Schimenti JC, Clark AG and Yu H.\n",
      "  TITLE     Extensive disruption of protein interactions by genetic variants\n",
      "            across the allele frequency spectrum in human populations\n",
      "  JOURNAL   Nat Commun 10 (1), 4141 (2019)\n",
      "   PUBMED   31515488\n",
      "  REMARK    Publication Status: Online-Only\n",
      "REFERENCE   3  (bases 1 to 1831)\n",
      "  AUTHORS   Chen S, Fragoza R, Klei L, Liu Y, Wang J, Roeder K, Devlin B and Yu\n",
      "            H.\n",
      "  TITLE     An interactome perturbation framework prioritizes damaging missense\n",
      "            mutations for developmental disorders\n",
      "  JOURNAL   Nat Genet 50 (7), 1032-1040 (2018)\n",
      "   PUBMED   29892012\n",
      "REFERENCE   4  (bases 1 to 1831)\n",
      "  AUTHORS   Yang X, Coulombe-Huntington J, Kang S, Sheynkman GM, Hao T,\n",
      "            Richardson A, Sun S, Yang F, Shen YA, Murray RR, Spirohn K, Begg\n",
      "            BE, Duran-Frigola M, MacWilliams A, Pevzner SJ, Zhong Q, Trigg SA,\n",
      "            Tam S, Ghamsari L, Sahni N, Yi S, Rodriguez MD, Balcha D, Tan G,\n",
      "            Costanzo M, Andrews B, Boone C, Zhou XJ, Salehi-Ashtiani K,\n",
      "            Charloteaux B, Chen AA, Calderwood MA, Aloy P, Roth FP, Hill DE,\n",
      "            Iakoucheva LM, Xia Y and Vidal M.\n",
      "  TITLE     Widespread Expansion of Protein Interaction Capabilities by\n",
      "            Alternative Splicing\n",
      "  JOURNAL   Cell 164 (4), 805-817 (2016)\n",
      "   PUBMED   26871637\n",
      "REFERENCE   5  (bases 1 to 1831)\n",
      "  AUTHORS   Sahni N, Yi S, Taipale M, Fuxman Bass JI, Coulombe-Huntington J,\n",
      "            Yang F, Peng J, Weile J, Karras GI, Wang Y, Kovacs IA, Kamburov A,\n",
      "            Krykbaeva I, Lam MH, Tucker G, Khurana V, Sharma A, Liu YY, Yachie\n",
      "            N, Zhong Q, Shen Y, Palagi A, San-Miguel A, Fan C, Balcha D, Dricot\n",
      "            A, Jordan DM, Walsh JM, Shah AA, Yang X, Stoyanova AK, Leighton A,\n",
      "            Calderwood MA, Jacob Y, Cusick ME, Salehi-Ashtiani K, Whitesell LJ,\n",
      "            Sunyaev S, Berger B, Barabasi AL, Charloteaux B, Hill DE, Hao T,\n",
      "            Roth FP, Xia Y, Walhout AJM, Lindquist S and Vidal M.\n",
      "  TITLE     Widespread macromolecular interaction perturbations in human\n",
      "            genetic disorders\n",
      "  JOURNAL   Cell 161 (3), 647-660 (2015)\n",
      "   PUBMED   25910212\n",
      "REFERENCE   6  (bases 1 to 1831)\n",
      "  AUTHORS   Langbein,L., Rogers,M.A., Praetzel-Wunder,S., Bockler,D.,\n",
      "            Schirmacher,P. and Schweizer,J.\n",
      "  TITLE     Novel type I hair keratins K39 and K40 are the last to be expressed\n",
      "            in differentiation of the hair: completion of the human hair\n",
      "            keratin catalog\n",
      "  JOURNAL   J Invest Dermatol 127 (6), 1532-1535 (2007)\n",
      "   PUBMED   17301834\n",
      "  REMARK    GeneRIF: KRT39 and KRT40 expression patterns complete sequential\n",
      "            expression of the 16 hair keratins in the human hair follicle\n",
      "REFERENCE   7  (bases 1 to 1831)\n",
      "  AUTHORS   Schweizer J, Bowden PE, Coulombe PA, Langbein L, Lane EB, Magin TM,\n",
      "            Maltais L, Omary MB, Parry DA, Rogers MA and Wright MW.\n",
      "  TITLE     New consensus nomenclature for mammalian keratins\n",
      "  JOURNAL   J Cell Biol 174 (2), 169-174 (2006)\n",
      "   PUBMED   16831889\n",
      "REFERENCE   8  (bases 1 to 1831)\n",
      "  AUTHORS   Rogers MA, Winter H, Langbein L, Bleiler R and Schweizer J.\n",
      "  TITLE     The human type I keratin gene family: characterization of new hair\n",
      "            follicle specific members and evaluation of the chromosome 17q21.2\n",
      "            gene domain\n",
      "  JOURNAL   Differentiation 72 (9-10), 527-540 (2004)\n",
      "   PUBMED   15617563\n",
      "REFERENCE   9  (bases 1 to 1831)\n",
      "  AUTHORS   Hesse M, Zimek A, Weber K and Magin TM.\n",
      "  TITLE     Comprehensive analysis of keratin gene clusters in humans and\n",
      "            rodents\n",
      "  JOURNAL   Eur J Cell Biol 83 (1), 19-26 (2004)\n",
      "   PUBMED   15085952\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AC007455.7.\n",
      "            On Dec 3, 2020 this sequence version replaced XM_017024189.2.\n",
      "            \n",
      "            Summary: This gene encodes a member of the type I (acidic) keratin\n",
      "            family, which belongs to the superfamily of intermediate filament\n",
      "            (IF) proteins. Keratins are heteropolymeric structural proteins\n",
      "            which form the intermediate filament. These filaments, along with\n",
      "            actin microfilaments and microtubules, compose the cytoskeleton of\n",
      "            epithelial cells. The type I keratin genes are clustered in a\n",
      "            region of chromosome 17q12-q21. [provided by RefSeq, Jul 2009].\n",
      "            \n",
      "            Sequence Note: The RefSeq transcript and protein were derived from\n",
      "            genomic sequence to make the sequence consistent with the reference\n",
      "            genome assembly. The genomic coordinates used for the transcript\n",
      "            record were based on alignments.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: AJ786658.1, BK004055.1 [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "            \n",
      "            ##RefSeq-Attributes-START##\n",
      "            RefSeq Select criteria :: based on manual assertion, conservation,\n",
      "                                      expression, longest protein\n",
      "            ##RefSeq-Attributes-END##\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-500               AC007455.7         65872-66371\n",
      "            501-583             AC007455.7         67070-67152\n",
      "            584-740             AC007455.7         67735-67891\n",
      "            741-902             AC007455.7         69047-69208\n",
      "            903-1028            AC007455.7         69288-69413\n",
      "            1029-1249           AC007455.7         71174-71394\n",
      "            1250-1831           AC007455.7         71902-72483\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..1831\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"17\"\n",
      "                     /map=\"17q21.2\"\n",
      "     gene            1..1831\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"keratin 40\"\n",
      "                     /db_xref=\"GeneID:125115\"\n",
      "                     /db_xref=\"HGNC:HGNC:26707\"\n",
      "                     /db_xref=\"MIM:616679\"\n",
      "     exon            1..500\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     CDS             54..1349\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"type I hair keratin KA36; cytokeratin-40; keratin\n",
      "                     40, type I\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"keratin, type I cytoskeletal 40\"\n",
      "                     /protein_id=\"NP_001376173.1\"\n",
      "                     /db_xref=\"GeneID:125115\"\n",
      "                     /db_xref=\"HGNC:HGNC:26707\"\n",
      "                     /db_xref=\"MIM:616679\"\n",
      "                     /translation=\"MTSDCSSTHCSPESCGTASGCAPASSCSVETACLPGTCATSRCQ\n",
      "                     TPSFLSRSRGLTGCLLPCYFTGSCNSPCLVGNCAWCEDGVFTSNEKETMQFLNDRLAS\n",
      "                     YLEKVRSLEETNAELESRIQEQCEQDIPMVCPDYQRYFNTIEDLQQKILCTKAENSRL\n",
      "                     AVQLDNCKLATDDFKSKYESELSLRQLLEADISSLHGILEELTLCKSDLEAHVESLKE\n",
      "                     DLLCLKKNHEEEVNLLREQLGDRLSVELDTAPTLDLNRVLDEMRCQCETVLANNRREA\n",
      "                     EEWLAVQTEELNQQQLSSAEQLQGCQMEILELKRTASALEIELQAQQSLTESLECTVA\n",
      "                     ETEAQYSSQLAQIQCLIDNLENQLAEIRCDLERQNQEYQVLLDVKARLEGEINTYWGL\n",
      "                     LDSEDSRLSCSPCSTTCTSSNTCEPCSAYVICTVENCCL\"\n",
      "     misc_feature    54..320\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (Q6A162.2);\n",
      "                     Region: Head\"\n",
      "     misc_feature    321..425\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (Q6A162.2);\n",
      "                     Region: Coil 1A\"\n",
      "     misc_feature    426..458\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (Q6A162.2);\n",
      "                     Region: Linker 1\"\n",
      "     misc_feature    459..761\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (Q6A162.2);\n",
      "                     Region: Coil 1B\"\n",
      "     misc_feature    762..809\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (Q6A162.2);\n",
      "                     Region: Linker 12\"\n",
      "     misc_feature    810..1241\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (Q6A162.2);\n",
      "                     Region: Coil 2\"\n",
      "     misc_feature    1065..1067\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"Stutter; propagated from UniProtKB/Swiss-Prot\n",
      "                     (Q6A162.2); other site\"\n",
      "     misc_feature    1242..1346\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (Q6A162.2);\n",
      "                     Region: Tail\"\n",
      "     exon            501..583\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            584..740\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            741..902\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            903..1028\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1029..1249\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1250..1831\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      1570..1575\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      1603\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"major polyA site\"\n",
      "     regulatory      1811..1816\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      1831\n",
      "                     /gene=\"KRT40\"\n",
      "                     /gene_synonym=\"CK-40; K40; KA36\"\n",
      "                     /note=\"major polyA site\"\n",
      "ORIGIN      \n",
      "        1 gagtcaaggc aactgaagtt ttgcagccag tctctgtctt tgcttctgga aggatgactt\n",
      "       61 ctgactgctc ctccacacac tgctctcctg agtcctgtgg cacggcttcc ggttgtgcac\n",
      "      121 ctgcctcaag ctgctccgtg gaaacagctt gtctccccgg tacctgtgct acatcccgat\n",
      "      181 gtcagactcc aagcttccta tccaggtctc gcgggctgac tggttgcctc ctgccatgct\n",
      "      241 actttactgg gagttgtaat agtccctgct tggtggggaa ctgtgcctgg tgtgaggatg\n",
      "      301 gggtgttcac tagcaatgag aaggagacga tgcagttcct gaatgacaga ctcgccagct\n",
      "      361 atctggagaa ggtgcgcagc ctggaggaga ccaacgcaga gctggaatcc aggatccaag\n",
      "      421 aacaatgtga acaggatatc ccaatggtgt gcccggatta tcagcgttac ttcaacacca\n",
      "      481 ttgaagatct ccaacaaaag atcttatgca cgaaagcaga gaattctaga cttgctgtac\n",
      "      541 agcttgacaa ctgcaaactg gccactgatg actttaagtc aaagtacgag agtgaactgt\n",
      "      601 cccttcgcca gctgttagag gctgacatca gcagcctgca tgggatcctg gaggaactga\n",
      "      661 ccctgtgcaa atctgatctg gaggcccatg tggagtctct gaaggaagat ctcctttgcc\n",
      "      721 ttaagaaaaa ccatgaagag gaagtcaact tgcttcgtga acagcttggc gaccgcctca\n",
      "      781 gtgtggagct ggacactgcc cccacccttg acctcaacag ggtcctggat gagatgcgct\n",
      "      841 gtcagtgtga aacggtgctt gccaacaatc gcagagaagc tgaagaatgg ttggctgttc\n",
      "      901 agacagaaga gctgaatcag cagcaactgt ccagcgcgga gcagctgcag ggctgccaga\n",
      "      961 tggagatctt ggaactgaaa cgcacagcca gtgctctgga aattgagctc caagcacagc\n",
      "     1021 aaagcctgac agaatctctg gaatgcaccg tggcagaaac cgaggcccag tacagctccc\n",
      "     1081 agctggccca aattcagtgt ctgatcgata acctggagaa ccagctggcc gagatccgct\n",
      "     1141 gcgacctgga gcgacagaac caggagtacc aggtgctcct ggacgtgaag gcccggctgg\n",
      "     1201 agggtgagat caacacgtac tggggcctgc tggacagcga ggacagcagg ctttcctgta\n",
      "     1261 gcccatgttc gaccacatgc acctcgagca acacttgtga gccatgctca gcctatgtca\n",
      "     1321 tctgcactgt tgaaaactgc tgcttgtgaa tgttaaggac agcaggtaga ggaatcaaag\n",
      "     1381 ccagcagcat cctagaaagg agatggggct tcaaccaggg acatccaaac cggaagcaca\n",
      "     1441 aatccaggag gctctcatat cctggaattg ccctccaggt atcctcagtc tttctgcttt\n",
      "     1501 actatttcct ctccttccgg cttatttgtg gtatactgga agcccataaa gcattcttgc\n",
      "     1561 tcatctccaa ataaaacgtg ttttattcct ttagcgtggt aaatatagct ctatggctga\n",
      "     1621 ttactgttga tagaatattc attttatggc aaggaaatca tctaaaccca taaccatagt\n",
      "     1681 ttcaatgtac ataaacccag gctgaaaata ttttttaact cctcaaatac aaataaatta\n",
      "     1741 atgaacccat gagaaatgtc atatatctta gaaaatagtt gtgtgtatct ttctgaggaa\n",
      "     1801 aaatattagg aataaaatat gataaaatgt a\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       NM_001386188            5584 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens centromere protein I (CENPI), transcript variant 4,\n",
      "            mRNA.\n",
      "ACCESSION   NM_001386188 XM_011530894\n",
      "VERSION     NM_001386188.2\n",
      "KEYWORDS    RefSeq; RefSeq Select.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 5584)\n",
      "  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\n",
      "            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\n",
      "            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\n",
      "            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\n",
      "            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\n",
      "            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\n",
      "            Colabella C, Coppin G, D'Amata C, De Ridder D, De Rouck S,\n",
      "            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\n",
      "            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\n",
      "            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\n",
      "            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\n",
      "            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\n",
      "            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\n",
      "            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\n",
      "            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\n",
      "            MA.\n",
      "  TITLE     A reference map of the human binary protein interactome\n",
      "  JOURNAL   Nature 580 (7803), 402-408 (2020)\n",
      "   PUBMED   32296183\n",
      "REFERENCE   2  (bases 1 to 5584)\n",
      "  AUTHORS   Ding N, Li R, Shi W and He C.\n",
      "  TITLE     CENPI is overexpressed in colorectal cancer and regulates cell\n",
      "            migration and invasion\n",
      "  JOURNAL   Gene 674, 80-86 (2018)\n",
      "   PUBMED   29936263\n",
      "  REMARK    GeneRIF: CENPI is overexpressed in colorectal cancer (CRC) and\n",
      "            functions as oncogene in modulating CRC cell migration, invasion\n",
      "            and epithelial-mesenchymal transition process.\n",
      "REFERENCE   3  (bases 1 to 5584)\n",
      "  AUTHORS   Shono N, Ohzeki J, Otake K, Martins NM, Nagase T, Kimura H,\n",
      "            Larionov V, Earnshaw WC and Masumoto H.\n",
      "  TITLE     CENP-C and CENP-I are key connecting factors for kinetochore and\n",
      "            CENP-A assembly\n",
      "  JOURNAL   J Cell Sci 128 (24), 4572-4587 (2015)\n",
      "   PUBMED   26527398\n",
      "  REMARK    GeneRIF: CENP-C and CENP-I are key factors connecting kinetochore\n",
      "            to CENP-A assembly.\n",
      "REFERENCE   4  (bases 1 to 5584)\n",
      "  AUTHORS   Jiang D, Li J, Xiao X, Li S, Jia X, Sun W, Guo X and Zhang Q.\n",
      "  TITLE     Detection of mutations in LRPAP1, CTSH, LEPREL1, ZNF644, SLC39A5,\n",
      "            and SCO2 in 298 families with early-onset high myopia by exome\n",
      "            sequencing\n",
      "  JOURNAL   Invest Ophthalmol Vis Sci 56 (1), 339-345 (2014)\n",
      "   PUBMED   25525168\n",
      "  REMARK    GeneRIF: mutation in LRPAP1 is associated with high myopia. Further\n",
      "            studies are expected to evaluate the pathogenicity of the variants\n",
      "            in CTSH, LEPREL1, ZNF644, SLC39A5, and SCO2.\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   5  (bases 1 to 5584)\n",
      "  AUTHORS   Foltz DR, Jansen LE, Black BE, Bailey AO, Yates JR 3rd and\n",
      "            Cleveland DW.\n",
      "  TITLE     The human CENP-A centromeric nucleosome-associated complex\n",
      "  JOURNAL   Nat Cell Biol 8 (5), 458-469 (2006)\n",
      "   PUBMED   16622419\n",
      "REFERENCE   6  (bases 1 to 5584)\n",
      "  AUTHORS   Nousiainen M, Sillje HH, Sauer G, Nigg EA and Korner R.\n",
      "  TITLE     Phosphoproteome analysis of the human mitotic spindle\n",
      "  JOURNAL   Proc Natl Acad Sci U S A 103 (14), 5391-5396 (2006)\n",
      "   PUBMED   16565220\n",
      "REFERENCE   7  (bases 1 to 5584)\n",
      "  AUTHORS   Obuse C, Yang H, Nozaki N, Goto S, Okazaki T and Yoda K.\n",
      "  TITLE     Proteomics analysis of the centromere complex from HeLa interphase\n",
      "            cells: UV-damaged DNA binding protein 1 (DDB-1) is a component of\n",
      "            the CEN-complex, while BMI-1 is transiently co-localized with the\n",
      "            centromeric region in interphase\n",
      "  JOURNAL   Genes Cells 9 (2), 105-120 (2004)\n",
      "   PUBMED   15009096\n",
      "REFERENCE   8  (bases 1 to 5584)\n",
      "  AUTHORS   Liu ST, Hittle JC, Jablonski SA, Campbell MS, Yoda K and Yen TJ.\n",
      "  TITLE     Human CENP-I specifies localization of CENP-F, MAD1 and MAD2 to\n",
      "            kinetochores and is essential for mitosis\n",
      "  JOURNAL   Nat Cell Biol 5 (4), 341-345 (2003)\n",
      "   PUBMED   12640463\n",
      "REFERENCE   9  (bases 1 to 5584)\n",
      "  AUTHORS   Roberts RG, Kendall E, Vetrie D and Bobrow M.\n",
      "  TITLE     Sequence and chromosomal location of a human homologue of LRPR1, an\n",
      "            FSH primary response gene\n",
      "  JOURNAL   Genomics 37 (1), 122-124 (1996)\n",
      "   PUBMED   8921378\n",
      "REFERENCE   10 (bases 1 to 5584)\n",
      "  AUTHORS   Slegtenhorst-Eegdeman KE, Post M, Baarends WM, Themmen AP and\n",
      "            Grootegoed JA.\n",
      "  TITLE     Regulation of gene expression in Sertoli cells by\n",
      "            follicle-stimulating hormone (FSH): cloning and characterization of\n",
      "            LRPR1, a primary response gene encoding a leucine-rich protein\n",
      "  JOURNAL   Mol Cell Endocrinol 108 (1-2), 115-124 (1995)\n",
      "   PUBMED   7758824\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AL109963.4.\n",
      "            On Nov 24, 2020 this sequence version replaced NM_001386188.1.\n",
      "            \n",
      "            Summary: This gene encodes a centromere protein that is a component\n",
      "            of the CENPA-NAC (nucleosome-associated) complex. This complex is\n",
      "            critical for accurate chromosome alignment and segregation and it\n",
      "            ensures proper mitotic progression. This protein regulates the\n",
      "            recruitment of kinetochore-associated proteins that are required to\n",
      "            generate the spindle checkpoint signal. The product of this gene is\n",
      "            involved in the response of gonadal tissues to follicle-stimulating\n",
      "            hormone. Mutations in this gene may be involved in human X-linked\n",
      "            disorders of gonadal development and gametogenesis. Alternate\n",
      "            splicing results in multiple transcript variants. A pseudogene of\n",
      "            this gene is found on chromosome 13. [provided by RefSeq, Jan\n",
      "            2016].\n",
      "            \n",
      "            Sequence Note: The RefSeq transcript and protein were derived from\n",
      "            genomic sequence to make the sequence consistent with the reference\n",
      "            genome assembly. The genomic coordinates used for the transcript\n",
      "            record were based on alignments.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##RefSeq-Attributes-START##\n",
      "            RefSeq Select criteria :: based on manual assertion, conservation,\n",
      "                                      expression, longest protein\n",
      "            ##RefSeq-Attributes-END##\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-45                AL109963.4         1302-1346\n",
      "            46-141              AL109963.4         1542-1637\n",
      "            142-380             AL109963.4         4156-4394\n",
      "            381-518             AL109963.4         5372-5509\n",
      "            519-637             AL109963.4         12571-12689\n",
      "            638-745             AL109963.4         12989-13096\n",
      "            746-794             AL109963.4         23500-23548\n",
      "            795-841             AL109963.4         23836-23882\n",
      "            842-931             AL109963.4         29807-29896\n",
      "            932-1060            AL109963.4         30236-30364\n",
      "            1061-1228           AL109963.4         30596-30763\n",
      "            1229-1349           AL109963.4         31814-31934\n",
      "            1350-1441           AL109963.4         33080-33171\n",
      "            1442-1559           AL109963.4         35288-35405\n",
      "            1560-1624           AL109963.4         35490-35554\n",
      "            1625-1719           AL109963.4         43764-43858\n",
      "            1720-1855           AL109963.4         48162-48297\n",
      "            1856-1980           AL109963.4         49251-49375\n",
      "            1981-2029           AL109963.4         50861-50909\n",
      "            2030-2248           AL109963.4         51041-51259\n",
      "            2249-2290           AL109963.4         64626-64667\n",
      "            2291-5584           AL109963.4         65931-69224\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..5584\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"X\"\n",
      "                     /map=\"Xq22.1\"\n",
      "     gene            1..5584\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /note=\"centromere protein I\"\n",
      "                     /db_xref=\"GeneID:2491\"\n",
      "                     /db_xref=\"HGNC:HGNC:3968\"\n",
      "                     /db_xref=\"MIM:300065\"\n",
      "     exon            1..45\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            46..141\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            142..380\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     CDS             155..2425\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /note=\"isoform 1 is encoded by transcript variant 4;\n",
      "                     leucine-rich primary response protein 1;\n",
      "                     follicle-stimulating hormone primary response protein; FSH\n",
      "                     primary response 1; interphase centromere complex protein\n",
      "                     19\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"centromere protein I isoform 1\"\n",
      "                     /protein_id=\"NP_001373117.1\"\n",
      "                     /db_xref=\"GeneID:2491\"\n",
      "                     /db_xref=\"HGNC:HGNC:3968\"\n",
      "                     /db_xref=\"MIM:300065\"\n",
      "                     /translation=\"MSPQKRVKNVQAQNRTSQGSSSFQTTLSAWKVKQDPSNSKNISK\n",
      "                     HGQNNPVGDYEHADDQAEEDALQMAVGYFEKGPIKASQNKDKTLEKHLKTVENVAWKN\n",
      "                     GLASEEIDILLNIALSGKFGNAVNTRILKCMIPATVISEDSVVKAVSWLCVGKCSGST\n",
      "                     KVLFYRWLVAMFDFIDRKEQINLLYGFFFASLQDDALCPYVCHLLYLLTKKENVKPFR\n",
      "                     VRKLLDLQAKMGMQPHLQALLSLYKFFAPALISVSLPVRKKIYFKNSENLWKTALLAV\n",
      "                     KQRNRGPSPEPLKLMLGPANVRPLKRKWNSLSVIPVLNSSSYTKECGKKEMSLSDCLN\n",
      "                     RSGSFPLEQLQSFPQLLQNIHCLELPSQMGSVLNNSLLLHYINCVRDEPVLLRFYYWL\n",
      "                     SQTLQEECIWYKVNNYEHGKEFTNFLDTIIRAECFLQEGFYSCEAFLYKSLPLWDGLC\n",
      "                     CRSQFLQLVSWIPFSSFSEVKPLLFDHLAQLFFTSTIYFKCSVLQSLKELLQNWLLWL\n",
      "                     SMDIHMKPVTNSPLETTLGGSMNSVSKLIHYVGWLSTTAMRLESNNTFLLHFILDFYE\n",
      "                     KVCDIYINYNLPLVVLFPPGIFYSALLSLDTSILNQLCFIMHRYRKNLTAAKKNELVQ\n",
      "                     KTKSEFNFSSKTYQEFNHYLTSMVGCLWTSKPFGKGIYIDPEILEKTGVAEYKNSLNV\n",
      "                     VHHPSFLSYAVSFLLQESPEERTVNVSSIRGKKWSWYLDYLFSQGLQGLKLFIRSSVH\n",
      "                     HSSIPRAEGINCNNQY\"\n",
      "     exon            381..518\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            519..637\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            638..745\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            746..794\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            795..841\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            842..931\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            932..1060\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1061..1228\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1229..1349\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1350..1441\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1442..1559\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1560..1624\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1625..1719\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1720..1855\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1856..1980\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1981..2029\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2030..2248\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2249..2290\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2291..5584\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      3110..3115\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      3136\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /note=\"major polyA site\"\n",
      "     regulatory      5460..5465\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /note=\"hexamer: AAGAAA\"\n",
      "     polyA_site      5481\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /note=\"major polyA site\"\n",
      "     regulatory      5560..5565\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      5584\n",
      "                     /gene=\"CENPI\"\n",
      "                     /gene_synonym=\"CENP-I; FSHPRH1; LRPR1\"\n",
      "ORIGIN      \n",
      "        1 attcggtggt cagaggcctg tgcggctgca ggtagagtgt cttaggaacc taggaaataa\n",
      "       61 ctcggaacct gtaacgtccc actggtttgg acatattcct ctcctgatct ggcctcatct\n",
      "      121 gttccaggga ggtgggattg aaacatatgc agtaatgtca cctcaaaaga gagttaagaa\n",
      "      181 cgtccaggca caaaacagga cttcacaagg tagtagtagt tttcagacca cgctttcagc\n",
      "      241 ctggaaagta aaacaggatc caagcaactc gaagaacatc tcaaaacatg gacaaaacaa\n",
      "      301 tccagtggga gattatgaac atgctgatga tcaagctgaa gaagatgctt tgcaaatggc\n",
      "      361 agtgggatat tttgagaaag gtcccattaa agcttcacag aataaagata aaaccttgga\n",
      "      421 aaaacacttg aaaactgtgg aaaatgtggc ttggaagaat gggttagctt cagaagaaat\n",
      "      481 tgatattcta ttaaatattg cactcagtgg caaatttgga aatgctgtaa acacacggat\n",
      "      541 attgaagtgc atgatcccag caacagtaat atcagaagat tctgtggtta aggcagtctc\n",
      "      601 ctggctttgt gttggcaagt gttctggtag caccaaggta cttttttatc gttggctggt\n",
      "      661 tgcaatgttt gacttcattg atcgtaagga gcaaattaac ttgctctatg gcttcttttt\n",
      "      721 tgcttcattg caagatgatg cactgtgccc ttatgtttgc catttgttat atttacttac\n",
      "      781 caaaaaagag aatgtcaaac catttcgtgt gagaaaactg cttgatcttc aggccaaaat\n",
      "      841 gggaatgcag cctcatctcc aggctttgtt gtcactgtat aagttctttg ctcctgctct\n",
      "      901 gatttcagta tctttgcctg taaggaagaa gatatatttt aagaattcag agaatctatg\n",
      "      961 gaagacggct ctgcttgccg tgaagcaaag aaaccgggga ccttctccag aacctctgaa\n",
      "     1021 gttgatgtta ggtccagcta atgttcgtcc tctaaaaaga aagtggaatt ctctctcagt\n",
      "     1081 tataccagtg ctcaattcca gtagctacac taaagaatgt ggaaaaaaag agatgagtct\n",
      "     1141 ttctgattgt ctgaatagaa gtggatcatt tccactagaa caacttcaaa gcttccccca\n",
      "     1201 acttttacag aacatccatt gcttagagct gccttctcag atgggctcag tgctaaacaa\n",
      "     1261 ctctctgctg cttcactaca ttaactgtgt cagagatgag ccagtcttgc tgaggtttta\n",
      "     1321 ttactggttg agtcaaacat tacaagaaga atgtatttgg tacaaggtga ataattatga\n",
      "     1381 acatggaaaa gaatttacca acttcctgga taccatcatc agggcagagt gcttcttaca\n",
      "     1441 agaggggttt tattcctgtg aagcattcct gtataagagc cttcctctct gggatggcct\n",
      "     1501 ttgttgtcgg tcacagttcc ttcagcttgt gagctggatt ccttttagta gcttctctga\n",
      "     1561 ggtgaaacca cttctttttg accatctagc gcagctcttc tttacatcaa ccatttattt\n",
      "     1621 caagtgtagt gtgcttcaga gtctgaaaga gctattgcag aattggctgt tgtggctttc\n",
      "     1681 tatggacatt cacatgaaac ctgttacaaa cagtcctcta gagacaactt tgggtggatc\n",
      "     1741 catgaactct gtgtctaaac tgatccacta tgtagggtgg ctatccacta ctgcaatgcg\n",
      "     1801 cttggagagc aacaatactt tcttgctgca ctttattttg gatttctatg agaaggtgtg\n",
      "     1861 tgacatatat ataaattata accttccatt agtggtattg tttcctcctg ggatcttcta\n",
      "     1921 ttctgcactc ctcagcctgg ataccagcat cctgaaccag ctgtgtttta ttatgcacag\n",
      "     1981 atatcgtaaa aatttgactg ccgcaaagaa aaatgagttg gtacaaaaga caaaatcaga\n",
      "     2041 gttcaatttc agcagcaaga cttatcaaga atttaatcac tatttgacat caatggttgg\n",
      "     2101 ttgcctgtgg acgtccaaac cctttgggaa aggaatatat attgaccctg aaatcctaga\n",
      "     2161 aaaaactgga gtggctgaat ataaaaacag tttaaatgta gtccatcatc cttctttctt\n",
      "     2221 gagttacgct gtttcctttt tgctacagga aagcccagaa gaaaggacag taaatgtgag\n",
      "     2281 ctctattcgg ggaaagaaat ggagctggta tttggactat ttattttcac aggggttaca\n",
      "     2341 aggcttgaaa ctttttataa gaagtagtgt tcatcattct tccattccca gagcagaggg\n",
      "     2401 cataaactgc aacaatcaat attaaatgaa tgttgacata aactgaacac actggactaa\n",
      "     2461 actcactcct cattgctaga gcaaagtggc tcatcttgag ttcccatttt catttcactg\n",
      "     2521 acagactgcc atcctcaagg agtactcaga ctggccttct gttcatggct taggagagcc\n",
      "     2581 ttggtgtgcc taactgattt ttcaaaattt agattttttt agcctaccag tgaaaaatga\n",
      "     2641 ccccttcatc atcaggctct gcgttctacc aaattgtatg taaaaagaca catctgtttt\n",
      "     2701 gtggtaggat tttttcacat ttttgggtac tatgagctgc attgatggaa gacagcaggc\n",
      "     2761 aatatgtggt gacagttaac tcacagacat aaacatgcaa aatactttgc tgtctctggg\n",
      "     2821 gatattgcca tttttcttac tgtgagcaac agcaccaaca ccaagttaac aggatgcaac\n",
      "     2881 atgtgtatga ctctaaaagc cctaagtagt tggtaacttc ctgggccttc aatcatagca\n",
      "     2941 atttgatgag ggaaggaagg ggagaggatt tgttgggtaa tcaagacatt cccgtatatg\n",
      "     3001 tctgatttca tggaactgct ctattttgtt tgtgtgtatt gtatatgtat atgtgtatgt\n",
      "     3061 gtgcgtgtat gtgtgtgtct gtagcttcag tttttaagtg taaggactaa ataaactaac\n",
      "     3121 tgaaatttta ctttcagaag gaattctgaa gttttaaaaa agttaataat agctagaata\n",
      "     3181 tcttgctgaa aatgtactat aaaattttgc ctgctgggcg tggtggctca cgcctgtaat\n",
      "     3241 cccagcactt tgggaggcca aggtgggtgg atcatgaggt caggagttcg agatgaggct\n",
      "     3301 ggccaatatg gtaaaaccct gtctctacta aaaatacaaa aaattggccg ggcatgccgg\n",
      "     3361 cacgtgcctg tagtcccagc tactcaggag gctggggcag gagaatcgct tgaacccggg\n",
      "     3421 aggcggaggt tgcagtgagc caagatcgcg ccactgcact ccagcgagac tctgtctcaa\n",
      "     3481 aaaaaaaaaa aaaaattacc tcaatgctat tgaataattc tcccttctag taaaattcgt\n",
      "     3541 atcatttgag aagaatcaag ctgaattaat ttttgtcttt aaggatgcat acttggggtt\n",
      "     3601 cttttttttc cccatttaaa aaacaaaccc taaactatat tattggatgg cgttcagact\n",
      "     3661 agaaatccca ctcgtccatt gactcattaa ttgattcaat aaatatttat tgagctcctt\n",
      "     3721 tctctgtgcc aggtactctt ctgtgtgctg gttatacagt tgtgaccaat acaaagtacc\n",
      "     3781 cgggtttttg tttttgtttt tgttttttga gatggagtct cactctgttg cccaggctgg\n",
      "     3841 agtgcagagg catgatcttg gctcactgca acctctactt tctggttcaa gcaattctcc\n",
      "     3901 tgcctcagcc tcacgaatag ctgggattac aggcacctac caccacgccc agccaatttt\n",
      "     3961 tgtattttta gtagagacag ggttccgtca tgttgaccag gctggacttg aactcttgat\n",
      "     4021 ctcaagtgat caacccgcct tggcctccca aagtgttggg attagaggtg tgagccattg\n",
      "     4081 cacccagccc aaagtacctg tttttatgga gtttacattc caatggagaa gacagacaat\n",
      "     4141 aacatatatc aaatgctata aagaaagggc agggaatagg gatacaatgt gttgaatttt\n",
      "     4201 gtcaaatgct ttttttgtat cttttgagat tatcatatgg cttttgtcct tcattctgtt\n",
      "     4261 aacatgatat atcacattta ctgatttgtg tatactgaat catctttgca tcccgggatg\n",
      "     4321 aatcccactg ggacagaatg tggagagggt tgaaacttta gataagatgg tttgggaaag\n",
      "     4381 cctgtttgaa gagatgacat atgcagaggc cccagtgaag taagcatatg agtcatataa\n",
      "     4441 atatttgggg aaagagcatt ctggacaaag gtaaaaaaca agtgaaaagg ccctaaaatg\n",
      "     4501 ttaacttact tggcatgttt gaggaataac aaagaggcca gtaaggttac agcatagtga\n",
      "     4561 ttaagagaga agaatggtaa gaagtgacat aaggaagaaa gccaggtcct tatagcctgg\n",
      "     4621 catggatatg taaatgggaa gcaattagat tgagaagagg agtgacataa tttgatttac\n",
      "     4681 ctttatgaaa tatggagcta caatgtcaag gttagactga agaagattaa agaggaaagc\n",
      "     4741 agagactggt taggttatta tagtgtccta ggtaacagtt ttggacaagt gtgataaatg\n",
      "     4801 ttgaggtggg aggggttaga ggttggattc agactctgtt ttgtaagtag agaagataat\n",
      "     4861 gtctgctgat agcttggata tgaggaggaa aaggagagga gtaaaggatg actcagattt\n",
      "     4921 ttgacctgtc aattgggtga actctgagat taaattctgt tttggctatg ttaggttgga\n",
      "     4981 aatgctgtgt aggcaattgg atatccaagt ctggacttca gagtacaatt tgggactaga\n",
      "     5041 aaattaattt gggagtcatt agggaataac catgactttg gatgagatca cctagtacag\n",
      "     5101 ctagagaaga gaaggtagca aaagacagag acctaaggta tgccagcatt gagaagtaga\n",
      "     5161 ggagaagagg aaccagcaaa agactgagaa ggacccacca gtgacctaga agaaaaatca\n",
      "     5221 ggaggctggt attctggaag ccatcagaag aaaatgtttc acaaaaggaa ggtagtattg\n",
      "     5281 aatggtgtga aatgttacct atatacctgg aaaaaagcaa cttcagctgc tttttaagta\n",
      "     5341 aatgtgatag tttgtactgc aaataacttt ccatattgct tttcagatca tgatattttg\n",
      "     5401 gtccttaaga tcatgtagaa ctttccattg tgagaaggga gttgattatt ttatcattaa\n",
      "     5461 agaaactttt ctaaacttaa atttatgaaa aataattttt cccttaggtt tgtctttttg\n",
      "     5521 ttcaataatg attccctggg tatctttttt tgccccacaa ataaaatact tagacatacc\n",
      "     5581 tgga\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       NR_138058               1272 bp    RNA     linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens ADP ribosylation factor like GTPase 16 (ARL16),\n",
      "            transcript variant 4, non-coding RNA.\n",
      "ACCESSION   NR_138058\n",
      "VERSION     NR_138058.2\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 1272)\n",
      "  AUTHORS   Haenig C, Atias N, Taylor AK, Mazza A, Schaefer MH, Russ J,\n",
      "            Riechers SP, Jain S, Coughlin M, Fontaine JF, Freibaum BD,\n",
      "            Brusendorf L, Zenkner M, Porras P, Stroedicke M, Schnoegl S,\n",
      "            Arnsburg K, Boeddrich A, Pigazzini L, Heutink P, Taylor JP,\n",
      "            Kirstein J, Andrade-Navarro MA, Sharan R and Wanker EE.\n",
      "  TITLE     Interactome Mapping Provides a Network of Neurodegenerative Disease\n",
      "            Proteins and Uncovers Widespread Protein Aggregation in Affected\n",
      "            Brains\n",
      "  JOURNAL   Cell Rep 32 (7), 108050 (2020)\n",
      "   PUBMED   32814053\n",
      "REFERENCE   2  (bases 1 to 1272)\n",
      "  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\n",
      "            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\n",
      "            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\n",
      "            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\n",
      "            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\n",
      "            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\n",
      "            Colabella C, Coppin G, D'Amata C, De Ridder D, De Rouck S,\n",
      "            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\n",
      "            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\n",
      "            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\n",
      "            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\n",
      "            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\n",
      "            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\n",
      "            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\n",
      "            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\n",
      "            MA.\n",
      "  TITLE     A reference map of the human binary protein interactome\n",
      "  JOURNAL   Nature 580 (7803), 402-408 (2020)\n",
      "   PUBMED   32296183\n",
      "REFERENCE   3  (bases 1 to 1272)\n",
      "  AUTHORS   Fragoza R, Das J, Wierbowski SD, Liang J, Tran TN, Liang S, Beltran\n",
      "            JF, Rivera-Erick CA, Ye K, Wang TY, Yao L, Mort M, Stenson PD,\n",
      "            Cooper DN, Wei X, Keinan A, Schimenti JC, Clark AG and Yu H.\n",
      "  TITLE     Extensive disruption of protein interactions by genetic variants\n",
      "            across the allele frequency spectrum in human populations\n",
      "  JOURNAL   Nat Commun 10 (1), 4141 (2019)\n",
      "   PUBMED   31515488\n",
      "  REMARK    Publication Status: Online-Only\n",
      "REFERENCE   4  (bases 1 to 1272)\n",
      "  AUTHORS   Yang YK, Qu H, Gao D, Di W, Chen HW, Guo X, Zhai ZH and Chen DY.\n",
      "  TITLE     ARF-like protein 16 (ARL16) inhibits RIG-I by binding with its\n",
      "            C-terminal domain in a GTP-dependent manner\n",
      "  JOURNAL   J Biol Chem 286 (12), 10568-10580 (2011)\n",
      "   PUBMED   21233210\n",
      "  REMARK    GeneRIF: ARF-like protein 16 (ARL16) inhibits RIG-I by binding with\n",
      "            its C-terminal domain in a GTP-dependent manner\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from BC105078.1, BG427716.1,\n",
      "            BE746724.1 and AI149177.1.\n",
      "            On Aug 24, 2020 this sequence version replaced NR_138058.1.\n",
      "            \n",
      "            Summary: The protein encoded by this gene belongs to the ARL\n",
      "            (ADP-ribosylation factor-like) family of proteins, which are\n",
      "            structurally related to ADP-ribosylation factors (ARFs). This\n",
      "            protein has been shown to have an inhibitory role in the cellular\n",
      "            antiviral response. This gene product interacts with the C-terminal\n",
      "            domain of the DEXD/H-box helicase 58 (DDX58) gene product. This\n",
      "            interaction was found to suppress the association between the DDX58\n",
      "            gene product and RNA, thereby negatively regulating the activity of\n",
      "            the DDX58 gene product. [provided by RefSeq, Jul 2016].\n",
      "            \n",
      "            Transcript Variant: This variant (4) contains an alternate internal\n",
      "            exon, compared to variant 1. This variant is represented as\n",
      "            non-coding because the use of the 5'-most expected translational\n",
      "            start codon, as used in variant 1, renders the transcript a\n",
      "            candidate for nonsense-mediated mRNA decay (NMD).\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: BG427716.1, SRR3476690.50518.1\n",
      "                                           [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-67                BC105078.1         128-194\n",
      "            68-672              BG427716.1         2-606\n",
      "            673-1240            BE746724.1         43-610\n",
      "            1241-1272           AI149177.1         40-71               c\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..1272\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"transcribed RNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"17\"\n",
      "                     /map=\"17q25.3\"\n",
      "     gene            1..1272\n",
      "                     /gene=\"ARL16\"\n",
      "                     /note=\"ADP ribosylation factor like GTPase 16\"\n",
      "                     /db_xref=\"GeneID:339231\"\n",
      "                     /db_xref=\"HGNC:HGNC:27902\"\n",
      "                     /db_xref=\"MIM:619117\"\n",
      "     misc_RNA        1..1272\n",
      "                     /gene=\"ARL16\"\n",
      "                     /product=\"ADP ribosylation factor like GTPase 16,\n",
      "                     transcript variant 4\"\n",
      "                     /db_xref=\"GeneID:339231\"\n",
      "                     /db_xref=\"HGNC:HGNC:27902\"\n",
      "                     /db_xref=\"MIM:619117\"\n",
      "     exon            1..105\n",
      "                     /gene=\"ARL16\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            106..164\n",
      "                     /gene=\"ARL16\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            165..278\n",
      "                     /gene=\"ARL16\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            279..463\n",
      "                     /gene=\"ARL16\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            464..579\n",
      "                     /gene=\"ARL16\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            580..1272\n",
      "                     /gene=\"ARL16\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "ORIGIN      \n",
      "        1 gccgcggcgc cgagctacgg gtgccgggtg gagcgaagca cggaatgtgt ctcctgctgg\n",
      "       61 gggccacggg cgtcgggaag acgctgctgg tgaaacggct gcaggaggtg agctcccggg\n",
      "      121 atgggaaagg cgacctgggg gagccgcccc cgacacggcc cacggtgggc accaatctta\n",
      "      181 ctgacatcgt ggcacagaga aagatcacca tccgggagct tggggggtgc atgggcccca\n",
      "      241 tctggtccag ttactatgga aactgccgtt ctctcctggc ctgtgccgcg gataccagta\n",
      "      301 tggagaactg atggccacat gaggacagag acgagtgaca ccaggtcgtc ttcataaaac\n",
      "      361 tcgattccct ctgcttctgc ttgttcccaa agctttgttt aaagctactt tgggccgggc\n",
      "      421 gcggtggctt actcctgtaa tcccagcact ttgggaagct gagtttgtga tggacgcctc\n",
      "      481 tgaccccacc cagctctctg catcctgtgt gcagctctta ggtctccttt ctgcagaaca\n",
      "      541 acttgcagaa gcatcggtgc tgatactctt caataaaatc gacctaccct gttacatgtc\n",
      "      601 cacggaggag atgaagtcat taatcaggct tccagacatc attgcttgtg ccaagcagaa\n",
      "      661 catcaccacg gcagaaatca gcgcccgtga aggcactggc ttagcagggg tgctggcctg\n",
      "      721 gctccaggcc acccacagag ccaacgattg actgcacggc agaggcgcag ctggcctgag\n",
      "      781 ctggggagag gtggcagagg gcagtatggc tttgctgcca atagtttctt ctcacagggg\n",
      "      841 cagaataacc caaagtaacc ctacatgatg gggctctgtg ctgagatgca atgatgtgta\n",
      "      901 aactgaggca tgtggagatg gaagttgaca tctggcctct gaaaaaagtg tccccagggg\n",
      "      961 ctaggcatgg tggctcacac ctgtaatccc agcactttga gaggccgagg cgggtgtatc\n",
      "     1021 acctgaggtc gggagttcga gactagcctg accaacatgg agaaaccctg tctctactaa\n",
      "     1081 aaatacaaaa ttagctgggt gtgctggtgc atgcctgtaa tctcagctac ttgggaggct\n",
      "     1141 gagacaggag aatcccttga acctgggagg tggaggttgc agtgagtcga gatcatgcca\n",
      "     1201 ttgcactgca cctgggcaac aagagtgaaa ctccgtctta aaaaatataa gaaataaaaa\n",
      "     1261 aataaaaacc tg\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       NM_001324459             847 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens myosin light chain, phosphorylatable, fast skeletal\n",
      "            muscle (MYLPF), transcript variant 3, mRNA.\n",
      "ACCESSION   NM_001324459\n",
      "VERSION     NM_001324459.2\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 847)\n",
      "  AUTHORS   Haenig C, Atias N, Taylor AK, Mazza A, Schaefer MH, Russ J,\n",
      "            Riechers SP, Jain S, Coughlin M, Fontaine JF, Freibaum BD,\n",
      "            Brusendorf L, Zenkner M, Porras P, Stroedicke M, Schnoegl S,\n",
      "            Arnsburg K, Boeddrich A, Pigazzini L, Heutink P, Taylor JP,\n",
      "            Kirstein J, Andrade-Navarro MA, Sharan R and Wanker EE.\n",
      "  TITLE     Interactome Mapping Provides a Network of Neurodegenerative Disease\n",
      "            Proteins and Uncovers Widespread Protein Aggregation in Affected\n",
      "            Brains\n",
      "  JOURNAL   Cell Rep 32 (7), 108050 (2020)\n",
      "   PUBMED   32814053\n",
      "REFERENCE   2  (bases 1 to 847)\n",
      "  AUTHORS   Chong JX, Talbot JC, Teets EM, Previs S, Martin BL, Shively KM,\n",
      "            Marvin CT, Aylsworth AS, Saadeh-Haddad R, Schatz UA, Inzana F,\n",
      "            Ben-Omran T, Almusafri F, Al-Mulla M, Buckingham KJ, Harel T,\n",
      "            Mor-Shaked H, Radhakrishnan P, Girisha KM, Nayak SS, Shukla A,\n",
      "            Dieterich K, Faure J, Rendu J, Capri Y, Latypova X, Nickerson DA,\n",
      "            Warshaw DM, Janssen PML, Amacher SL and Bamshad MJ.\n",
      "  CONSRTM   University of Washington Center for Mendelian Genomics\n",
      "  TITLE     Mutations in MYLPF Cause a Novel Segmental Amyoplasia that\n",
      "            Manifests as Distal Arthrogryposis\n",
      "  JOURNAL   Am J Hum Genet 107 (2), 293-310 (2020)\n",
      "   PUBMED   32707087\n",
      "  REMARK    GeneRIF: Mutations in MYLPF Cause a Novel Segmental Amyoplasia that\n",
      "            Manifests as Distal Arthrogryposis.\n",
      "REFERENCE   3  (bases 1 to 847)\n",
      "  AUTHORS   Kondo T, Okada M, Kunihiro K, Takahashi M, Yaoita Y, Hosoya H and\n",
      "            Hamao K.\n",
      "  TITLE     Characterization of myosin II regulatory light chain isoforms in\n",
      "            HeLa cells\n",
      "  JOURNAL   Cytoskeleton (Hoboken) 72 (12), 609-620 (2015)\n",
      "   PUBMED   26663899\n",
      "REFERENCE   4  (bases 1 to 847)\n",
      "  AUTHORS   Sachdev S, Raychowdhury MK and Sarkar S.\n",
      "  TITLE     Human fast skeletal myosin light chain 2 cDNA: isolation, tissue\n",
      "            specific expression of the single copy gene, comparative sequence\n",
      "            analysis of isoforms and evolutionary relationships\n",
      "  JOURNAL   DNA Seq 14 (5), 339-350 (2003)\n",
      "   PUBMED   14756420\n",
      "  REMARK    GeneRIF: A cDNA clone encoding human fast skeletal myosin\n",
      "            regulatory light chain (HSRLC) has been isolated and characterized\n",
      "            from a fetal muscle cDNA library.\n",
      "REFERENCE   5  (bases 1 to 847)\n",
      "  AUTHORS   Nudel,U., Calvo,J.M., Shani,M. and Levy,Z.\n",
      "  TITLE     The nucleotide sequence of a rat myosin light chain 2 gene\n",
      "  JOURNAL   Nucleic Acids Res 12 (18), 7175-7186 (1984)\n",
      "   PUBMED   6091059\n",
      "REFERENCE   6  (bases 1 to 847)\n",
      "  AUTHORS   Garfinkel,L.I., Periasamy,M. and Nadal-Ginard,B.\n",
      "  TITLE     Cloning and characterization of cDNA sequences corresponding to\n",
      "            myosin light chains 1, 2, and 3, troponin-C, troponin-T,\n",
      "            alpha-tropomyosin, and alpha-actin\n",
      "  JOURNAL   J Biol Chem 257 (18), 11078-11086 (1982)\n",
      "   PUBMED   6179945\n",
      "COMMENT     VALIDATED REFSEQ: This record has undergone validation or\n",
      "            preliminary review. The reference sequence was derived from\n",
      "            BI828249.1, BC012571.1 and AA662706.1.\n",
      "            On Aug 14, 2020 this sequence version replaced NM_001324459.1.\n",
      "            \n",
      "            Transcript Variant: This variant (3) and variants 1 and 2 all\n",
      "            encode the same protein.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: BI828249.1 [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-41                BI828249.1         1-41\n",
      "            42-520              BI828249.1         43-521\n",
      "            521-734             BC012571.1         334-547\n",
      "            735-847             AA662706.1         29-141              c\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..847\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"16\"\n",
      "                     /map=\"16p11.2\"\n",
      "     gene            1..847\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"myosin light chain, phosphorylatable, fast skeletal\n",
      "                     muscle\"\n",
      "                     /db_xref=\"GeneID:29895\"\n",
      "                     /db_xref=\"HGNC:HGNC:29824\"\n",
      "                     /db_xref=\"MIM:617378\"\n",
      "     exon            1..71\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            72..210\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     misc_feature    199..201\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     exon            211..249\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     CDS             247..756\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"myosin light chain 2; fast skeletal myosin light\n",
      "                     chain 2; myosin, light chain 11, regulatory\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"myosin regulatory light chain 2, skeletal muscle\n",
      "                     isoform\"\n",
      "                     /protein_id=\"NP_001311388.1\"\n",
      "                     /db_xref=\"CCDS:CCDS10677.1\"\n",
      "                     /db_xref=\"GeneID:29895\"\n",
      "                     /db_xref=\"HGNC:HGNC:29824\"\n",
      "                     /db_xref=\"MIM:617378\"\n",
      "                     /translation=\"MAPKRAKRRTVEGGSSSVFSMFDQTQIQEFKEAFTVIDQNRDGI\n",
      "                     IDKEDLRDTFAAMGRLNVKNEELDAMMKEASGPINFTVFLTMFGEKLKGADPEDVITG\n",
      "                     AFKVLDPEGKGTIKKKFLEELLTTQCDRFSQEEIKNMWAAFPPDVGGNVDYKNICYVI\n",
      "                     THGDAKDQE\"\n",
      "     misc_feature    250..252\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"N,N,N-trimethylalanine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:P02608; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q96A32.1); methylation site\"\n",
      "     misc_feature    289..291\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"Phosphoserine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:P97457; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q96A32.1); phosphorylation site\"\n",
      "     misc_feature    292..294\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"Phosphoserine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:P97457; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q96A32.1); phosphorylation site\"\n",
      "     misc_feature    319..321\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"Phosphothreonine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:P04466; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q96A32.1); phosphorylation site\"\n",
      "     misc_feature    349..351\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"Phosphothreonine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:P04466; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q96A32.1); phosphorylation site\"\n",
      "     misc_feature    469..471\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"Phosphoserine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:P04466; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q96A32.1); phosphorylation site\"\n",
      "     misc_feature    547..549\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"Phosphothreonine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:P04466; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q96A32.1); phosphorylation site\"\n",
      "     exon            250..342\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            343..418\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            419..523\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            524..602\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            603..651\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            652..847\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      819..824\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      847\n",
      "                     /gene=\"MYLPF\"\n",
      "                     /gene_synonym=\"DA1C; HUMMLC2B; MLC2B; MRLC2; MYL11\"\n",
      "                     /note=\"major polyA site\"\n",
      "ORIGIN      \n",
      "        1 atttaggcgg tgacagacag catccgcagt caataggaag agctgaagaa ttctttaggt\n",
      "       61 ctcccaaaat ggcctttgct caggccatgc cctccgccca gcacatctgc tcctccacct\n",
      "      121 gcacctgact ccctggccct cgtgaagcca ctcccatact cctagacagg gctgctgcgc\n",
      "      181 cctccacttc tgggctcctg accagctctg ccggagccgc tgccttgccc cccggagact\n",
      "      241 gaagacatgg cacccaagag ggccaagaga aggacagtag agggcggaag ctccagcgtc\n",
      "      301 ttctccatgt tcgaccagac tcagatccag gagttcaaag aggccttcac tgtgatcgac\n",
      "      361 cagaaccgtg atggtattat agacaaggag gaccttcggg acaccttcgc agccatgggc\n",
      "      421 cgcctcaatg tgaagaatga ggagttggat gccatgatga aggaagccag cggtcccatc\n",
      "      481 aacttcaccg tcttcctgac catgttcggg gagaagctca agggtgccga ccctgaggat\n",
      "      541 gtgatcaccg gagccttcaa ggtcttggac cctgagggaa agggcaccat caagaagaag\n",
      "      601 ttcctggagg agctgctgac cacgcagtgt gaccgcttct cccaggagga gatcaagaac\n",
      "      661 atgtgggcgg ccttcccccc cgacgtgggc ggcaacgtcg actacaaaaa catctgctac\n",
      "      721 gtcatcacgc acggcgacgc caaggaccag gagtaggggc acccgcgggc ctccgctgcc\n",
      "      781 cgacgcttct gttcggcccg acctccaccc cggctcccaa taaaatttaa ctgatctttg\n",
      "      841 tttctta\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       NM_001165037            3169 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens gamma-aminobutyric acid type A receptor subunit alpha5\n",
      "            (GABRA5), transcript variant 2, mRNA.\n",
      "ACCESSION   NM_001165037\n",
      "VERSION     NM_001165037.2\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 3169)\n",
      "  AUTHORS   Hernandez CC, XiangWei W, Hu N, Shen D, Shen W, Lagrange AH, Zhang\n",
      "            Y, Dai L, Ding C, Sun Z, Hu J, Zhu H, Jiang Y and Macdonald RL.\n",
      "  TITLE     Altered inhibitory synapses in de novo GABRA5 and GABRA1 mutations\n",
      "            associated with early onset epileptic encephalopathies\n",
      "  JOURNAL   Brain 142 (7), 1938-1954 (2019)\n",
      "   PUBMED   31056671\n",
      "  REMARK    GeneRIF: study identified GABRA5 as a causative gene for early\n",
      "            onset epileptic encephalopathy and expands the mutant GABRA1\n",
      "            phenotypic spectrum, supporting growing evidence that defects in\n",
      "            GABAergic neurotransmission contribute to early onset epileptic\n",
      "            encephalopathy phenotypes.\n",
      "REFERENCE   2  (bases 1 to 3169)\n",
      "  AUTHORS   Horder J, Andersson M, Mendez MA, Singh N, Tangen A, Lundberg J,\n",
      "            Gee A, Halldin C, Veronese M, Bolte S, Farde L, Sementa T, Cash D,\n",
      "            Higgins K, Spain D, Turkheimer F, Mick I, Selvaraj S, Nutt DJ,\n",
      "            Lingford-Hughes A, Howes OD, Murphy DG and Borg J.\n",
      "  TITLE     GABAA receptor availability is not altered in adults with autism\n",
      "            spectrum disorder or in mouse models\n",
      "  JOURNAL   Sci Transl Med 10 (461) (2018)\n",
      "   PUBMED   30282698\n",
      "  REMARK    GeneRIF: both total GABAA and GABAA alpha5 receptor availability in\n",
      "            two positron emission tomography imaging studies, are reported.\n",
      "REFERENCE   3  (bases 1 to 3169)\n",
      "  AUTHORS   Liu S, Xu L, Guan F, Liu YT, Cui Y, Zhang Q, Zheng X, Bi GQ, Zhou\n",
      "            ZH, Zhang X and Ye S.\n",
      "  TITLE     Cryo-EM structure of the human alpha5beta3 GABAA receptor\n",
      "  JOURNAL   Cell Res 28 (9), 958-961 (2018)\n",
      "   PUBMED   30140029\n",
      "  REMARK    GeneRIF: cryo-EM structure of the human alpha5beta3 GABAA receptor\n",
      "REFERENCE   4  (bases 1 to 3169)\n",
      "  AUTHORS   Butler KM, Moody OA, Schuler E, Coryell J, Alexander JJ, Jenkins A\n",
      "            and Escayg A.\n",
      "  TITLE     De novo variants in GABRA2 and GABRA5 alter receptor function and\n",
      "            contribute to early-onset epilepsy\n",
      "  JOURNAL   Brain 141 (8), 2392-2405 (2018)\n",
      "   PUBMED   29961870\n",
      "  REMARK    GeneRIF: The GABRA5 p.V294L variant produced receptors that were\n",
      "            10-times more sensitive to GABA but had reduced maximal GABA-evoked\n",
      "            current due to increased receptor desensitization.\n",
      "REFERENCE   5  (bases 1 to 3169)\n",
      "  AUTHORS   Wielders CLC, van Nierop P, Vormer TL, Foijer F, Verheij J, Lodder\n",
      "            JC, Andersen JB, Mansvelder HD and Te Riele H.\n",
      "  TITLE     RNAi screening of subtracted transcriptomes reveals tumor\n",
      "            suppression by taurine-activated GABAA receptors involved in volume\n",
      "            regulation\n",
      "  JOURNAL   PLoS One 13 (5), e0196979 (2018)\n",
      "   PUBMED   29787571\n",
      "  REMARK    GeneRIF: Low GABRA5 expression typified hyperproliferative tumors,\n",
      "            and loss of taurine signaling correlated with reduced patient\n",
      "            survival, suggesting this tumor suppressive mechanism operates in\n",
      "            vivo.\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   6  (bases 1 to 3169)\n",
      "  AUTHORS   Glatt K, Glatt H and Lalande M.\n",
      "  TITLE     Structure and organization of GABRB3 and GABRA5\n",
      "  JOURNAL   Genomics 41 (1), 63-69 (1997)\n",
      "   PUBMED   9126483\n",
      "  REMARK    Erratum:[1997 Aug 15;44(1):155]\n",
      "REFERENCE   7  (bases 1 to 3169)\n",
      "  AUTHORS   Russek SJ and Farb DH.\n",
      "  TITLE     Mapping of the beta 2 subunit gene (GABRB2) to microdissected human\n",
      "            chromosome 5q34-q35 defines a gene cluster for the most abundant\n",
      "            GABAA receptor isoform\n",
      "  JOURNAL   Genomics 23 (3), 528-533 (1994)\n",
      "   PUBMED   7851879\n",
      "REFERENCE   8  (bases 1 to 3169)\n",
      "  AUTHORS   Knoll JH, Sinnett D, Wagstaff J, Glatt K, Wilcox AS, Whiting PM,\n",
      "            Wingrove P, Sikela JM and Lalande M.\n",
      "  TITLE     FISH ordering of reference markers and of the gene for the alpha 5\n",
      "            subunit of the gamma-aminobutyric acid receptor (GABRA5) within the\n",
      "            Angelman and Prader-Willi syndrome chromosomal regions\n",
      "  JOURNAL   Hum Mol Genet 2 (2), 183-189 (1993)\n",
      "   PUBMED   8388764\n",
      "REFERENCE   9  (bases 1 to 3169)\n",
      "  AUTHORS   Glatt KA, Sinnett D and Lalande M.\n",
      "  TITLE     Dinucleotide repeat polymorphism at the GABAA receptor alpha 5\n",
      "            (GABRA5) locus at chromosome 15q11-q13\n",
      "  JOURNAL   Hum Mol Genet 1 (5), 348 (1992)\n",
      "   PUBMED   1338907\n",
      "REFERENCE   10 (bases 1 to 3169)\n",
      "  AUTHORS   Wingrove P, Hadingham K, Wafford K, Kemp JA, Ragan CI and Whiting\n",
      "            P.\n",
      "  TITLE     Cloning and expression of a cDNA encoding the human GABA-A receptor\n",
      "            alpha 5 subunit\n",
      "  JOURNAL   Biochem Soc Trans 20 (1), 18S (1992)\n",
      "   PUBMED   1321750\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AC135999.4, DB506215.1,\n",
      "            BC011403.1, BC111979.1 and R49008.1.\n",
      "            On Aug 14, 2020 this sequence version replaced NM_001165037.1.\n",
      "            \n",
      "            Summary: GABA is the major inhibitory neurotransmitter in the\n",
      "            mammalian brain where it acts at GABA-A receptors, which are\n",
      "            ligand-gated chloride channels. Chloride conductance of these\n",
      "            channels can be modulated by agents such as benzodiazepines that\n",
      "            bind to the GABA-A receptor. At least 16 distinct subunits of\n",
      "            GABA-A receptors have been identified. Transcript variants\n",
      "            utilizing three different alternative non-coding first exons have\n",
      "            been described. [provided by RefSeq, Jul 2008].\n",
      "            \n",
      "            Transcript Variant: This variant (2) represents use of an alternate\n",
      "            promoter and 5' UTR, compared to variant 1. Both variants 1 and 2\n",
      "            encode the same protein.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: BC011403.1, SRR1803612.239518.1\n",
      "                                           [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-407               AC135999.4         7980-8386           c\n",
      "            408-658             DB506215.1         3-253\n",
      "            659-1410            BC011403.1         1-752\n",
      "            1411-1411           BC111979.1         750-750\n",
      "            1412-3163           BC011403.1         754-2505\n",
      "            3164-3169           R49008.1           15-20               c\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..3169\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"15\"\n",
      "                     /map=\"15q12\"\n",
      "     gene            1..3169\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"gamma-aminobutyric acid type A receptor subunit\n",
      "                     alpha5\"\n",
      "                     /db_xref=\"GeneID:2558\"\n",
      "                     /db_xref=\"HGNC:HGNC:4079\"\n",
      "                     /db_xref=\"MIM:137142\"\n",
      "     exon            1..666\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            667..731\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     misc_feature    686..688\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     exon            732..891\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     CDS             806..2194\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"gamma-aminobutyric acid (GABA) A receptor, alpha 5;\n",
      "                     GABA(A) receptor subunit alpha-5; gamma-aminobutyric acid\n",
      "                     type A receptor alpha5 subunit\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"gamma-aminobutyric acid receptor subunit alpha-5\n",
      "                     precursor\"\n",
      "                     /protein_id=\"NP_001158509.1\"\n",
      "                     /db_xref=\"CCDS:CCDS45194.1\"\n",
      "                     /db_xref=\"GeneID:2558\"\n",
      "                     /db_xref=\"HGNC:HGNC:4079\"\n",
      "                     /db_xref=\"MIM:137142\"\n",
      "                     /translation=\"MDNGMFSGFIMIKNLLLFCISMNLSSHFGFSQMPTSSVKDETND\n",
      "                     NITIFTRILDGLLDGYDNRLRPGLGERITQVRTDIYVTSFGPVSDTEMEYTIDVFFRQ\n",
      "                     SWKDERLRFKGPMQRLPLNNLLASKIWTPDTFFHNGKKSIAHNMTTPNKLLRLEDDGT\n",
      "                     LLYTMRLTISAECPMQLEDFPMDAHACPLKFGSYAYPNSEVVYVWTNGSTKSVVVAED\n",
      "                     GSRLNQYHLMGQTVGTENISTSTGEYTIMTAHFHLKRKIGYFVIQTYLPCIMTVILSQ\n",
      "                     VSFWLNRESVPARTVFGVTTVLTMTTLSISARNSLPKVAYATAMDWFIAVCYAFVFSA\n",
      "                     LIEFATVNYFTKRGWAWDGKKALEAAKIKKKREVILNKSTNAFTTGKMSHPPNIPKEQ\n",
      "                     TPAGTSNTTSVSVKPSEEKTSESKKTYNSISKIDKMSRIVFPVLFGTFNLVYWATYLN\n",
      "                     REPVIKGAASPK\"\n",
      "     sig_peptide     806..898\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"COORDINATES: ab initio prediction:SignalP:4.0\"\n",
      "     mat_peptide     899..2191\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /product=\"gamma-aminobutyric acid receptor subunit\n",
      "                     alpha-5\"\n",
      "     misc_feature    938..940\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P31644.1); glycosylation site\"\n",
      "     misc_feature    1238..1240\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P31644.1); glycosylation site\"\n",
      "     misc_feature    1424..1426\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P31644.1); glycosylation site\"\n",
      "     misc_feature    1511..1513\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (P31644.1); glycosylation site\"\n",
      "     misc_feature    1583..1648\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P31644.1);\n",
      "                     transmembrane region\"\n",
      "     misc_feature    1661..1726\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P31644.1);\n",
      "                     transmembrane region\"\n",
      "     misc_feature    1760..1828\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P31644.1);\n",
      "                     transmembrane region\"\n",
      "     misc_feature    2087..2152\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (P31644.1);\n",
      "                     transmembrane region\"\n",
      "     exon            892..1013\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1014..1081\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1082..1302\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1303..1385\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1386..1529\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1530..1682\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1683..1894\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1895..3169\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      3144..3149\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      3169\n",
      "                     /gene=\"GABRA5\"\n",
      "                     /gene_synonym=\"DEE79; EIEE79\"\n",
      "                     /note=\"major polyA site\"\n",
      "ORIGIN      \n",
      "        1 agaagcgcct aagagatgcg catgtaaggg gtgccatgga gcacccggga aggacccagg\n",
      "       61 cctgggcagg gaggcgttgg ggccggctct agggagcaaa gctgtcggag tccaggcctc\n",
      "      121 tatcacccga ctgctgacac tgccctccca tccctgctct acaagcgggc ggtggtgact\n",
      "      181 acacgaggcg ccgaagatgc tgttgagggc cctggagaaa cttcagcaga acagggcctc\n",
      "      241 tccccttgca ggccgagccg ggggccctgc gccctccccc tccgcccagc tcggccaagg\n",
      "      301 gcgcatttgc tgagcgtctg gcggcctcta ccggagcacc tctgcagagg gccgatcctc\n",
      "      361 cagcccagag acgacatgtg gcgctcgggc gaggtgagag cgggcgcgag tgcgccgggg\n",
      "      421 gcgctggggg gctctgcggc gggcggcgcg cggcccgggg cgcggcgcgg agcggagctg\n",
      "      481 cagggcggcg gcgggagcgc ggggcgcaag agccgctccg ccgggagtgc cggggaagtt\n",
      "      541 cgcgctggca gcatggggcg gtgacgccgc accggccttc cgcgcctgcc agccgggcga\n",
      "      601 gagcaggcgg aggagaagga ggatgcatcc tcaccgacgg ctcgcctccc ggggccggcg\n",
      "      661 cgcaggtgcc ttgcagagag aggagtagct tgctggcttt gaacgcgtgg cgtggcagat\n",
      "      721 atttcagaaa gcttcaagaa caagctggag aagggaagag ttattcctcc atattcacct\n",
      "      781 gcttcaacta ctattcttat tgggaatgga caatggaatg ttctctggtt ttatcatgat\n",
      "      841 caaaaacctc cttctctttt gtatttccat gaacttatcc agtcactttg gcttttcaca\n",
      "      901 gatgccaacc agttcagtga aagatgagac caatgacaac atcacgatat ttaccaggat\n",
      "      961 cttggatggg ctcttggatg gctacgacaa cagacttcgg cccgggctgg gagagcgcat\n",
      "     1021 cactcaggtg aggaccgaca tctacgtcac cagcttcggc ccggtgtccg acacggaaat\n",
      "     1081 ggagtacacc atagacgtgt ttttccgaca aagctggaaa gatgaaaggc ttcggtttaa\n",
      "     1141 ggggcccatg cagcgcctcc ctctcaacaa cctccttgcc agcaagatct ggaccccaga\n",
      "     1201 cacgttcttc cacaacggga agaagtccat cgctcacaac atgaccacgc ccaacaagct\n",
      "     1261 gctgcggctg gaggacgacg gcaccctgct ctacaccatg cgcttgacca tctctgcaga\n",
      "     1321 gtgccccatg cagcttgagg acttcccgat ggatgcgcac gcttgccctc tgaaatttgg\n",
      "     1381 cagctatgcg taccctaatt ctgaagtcgt ttacgtctgg accaacggct ccaccaagtc\n",
      "     1441 ggtggtggtg gcggaagatg gctccagact gaaccagtac cacctgatgg ggcagacggt\n",
      "     1501 gggcactgag aacatcagca ccagcacagg cgaatacaca atcatgacag ctcacttcca\n",
      "     1561 cctgaaaagg aagattggct actttgtcat ccagacctac cttccctgca taatgaccgt\n",
      "     1621 gatcttatca caggtgtcct tttggctgaa ccgggaatca gtcccagcca ggacagtttt\n",
      "     1681 tggggtcacc acggtgctga ccatgacgac cctcagcatc agcgccagga actctctgcc\n",
      "     1741 caaagtggcc tacgccaccg ccatggactg gttcatagcc gtgtgctatg ccttcgtctt\n",
      "     1801 ctcggcgctg atagagtttg ccacggtcaa ttactttacc aagagaggct gggcctggga\n",
      "     1861 tggcaaaaaa gccttggaag cagccaagat caagaaaaag cgtgaagtca tactaaataa\n",
      "     1921 gtcaacaaac gcttttacaa ctgggaagat gtctcacccc ccaaacattc cgaaggaaca\n",
      "     1981 gaccccagca gggacgtcga atacaacctc agtctcagta aaaccctctg aagagaagac\n",
      "     2041 ttctgaaagc aaaaagactt acaacagtat cagcaaaatt gacaaaatgt cccgaatcgt\n",
      "     2101 attcccagtc ttgttcggca ctttcaactt agtttactgg gcaacgtatt tgaataggga\n",
      "     2161 gccggtgata aaaggagccg cctctccaaa ataaccggcc acactcccaa actccaagac\n",
      "     2221 agccatactt ccagcgaaat ggtaccaagg agaggtcttg ctcacaggga ctctccatat\n",
      "     2281 gtgagcacta tctttcagga aatttttgca tgtttaataa tatgtacaaa taatattgcc\n",
      "     2341 ttgatgtttc tatatgtaac ttcagatgtt tccaagatgt cccattgata attcgagcaa\n",
      "     2401 acaactttct ggaaaaacag gatacgatga ctgacactca gatgcccagt atcatacgtt\n",
      "     2461 gatagtttac aaacaagata cgtatatttt taactgcttc aagtgttacc taacaatgtt\n",
      "     2521 ttttatactt caaatgtcat ttcatacaaa ttttcccagt gaataaatat tttaggaaac\n",
      "     2581 tctccatgat tattagttga ccaactatat tgcgagaaac agagatcata aagagcacgt\n",
      "     2641 tttccattat gaggaaactt ggacatttat gtacaaaatg aattgccttt gataattctt\n",
      "     2701 actgttctga aattaggaaa gtactgcatg atcttacacg aagaaataga ataggcaaac\n",
      "     2761 ttttatgcag gcagattaat aacagaaata catcatatgt tagatacaca aaatatttcc\n",
      "     2821 ctgaggaaaa aaaatcaact gcttaaattt tttgggaaga aaaagaaaca tttcttaccc\n",
      "     2881 cacaccaccc actacctgaa caatagccaa gaaaggaaag gagatgttaa aacaaacagt\n",
      "     2941 gttttgatga cattctcagc acttggccca tgtgatcgct gtctctgaaa cactgaccac\n",
      "     3001 ctctccctac agccagcgtg tagtgcttca gtggtgagaa attgtaattg agttattttc\n",
      "     3061 cattttattt ccttgtatgt atttcatgac tggacttact gctctgtcag cttttgtata\n",
      "     3121 tgaatcttaa atgttcatta aaaaataaaa aatgaactga tcttgtgga\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       NM_001265609            4677 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens trafficking kinesin protein 1 (TRAK1), transcript\n",
      "            variant 4, mRNA.\n",
      "ACCESSION   NM_001265609\n",
      "VERSION     NM_001265609.2\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 4677)\n",
      "  AUTHORS   Henrichs V, Grycova L, Barinka C, Nahacka Z, Neuzil J, Diez S,\n",
      "            Rohlena J, Braun M and Lansky Z.\n",
      "  TITLE     Mitochondria-adaptor TRAK1 promotes kinesin-1 driven transport in\n",
      "            crowded environments\n",
      "  JOURNAL   Nat Commun 11 (1), 3123 (2020)\n",
      "   PUBMED   32561740\n",
      "  REMARK    GeneRIF: Mitochondria-adaptor TRAK1 promotes kinesin-1 driven\n",
      "            transport in crowded environments.\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   2  (bases 1 to 4677)\n",
      "  AUTHORS   Lee CA, Chin LS and Li L.\n",
      "  TITLE     Hypertonia-linked protein Trak1 functions with mitofusins to\n",
      "            promote mitochondrial tethering and fusion\n",
      "  JOURNAL   Protein Cell 9 (8), 693-716 (2018)\n",
      "   PUBMED   28924745\n",
      "  REMARK    GeneRIF: The findings uncover a novel function of Trak1 as a\n",
      "            regulator of mitochondrial fusion and provide evidence linking\n",
      "            dysregulated mitochondrial dynamics to hypertonia pathogenesis.\n",
      "REFERENCE   3  (bases 1 to 4677)\n",
      "  AUTHORS   Onodera Y, Nam JM, Horikawa M, Shirato H and Sabe H.\n",
      "  TITLE     Arf6-driven cell invasion is intrinsically linked to TRAK1-mediated\n",
      "            mitochondrial anterograde trafficking to avoid oxidative\n",
      "            catastrophe\n",
      "  JOURNAL   Nat Commun 9 (1), 2682 (2018)\n",
      "   PUBMED   29992963\n",
      "  REMARK    GeneRIF: Data show that the ADP-ribosylation factor 6\n",
      "            (Arf6)-trafficking kinesin protein 1 (trak1) promote the\n",
      "            anterograde trafficking of mitochondria.\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   4  (bases 1 to 4677)\n",
      "  AUTHORS   Barel O, Malicdan MCV, Ben-Zeev B, Kandel J, Pri-Chen H, Stephen J,\n",
      "            Castro IG, Metz J, Atawa O, Moshkovitz S, Ganelin E, Barshack I,\n",
      "            Polak-Charcon S, Nass D, Marek-Yagel D, Amariglio N, Shalva N,\n",
      "            Vilboux T, Ferreira C, Pode-Shakked B, Heimer G, Hoffmann C,\n",
      "            Yardeni T, Nissenkorn A, Avivi C, Eyal E, Kol N, Glick Saar E,\n",
      "            Wallace DC, Gahl WA, Rechavi G, Schrader M, Eckmann DM and Anikster\n",
      "            Y.\n",
      "  TITLE     Deleterious variants in TRAK1 disrupt mitochondrial movement and\n",
      "            cause fatal encephalopathy\n",
      "  JOURNAL   Brain 140 (3), 568-581 (2017)\n",
      "   PUBMED   28364549\n",
      "  REMARK    GeneRIF: This study confirms the role of TRAK1 in mitochondrial\n",
      "            dynamics and constitutes the first report of this gene in\n",
      "            association with a severe neurodevelopmental disorder.\n",
      "            Erratum:[Brain. 2018 Apr 1;141(4):e34. PMID: 29528374]\n",
      "REFERENCE   5  (bases 1 to 4677)\n",
      "  AUTHORS   Hauri S, Comoglio F, Seimiya M, Gerstung M, Glatter T, Hansen K,\n",
      "            Aebersold R, Paro R, Gstaiger M and Beisel C.\n",
      "  TITLE     A High-Density Map for Navigating the Human Polycomb Complexome\n",
      "  JOURNAL   Cell Rep 17 (2), 583-595 (2016)\n",
      "   PUBMED   27705803\n",
      "REFERENCE   6  (bases 1 to 4677)\n",
      "  AUTHORS   Ozyildirim AM, Wistow GJ, Gao J, Wang J, Dickinson DP, Frierson HF\n",
      "            Jr and Laurie GW.\n",
      "  TITLE     The lacrimal gland transcriptome is an unusually rich source of\n",
      "            rare and poorly characterized gene transcripts\n",
      "  JOURNAL   Invest Ophthalmol Vis Sci 46 (5), 1572-1580 (2005)\n",
      "   PUBMED   15851553\n",
      "REFERENCE   7  (bases 1 to 4677)\n",
      "  AUTHORS   Brickley K, Smith MJ, Beck M and Stephenson FA.\n",
      "  TITLE     GRIF-1 and OIP106, members of a novel gene family of coiled-coil\n",
      "            domain proteins: association in vivo and in vitro with kinesin\n",
      "  JOURNAL   J Biol Chem 280 (15), 14723-14732 (2005)\n",
      "   PUBMED   15644324\n",
      "  REMARK    GeneRIF: GRIF-1 and OIP106 have roles in protein and/or organelle\n",
      "            transport in excitable cells in a manner analogous to glutamate\n",
      "            receptor-interacting-protein 1\n",
      "REFERENCE   8  (bases 1 to 4677)\n",
      "  AUTHORS   Colland F, Jacq X, Trouplin V, Mougin C, Groizeleau C, Hamburger A,\n",
      "            Meil A, Wojcik J, Legrain P and Gauthier JM.\n",
      "  TITLE     Functional proteomics mapping of a human signaling pathway\n",
      "  JOURNAL   Genome Res 14 (7), 1324-1332 (2004)\n",
      "   PUBMED   15231748\n",
      "REFERENCE   9  (bases 1 to 4677)\n",
      "  AUTHORS   Iyer SP and Hart GW.\n",
      "  TITLE     Roles of the tetratricopeptide repeat domain in O-GlcNAc\n",
      "            transferase targeting and protein substrate specificity\n",
      "  JOURNAL   J Biol Chem 278 (27), 24608-24616 (2003)\n",
      "   PUBMED   12724313\n",
      "REFERENCE   10 (bases 1 to 4677)\n",
      "  AUTHORS   Iyer SP, Akimoto Y and Hart GW.\n",
      "  TITLE     Identification and cloning of a novel family of coiled-coil domain\n",
      "            proteins that interact with O-GlcNAc transferase\n",
      "  JOURNAL   J Biol Chem 278 (7), 5399-5409 (2003)\n",
      "   PUBMED   12435728\n",
      "COMMENT     VALIDATED REFSEQ: This record has undergone validation or\n",
      "            preliminary review. The reference sequence was derived from\n",
      "            DA193280.1, AK294221.1, AC018358.20 and AA534633.1.\n",
      "            On Aug 14, 2020 this sequence version replaced NM_001265609.1.\n",
      "            \n",
      "            Transcript Variant: This variant (4) differs in the 5' and 3' UTRs\n",
      "            and initiates translation at an alternate start codon, compared to\n",
      "            variant 1. The encoded isoform (4) is shorter and has distinct N-\n",
      "            and C-termini, compared to isoform 1.\n",
      "            \n",
      "            Sequence Note: This RefSeq record was created from transcript and\n",
      "            genomic sequence data to make the sequence consistent with the\n",
      "            reference genome assembly. The genomic coordinates used for the\n",
      "            transcript record were based on transcript alignments.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: AK294221.1, SRR3476690.1071445.1\n",
      "                                           [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-138               DA193280.1         1-138\n",
      "            139-193             DA193280.1         140-194\n",
      "            194-1173            AK294221.1         160-1139\n",
      "            1174-1174           AC018358.20        74955-74955         c\n",
      "            1175-1827           AK294221.1         1141-1793\n",
      "            1828-4298           AC018358.20        57637-60107         c\n",
      "            4299-4677           AA534633.1         7-385               c\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..4677\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"3\"\n",
      "                     /map=\"3p22.1\"\n",
      "     gene            1..4677\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /note=\"trafficking kinesin protein 1\"\n",
      "                     /db_xref=\"GeneID:22906\"\n",
      "                     /db_xref=\"HGNC:HGNC:29947\"\n",
      "                     /db_xref=\"MIM:608112\"\n",
      "     exon            1..365\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     misc_feature    176..178\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     CDS             302..2314\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /note=\"isoform 4 is encoded by transcript variant 4;\n",
      "                     OGT(O-Glc-NAc transferase)-interacting protein 106 KDa;\n",
      "                     trafficking protein, kinesin binding 1; 106 kDa O-GlcNAc\n",
      "                     transferase-interacting protein; milton homolog 1;\n",
      "                     O-linked N-acetylglucosamine transferase interacting\n",
      "                     protein 106; protein Milton\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"trafficking kinesin-binding protein 1 isoform 4\"\n",
      "                     /protein_id=\"NP_001252538.1\"\n",
      "                     /db_xref=\"CCDS:CCDS74922.1\"\n",
      "                     /db_xref=\"GeneID:22906\"\n",
      "                     /db_xref=\"HGNC:HGNC:29947\"\n",
      "                     /db_xref=\"MIM:608112\"\n",
      "                     /translation=\"MQKFIEADYYELDWYYEECSDVLCAERVGQMTKTYNDIDAVTRL\n",
      "                     LEEKERDLELAARIGQSLLKKNKTLTERNELLEEQVEHIREEVSQLRHELSMKDELLQ\n",
      "                     FYTSAAEESEPESVCSTPLKRNESSSSVQNYFHLDSLQKKLKDLEEENVVLRSEASQL\n",
      "                     KTETITYEEKEQQLVNDCVKELRDANVQIASISEELAKKTEDAARQQEEITHLLSQIV\n",
      "                     DLQKKAKACAVENEELVQHLGAAKDAQRQLTAELRELEDKYAECMEMLHEAQEELKNL\n",
      "                     RNKTMPNTTSRRYHSLGLFPMDSLAAEIEGTMRKELQLEEAESPDITHQKRVFETVRN\n",
      "                     INQVVKQRSLTPSPMNIPGSNQSSAMNSLLSSCVSTPRSSFYGSDIGNVVLDNKTNSI\n",
      "                     ILETEAADLGNDERSKKPGTPGTPGSHDLETALRRLSLRRENYLSERRFFEEEQERKL\n",
      "                     QELAEKGELRSGSLTPTESIMSLGTHSRFSEFTGFSGMSFSSRSYLPEKLQIVKPLEG\n",
      "                     SATLHHWQQLAQPHLGGILDPRPGVVTKGFRTLDVDLDEVYCLNDFEEDDTGDHISLP\n",
      "                     RLATSTPVQHPETSGERSQARVTVSGSRSYPSRPQASPEEMQEPPAATEEEEEEEEEE\n",
      "                     GSGEGTTISPVNLAPFPEAEFWAILTSVPGTIRSGSLSVASARLCG\"\n",
      "     exon            366..442\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            443..559\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            560..660\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            661..769\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            770..848\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            849..979\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            980..1054\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1055..1192\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1193..1269\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1270..1506\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1507..1823\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1824..4677\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      2317..2322\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /note=\"hexamer: ATTAAA\"\n",
      "     polyA_site      2352\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /note=\"major polyA site\"\n",
      "     regulatory      4656..4661\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /note=\"hexamer: ATTAAA\"\n",
      "     polyA_site      4677\n",
      "                     /gene=\"TRAK1\"\n",
      "                     /gene_synonym=\"DEE68; EIEE68; MILT1; OIP106\"\n",
      "                     /note=\"major polyA site\"\n",
      "ORIGIN      \n",
      "        1 attcagggct gctgtccagt gctgctttat tggcagtgct gccagggtct ccgttagctc\n",
      "       61 tctgcaaatt gccttccttt ctgctcctcc tactccctcc ttcccccata gaatttttct\n",
      "      121 tttcattgcc cactttactg ttttggctcc agactgtcgt taagaatgta cagcctaatt\n",
      "      181 ctggtgtgtt tcgggatatt cttctgtcca gtattctgga agggcgggga ggcatggcag\n",
      "      241 cgttttactt gacgttgatg gtgctgtgaa gtccattctt tcctctgcaa gactactgac\n",
      "      301 tatgcagaaa tttatcgaag cggattatta tgaactagac tggtattatg aagaatgctc\n",
      "      361 ggatgtttta tgtgctgaaa gagttggcca gatgactaag acatataatg acatagatgc\n",
      "      421 tgtcactcgg cttcttgagg agaaagagcg ggatttagaa ttggccgctc gcatcggcca\n",
      "      481 gtcgttgttg aagaagaaca agaccctaac cgagaggaac gagctgctgg aggagcaggt\n",
      "      541 ggaacacatc agggaggagg tgtctcagct ccggcatgag ctgtccatga aggatgagct\n",
      "      601 gcttcagttc tacaccagcg ctgcggagga gagtgagccc gagtccgttt gctcaacccc\n",
      "      661 gttgaagagg aatgagtcgt cctcctcagt ccagaattac tttcatttgg attctcttca\n",
      "      721 aaagaagctg aaagaccttg aagaggagaa tgttgtactt cgatccgagg ccagccagct\n",
      "      781 gaagacagag accatcacct atgaggagaa ggagcagcag ctggtcaatg actgcgtgaa\n",
      "      841 ggagctgagg gatgccaatg tccagattgc tagtatctca gaggaactgg ccaagaagac\n",
      "      901 ggaagatgct gcccgccagc aagaggagat cacacacctg ctatcgcaaa tagttgattt\n",
      "      961 gcagaaaaag gcaaaagctt gcgcagtgga aaatgaagaa cttgtccagc atctgggggc\n",
      "     1021 tgctaaggat gcccagcggc agctcacagc cgagctgcgt gagctggagg acaagtacgc\n",
      "     1081 agagtgcatg gagatgctgc atgaggcgca ggaggagctg aagaacctcc ggaacaaaac\n",
      "     1141 catgcccaat accacgtctc ggcgctacca ctcactgggc ctgtttccca tggattcctt\n",
      "     1201 ggcagcagag attgagggaa cgatgcgcaa ggagctgcag ttggaagagg ccgagtctcc\n",
      "     1261 agacatcact caccagaagc gtgtctttga gacagtaaga aacatcaacc aggttgtcaa\n",
      "     1321 gcagagatct ctgacccctt ctcccatgaa catccccggc tccaaccagt cctcggccat\n",
      "     1381 gaactccctc ctgtccagct gcgtcagcac cccccggtcc agcttctacg gcagcgacat\n",
      "     1441 aggcaacgtc gtcctcgaca acaagaccaa cagcatcatt ctggaaacag aggcagccga\n",
      "     1501 cctgggaaac gatgagcgga gtaagaagcc ggggacgccg ggcaccccag gctcccacga\n",
      "     1561 cctggagacg gcgctgaggc ggctgtccct gcgccgggag aactacctct cggagaggag\n",
      "     1621 gttctttgag gaggagcaag agaggaagct ccaggagctg gcggagaagg gcgagctgcg\n",
      "     1681 cagcggctcc ctcacaccca ctgagagcat catgtccctg ggcacgcact cccgcttctc\n",
      "     1741 cgagttcacc ggcttctctg gcatgtcctt cagcagccgc tcctacctgc ctgagaagct\n",
      "     1801 ccagatcgtg aagccgctgg aaggttccgc cacacttcac cactggcagc agttggccca\n",
      "     1861 acctcacctt gggggcatcc tggacccccg gcccggtgtg gtcaccaagg gcttccggac\n",
      "     1921 gctggatgtt gacctggacg aagtgtactg ccttaacgac tttgaagaag atgacacagg\n",
      "     1981 tgaccacatt tctctcccac gcctagctac ctccactcca gttcagcacc cagagacctc\n",
      "     2041 aggtgagagg tcccaagcac gtgtgactgt ctcaggcagc agaagttacc cgagccggcc\n",
      "     2101 tcaggcttcc ccagaggaga tgcaggagcc gccagcggcc acggaggagg aggaggagga\n",
      "     2161 ggaggaggag gaggggtctg gtgagggcac cacgataagt cctgtaaact tggcaccttt\n",
      "     2221 cccggaggca gagttttggg ccattctcac ctctgttcca ggcaccatcc gtagtggttc\n",
      "     2281 tctgtctgta gcttccgctc gtctgtgtgg gtgatgatta aagcattctc attgcacagt\n",
      "     2341 tctgttttta aatacagagt ctgatgcctc ctatttgtaa caatgggtgt agctcccctg\n",
      "     2401 cccatcttgg aggtgcatgg cccatcaggg atctttaaag tgggagcagg aaaggctgct\n",
      "     2461 aaaaaaaaaa aaaaaaaaaa aagtgggctt ttgggtccct gaaaacatca gtgcccttct\n",
      "     2521 tcctggtctg ggtgtctccc tgagtctaag gggaagattc tcaagtcccc tggtgatttc\n",
      "     2581 caagtggagc tgagcagttt tagggaaatt gagtgctggg tcattcagaa ggtaaatgag\n",
      "     2641 atcatctgtt acctgtacgc tgtattaaaa tagaaccagg aaaggctcag gatttcagac\n",
      "     2701 atttcatcag ccttttcact ttcccagctt caatggaggt atatatgtca ttttcttttc\n",
      "     2761 agcttacaca tgtgttcaaa gtggattttt aaaaagtgtt ttagcaatac tccttaacca\n",
      "     2821 aataaacctt cggagaacgt cactaagctt ttccagagag aagaccctag atgaagttgg\n",
      "     2881 aaaagagctc tggctgactc cacccactgt ccccaagcat taaaggtgtg gccatgagtt\n",
      "     2941 tacagaatta cccacattcc agttgccact gggatgaaaa gctttgtgct cagagctctg\n",
      "     3001 ggaatcatgg gatcacatgg ttactgttta ccccaacagc atgctctgtc tagactggac\n",
      "     3061 ctcccagccc cttgtgttgg gaaggcaaag cttttgtgga gtcagggggg agactgagaa\n",
      "     3121 aaaatgaatt aaccctgtgt tgtcatcctc atgacattcc tgaggattca ccaggttaga\n",
      "     3181 agtgaggacg tttatctttg tgattcataa ttttcatttg tgaaggccac aacactcccc\n",
      "     3241 ttgaaataca gggcaggaaa gagctgaggt tcttggtggt gtctcccatt cccctggcta\n",
      "     3301 atttcagaca gctgtggtca agggatttta cttgggatca acttttcctt ttttccttga\n",
      "     3361 cattaatttt agagaagtca tcaagtcatg tgatttgttt agcacatagg tttatttatg\n",
      "     3421 gtttgatttt ttttaggcag ttatattact agattaagct tgtgagggaa tgaaaatgtt\n",
      "     3481 ttttatttgt tatctacaca ctcgaaaaag agaaaccagc tgcggtactg tcccattttt\n",
      "     3541 gtcatcagca ccagtgtccg tcaggaaggc aggtggtggt gcagaaacat gatgcctggc\n",
      "     3601 tgattttcgt ggctaaaggg gtaggcctta tgttgattgg gatgctcccc tacagcctta\n",
      "     3661 caggtagaat agaaggtgag ttctggaagt gcaaagaagc accattaagt gcatcttcta\n",
      "     3721 gaagttgtac agaaggacta aagcttatca agtcaacaaa gaacttacct tggaggatag\n",
      "     3781 agagagagaa gggaccaatt atgaggactg acatcctggc cactctcctt aaaataaaca\n",
      "     3841 ctgacatttt tcttgcttgt tctctctgta ctcaaacctg tggcaaattc atcctagcaa\n",
      "     3901 cgttatttga cgaggggcat gaacatttat agttgaaact gtagaaaggg tcaggttgga\n",
      "     3961 ggtgtgtaat aaaaaagaat tacctaggtt gccaaaggta atttaggaag ggtctgatca\n",
      "     4021 tttagatgag agttctttgg ggcttatttt ctgggtaagg ctcatcttta aaaactggct\n",
      "     4081 tcagagggga gaggggagaa caatgaattg gctctatttt ctctattggg aattacagga\n",
      "     4141 ccattttgat tcttagaatg taaaaagcat atcgctaagt aaatcatcct ggaggtccca\n",
      "     4201 agtagctcta tgcctgcaat catggagaca caggcagaca gataagcttc atggggaagg\n",
      "     4261 catggggcat cctctgtctt gggatttgta tccatggtgg tctggtccct gccttttaat\n",
      "     4321 ccgtcctcta cgcttgggct tttctgttac caaacagcac tatcccagga actattgtct\n",
      "     4381 gcctgggaac actcagtagg gagacacttt ggagacagga ggtgatgaac ctttttatgt\n",
      "     4441 gcagctggta tgatagaagg aaattgggaa aacttgtatg ctaggcactt ttgtccagag\n",
      "     4501 cctgctgtcc catggagaaa aagttttaag cactgaaaaa atttgattaa tgtatttaaa\n",
      "     4561 tgtattattt gaagcatcat tcacttgttg atttttacaa tcccatgtct taaaaaggat\n",
      "     4621 gaatccatgt tattgtattg taaataattt agatgattaa aatggattgt ttaaaaa\n",
      "//\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "LOCUS       NM_001286378            1701 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens maelstrom spermatogenic transposon silencer (MAEL),\n",
      "            transcript variant 3, mRNA.\n",
      "ACCESSION   NM_001286378\n",
      "VERSION     NM_001286378.2\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 1701)\n",
      "  AUTHORS   Taei A, Kiani T, Taghizadeh Z, Moradi S, Samadian A, Mollamohammadi\n",
      "            S, Sharifi-Zarchi A, Guenther S, Akhlaghpour A, Asgari Abibeiglou\n",
      "            B, Najar-Asl M, Karamzadeh R, Khalooghi K, Braun T, Hassani SN and\n",
      "            Baharvand H.\n",
      "  TITLE     Temporal activation of LRH-1 and RAR-gamma in human pluripotent\n",
      "            stem cells induces a functional naive-like state\n",
      "  JOURNAL   EMBO Rep 21 (10), e47533 (2020)\n",
      "   PUBMED   33252195\n",
      "REFERENCE   2  (bases 1 to 1701)\n",
      "  AUTHORS   Guo W, Yu Q, Zhang M, Li F, Liu Y, Jiang W, Jiang H and Li H.\n",
      "  TITLE     Long intergenic non-protein coding RNA 511 promotes the progression\n",
      "            of osteosarcoma cells through sponging microRNA 618 to upregulate\n",
      "            the expression of maelstrom\n",
      "  JOURNAL   Aging (Albany NY) 11 (15), 5351-5367 (2019)\n",
      "   PUBMED   31386627\n",
      "  REMARK    GeneRIF: LINC00511 was confirmed to be beneficial for osteosarcoma\n",
      "            development via sponging miR-618 and increasing MAEL expression\n",
      "REFERENCE   3  (bases 1 to 1701)\n",
      "  AUTHORS   Li P, Chen X, Qin G, Yue D, Zhang Z, Ping Y, Wang D, Zhao X, Song\n",
      "            M, Zhao Q, Li J, Liu S, Wang D, Zhang C, Lian J, Cao L, Li F, Huang\n",
      "            L, Wang L, Yang L, Huang J, Li H, Zhang B and Zhang Y.\n",
      "  TITLE     Maelstrom Directs Myeloid-Derived Suppressor Cells to Promote\n",
      "            Esophageal Squamous Cell Carcinoma Progression via Activation of\n",
      "            the Akt1/RelA/IL8 Signaling Pathway\n",
      "  JOURNAL   Cancer Immunol Res 6 (10), 1246-1259 (2018)\n",
      "   PUBMED   30082413\n",
      "  REMARK    GeneRIF: This study revealed a mechanism by which MAEL could\n",
      "            upregulate IL8 through Akt1/RelA to direct myeloid-derived\n",
      "            suppressor cells homing into the tumor, suggesting that MAEL could\n",
      "            be an attractive therapeutic target and a prognostic marker against\n",
      "            esophageal squamous cell carcinoma.\n",
      "REFERENCE   4  (bases 1 to 1701)\n",
      "  AUTHORS   Cheng YS, Wee SK, Lin TY and Lin YM.\n",
      "  TITLE     MAEL promoter hypermethylation is associated with de-repression of\n",
      "            LINE-1 in human hypospermatogenesis\n",
      "  JOURNAL   Hum Reprod 32 (12), 2373-2381 (2017)\n",
      "   PUBMED   29095993\n",
      "  REMARK    GeneRIF: MAEL promoter hypermethylation may have a role in\n",
      "            de-repression of LINE-1 in human hypospermatogenesis\n",
      "REFERENCE   5  (bases 1 to 1701)\n",
      "  AUTHORS   Cheng YS, Lu CW, Lin TY, Lin PY and Lin YM.\n",
      "  TITLE     Causes and Clinical Features of Infertile Men With Nonobstructive\n",
      "            Azoospermia and Histopathologic Diagnosis of Hypospermatogenesis\n",
      "  JOURNAL   Urology 105, 62-68 (2017)\n",
      "   PUBMED   28342926\n",
      "  REMARK    GeneRIF: Four sterility-related genes, including BOLL, DDX4,\n",
      "            HORMAD1, and MAEL, were found to have increased methylation at CpGs\n",
      "            of the promoter regions and decreased mRNA expressions in\n",
      "            nonobstructive azoospermia and hypospermatogenesis (HS) testis and\n",
      "            are believed to be associated with HS.\n",
      "REFERENCE   6  (bases 1 to 1701)\n",
      "  AUTHORS   Xiao L, Wang Y, Zhou Y, Sun Y, Sun W, Wang L, Zhou C, Zhou J and\n",
      "            Zhang J.\n",
      "  TITLE     Identification of a novel human cancer/testis gene MAEL that is\n",
      "            regulated by DNA methylation\n",
      "  JOURNAL   Mol Biol Rep 37 (5), 2355-2360 (2010)\n",
      "   PUBMED   19693694\n",
      "  REMARK    GeneRIF: These results suggest that MAEL is a novel\n",
      "            cancer/testis-associated gene and is regulated by DNA methylation.\n",
      "REFERENCE   7  (bases 1 to 1701)\n",
      "  AUTHORS   Ehret GB, O'Connor AA, Weder A, Cooper RS and Chakravarti A.\n",
      "  TITLE     Follow-up of a major linkage peak on chromosome 1 reveals\n",
      "            suggestive QTLs associated with essential hypertension: GenNet\n",
      "            study\n",
      "  JOURNAL   Eur J Hum Genet 17 (12), 1650-1657 (2009)\n",
      "   PUBMED   19536175\n",
      "  REMARK    GeneRIF: Observational study of gene-disease association. (HuGE\n",
      "            Navigator)\n",
      "REFERENCE   8  (bases 1 to 1701)\n",
      "  AUTHORS   Vaquerizas JM, Kummerfeld SK, Teichmann SA and Luscombe NM.\n",
      "  TITLE     A census of human transcription factors: function, expression and\n",
      "            evolution\n",
      "  JOURNAL   Nat Rev Genet 10 (4), 252-263 (2009)\n",
      "   PUBMED   19274049\n",
      "  REMARK    Review article\n",
      "REFERENCE   9  (bases 1 to 1701)\n",
      "  AUTHORS   Cheung CL, Chan BY, Chan V, Ikegawa S, Kou I, Ngai H, Smith D, Luk\n",
      "            KD, Huang QY, Mori S, Sham PC and Kung AW.\n",
      "  TITLE     Pre-B-cell leukemia homeobox 1 (PBX1) shows functional and possible\n",
      "            genetic association with bone mineral density variation\n",
      "  JOURNAL   Hum Mol Genet 18 (4), 679-687 (2009)\n",
      "   PUBMED   19064610\n",
      "  REMARK    GeneRIF: Observational study of gene-disease association. (HuGE\n",
      "            Navigator)\n",
      "REFERENCE   10 (bases 1 to 1701)\n",
      "  AUTHORS   Soper SF, van der Heijden GW, Hardiman TC, Goodheart M, Martin SL,\n",
      "            de Boer P and Bortvin A.\n",
      "  TITLE     Mouse maelstrom, a component of nuage, is essential for\n",
      "            spermatogenesis and transposon repression in meiosis\n",
      "  JOURNAL   Dev Cell 15 (2), 285-297 (2008)\n",
      "   PUBMED   18694567\n",
      "COMMENT     VALIDATED REFSEQ: This record has undergone validation or\n",
      "            preliminary review. The reference sequence was derived from\n",
      "            GU383114.1 and DQ076156.2.\n",
      "            On Aug 14, 2020 this sequence version replaced NM_001286378.1.\n",
      "            \n",
      "            Transcript Variant: This variant (3) differs in the 5' UTR and\n",
      "            lacks a portion of the 5' coding region compared to variant 1.\n",
      "            These  differences cause translation initiation at a downstream\n",
      "            start codon compared to variant 1. The encoded isoform (3) is\n",
      "            shorter than isoform 1.\n",
      "            \n",
      "            Sequence Note: This RefSeq record was created from transcript and\n",
      "            genomic sequence data to make the sequence consistent with the\n",
      "            reference genome assembly. The genomic coordinates used for the\n",
      "            transcript record were based on transcript alignments.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: GU383114.1, SRR1803613.365999.1\n",
      "                                           [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-1700              GU383114.1         1-1700\n",
      "            1701-1701           DQ076156.2         1737-1737\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..1701\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"1\"\n",
      "                     /map=\"1q24.1\"\n",
      "     gene            1..1701\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /note=\"maelstrom spermatogenic transposon silencer\"\n",
      "                     /db_xref=\"GeneID:84944\"\n",
      "                     /db_xref=\"HGNC:HGNC:25929\"\n",
      "                     /db_xref=\"MIM:611368\"\n",
      "     exon            1..85\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            86..169\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            170..262\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     CDS             206..1342\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /note=\"isoform 3 is encoded by transcript variant 3;\n",
      "                     maelstrom homolog; protein maelstrom homolog;\n",
      "                     cancer/testis antigen 128; spermatogenesis associated 35;\n",
      "                     testicular tissue protein Li 116\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"protein maelstrom homolog isoform 3\"\n",
      "                     /protein_id=\"NP_001273307.1\"\n",
      "                     /db_xref=\"CCDS:CCDS72975.1\"\n",
      "                     /db_xref=\"GeneID:84944\"\n",
      "                     /db_xref=\"HGNC:HGNC:25929\"\n",
      "                     /db_xref=\"MIM:611368\"\n",
      "                     /translation=\"MAREWRAAQGKDPGPSEKQKPVFTPLRRPGMLVPKQNVSPPDMS\n",
      "                     ALSLKGDQALLGGIFYFLNIFSHGELPPHCEQRFLPCEIGCVKYSLQEGIMADFHSFI\n",
      "                     NPGEIPRGFRFHCQAASDSSHKIPISNFERGHNQATVLQNLYRFIHPNPGNWPPIYCK\n",
      "                     SDDRTRVNWCLKHMAKASEIRQDLQLLTVEDLVVGIYQQKFLKEPSKTWIRSLLDVAM\n",
      "                     WDYSSNTRCKWHEENDILFCALAVCKKIAYCISNSLATLFGIQLTEAHVPLQDYEASN\n",
      "                     SVTPKMVVLDAGRYQKLRVGSSGFSHFNSSNEEQRSNTPIGDYPSRAKISGQNSSVRG\n",
      "                     RGITRLLESISNSSSNIHKFSNCDTSLSPYMSQKDGYKSFSSLS\"\n",
      "     exon            263..362\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            363..518\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            519..560\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            561..685\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            686..740\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            741..882\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            883..945\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            946..1078\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1079..1154\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1155..1701\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      1573..1578\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /note=\"hexamer: ATTAAA\"\n",
      "     polyA_site      1602\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /note=\"major polyA site\"\n",
      "     regulatory      1663..1668\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      1701\n",
      "                     /gene=\"MAEL\"\n",
      "                     /gene_synonym=\"CT128; SPATA35\"\n",
      "                     /note=\"major polyA site\"\n",
      "ORIGIN      \n",
      "        1 accgcctccc gcgcctccgc cccgccgccc gctgccgcga ctgccaaagt ttctcggtca\n",
      "       61 cgtgctggcc cccggcggcc caaaggagaa gatccccgaa ctacggcgac gaggcctgcc\n",
      "      121 tgtggctcgc gttgctgatg ccatccctta ctgctcctca gactgggcgc ttctgaggga\n",
      "      181 ggaagaaaag gagaaatacg cagaaatggc tcgagaatgg agggccgctc agggaaagga\n",
      "      241 ccctgggccc tcagagaagc agaaacctgt tttcacacca ctgaggaggc caggcatgct\n",
      "      301 tgtaccaaag cagaatgttt cacctccaga tatgtcagct ttgtctttaa aaggtgatca\n",
      "      361 agctctcctt ggaggcattt tttatttttt gaacattttt agccatggcg agctacctcc\n",
      "      421 tcattgtgaa cagcgcttcc tcccttgtga aattggctgt gttaagtatt ctctccaaga\n",
      "      481 aggtattatg gcagatttcc acagttttat aaatcctggt gaaattccac gaggatttcg\n",
      "      541 atttcattgt caggctgcaa gtgattctag tcacaagatt cctatttcaa attttgaacg\n",
      "      601 tgggcataac caagcaactg tgttacaaaa cctttataga tttattcatc ccaacccagg\n",
      "      661 gaactggcca cctatctact gcaagtctga tgatagaacc agagtcaact ggtgtttgaa\n",
      "      721 gcatatggca aaggcatcag aaatcaggca agatctacaa cttctcactg tagaggacct\n",
      "      781 tgtagtgggg atctaccaac aaaaatttct caaggagccc tctaagactt ggattcgaag\n",
      "      841 cctcctagat gtggccatgt gggattattc tagcaacaca aggtgcaagt ggcatgaaga\n",
      "      901 aaatgatatt ctcttctgtg ctttagctgt ttgcaagaag attgcgtact gcatcagtaa\n",
      "      961 ttctctggcc actctctttg gaatccagct cacagaggct catgtaccac tacaagatta\n",
      "     1021 tgaggccagc aatagtgtga cacccaaaat ggttgtattg gatgcagggc gttaccagaa\n",
      "     1081 gctaagggtt gggagttcag gattctctca tttcaactct tctaatgagg aacaaagatc\n",
      "     1141 aaacacaccc attggtgact acccatctag ggcaaaaatt tctggccaaa acagcagcgt\n",
      "     1201 tcggggaaga ggaattaccc gcttactaga gagcatttcc aattcttcca gcaatatcca\n",
      "     1261 caaattctcc aactgtgaca cttcactctc accttacatg tcccaaaaag atggatacaa\n",
      "     1321 atctttctct tccttatctt aatgatggta ctcttttcaa tttctgaaaa cagtaacagg\n",
      "     1381 cccaacttcc ttcttactac agtcatatta aacagatcac atcaatgaca aatgtcacta\n",
      "     1441 ctataaaaac tacttaattt gtaaggaaat tgtttcatag atttaaaaaa attgtggttg\n",
      "     1501 gagagcatct tggcatttgt gctttttttc ttgagggatt gttctgcttc ctggctgtat\n",
      "     1561 gatgggtata tcattaaagt ttggagtcct atatgaacaa aactgacatt tttagagttg\n",
      "     1621 tacttttggg aatgttatag attgatcatt ctttctcctg ataataaagg tattgaatat\n",
      "     1681 ctgttatgaa aggttctaaa a\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       NM_001329906            1879 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens nodal growth differentiation factor (NODAL),\n",
      "            transcript variant 2, mRNA.\n",
      "ACCESSION   NM_001329906\n",
      "VERSION     NM_001329906.2\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 1879)\n",
      "  AUTHORS   Taei A, Kiani T, Taghizadeh Z, Moradi S, Samadian A, Mollamohammadi\n",
      "            S, Sharifi-Zarchi A, Guenther S, Akhlaghpour A, Asgari Abibeiglou\n",
      "            B, Najar-Asl M, Karamzadeh R, Khalooghi K, Braun T, Hassani SN and\n",
      "            Baharvand H.\n",
      "  TITLE     Temporal activation of LRH-1 and RAR-gamma in human pluripotent\n",
      "            stem cells induces a functional naive-like state\n",
      "  JOURNAL   EMBO Rep 21 (10), e47533 (2020)\n",
      "   PUBMED   33252195\n",
      "REFERENCE   2  (bases 1 to 1879)\n",
      "  AUTHORS   Luck K, Kim DK, Lambourne L, Spirohn K, Begg BE, Bian W, Brignall\n",
      "            R, Cafarelli T, Campos-Laborie FJ, Charloteaux B, Choi D, Cote AG,\n",
      "            Daley M, Deimling S, Desbuleux A, Dricot A, Gebbia M, Hardy MF,\n",
      "            Kishore N, Knapp JJ, Kovacs IA, Lemmens I, Mee MW, Mellor JC,\n",
      "            Pollis C, Pons C, Richardson AD, Schlabach S, Teeking B, Yadav A,\n",
      "            Babor M, Balcha D, Basha O, Bowman-Colin C, Chin SF, Choi SG,\n",
      "            Colabella C, Coppin G, D'Amata C, De Ridder D, De Rouck S,\n",
      "            Duran-Frigola M, Ennajdaoui H, Goebels F, Goehring L, Gopal A,\n",
      "            Haddad G, Hatchi E, Helmy M, Jacob Y, Kassa Y, Landini S, Li R, van\n",
      "            Lieshout N, MacWilliams A, Markey D, Paulson JN, Rangarajan S,\n",
      "            Rasla J, Rayhan A, Rolland T, San-Miguel A, Shen Y, Sheykhkarimli\n",
      "            D, Sheynkman GM, Simonovsky E, Tasan M, Tejeda A, Tropepe V,\n",
      "            Twizere JC, Wang Y, Weatheritt RJ, Weile J, Xia Y, Yang X,\n",
      "            Yeger-Lotem E, Zhong Q, Aloy P, Bader GD, De Las Rivas J, Gaudet S,\n",
      "            Hao T, Rak J, Tavernier J, Hill DE, Vidal M, Roth FP and Calderwood\n",
      "            MA.\n",
      "  TITLE     A reference map of the human binary protein interactome\n",
      "  JOURNAL   Nature 580 (7803), 402-408 (2020)\n",
      "   PUBMED   32296183\n",
      "REFERENCE   3  (bases 1 to 1879)\n",
      "  AUTHORS   Tewary M, Dziedzicka D, Ostblom J, Prochazka L, Shakiba N, Heydari\n",
      "            T, Aguilar-Hidalgo D, Woodford C, Piccinini E, Becerra-Alonso D,\n",
      "            Vickers A, Louis B, Rahman N, Danovi D, Geens M, Watt FM and\n",
      "            Zandstra PW.\n",
      "  TITLE     High-throughput micropatterning platform reveals Nodal-dependent\n",
      "            bisection of peri-gastrulation-associated versus\n",
      "            preneurulation-associated fate patterning\n",
      "  JOURNAL   PLoS Biol 17 (10), e3000081 (2019)\n",
      "   PUBMED   31634368\n",
      "  REMARK    GeneRIF: Nodal expression determines peri-gastrulation-associated\n",
      "            versus preneurulation-associated fate patterning of pluripotent\n",
      "            stem cells.\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   4  (bases 1 to 1879)\n",
      "  AUTHORS   Chhabra S, Liu L, Goh R, Kong X and Warmflash A.\n",
      "  TITLE     Dissecting the dynamics of signaling events in the BMP, WNT, and\n",
      "            NODAL cascade during self-organized fate patterning in human\n",
      "            gastruloids\n",
      "  JOURNAL   PLoS Biol 17 (10), e3000498 (2019)\n",
      "   PUBMED   31613879\n",
      "  REMARK    GeneRIF: The study showed that the dynamics of signaling events in\n",
      "            the BMP4, WNT, and NODAL cascade in the absence of a stable\n",
      "            signaling gradient control fate patterning of human gastruloids.\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   5  (bases 1 to 1879)\n",
      "  AUTHORS   Sampath K and Robertson EJ.\n",
      "  TITLE     Keeping a lid on nodal: transcriptional and translational\n",
      "            repression of nodal signalling\n",
      "  JOURNAL   Open Biol 6 (1), 150200 (2016)\n",
      "   PUBMED   26791244\n",
      "  REMARK    Review article\n",
      "REFERENCE   6  (bases 1 to 1879)\n",
      "  AUTHORS   Roberts HJ, Hu S, Qiu Q, Leung PC, Caniggia I, Gruslin A, Tsang B\n",
      "            and Peng C.\n",
      "  TITLE     Identification of novel isoforms of activin receptor-like kinase 7\n",
      "            (ALK7) generated by alternative splicing and expression of ALK7 and\n",
      "            its ligand, Nodal, in human placenta\n",
      "  JOURNAL   Biol Reprod 68 (5), 1719-1726 (2003)\n",
      "   PUBMED   12606401\n",
      "  REMARK    GeneRIF: mRNA expression of Nodal, a ligand for ALK7, was detected\n",
      "            in placentae of different gestational age\n",
      "REFERENCE   7  (bases 1 to 1879)\n",
      "  AUTHORS   Yan YT, Liu JJ, Luo Y, E C, Haltiwanger RS, Abate-Shen C and Shen\n",
      "            MM.\n",
      "  TITLE     Dual roles of Cripto as a ligand and coreceptor in the nodal\n",
      "            signaling pathway\n",
      "  JOURNAL   Mol Cell Biol 22 (13), 4439-4449 (2002)\n",
      "   PUBMED   12052855\n",
      "REFERENCE   8  (bases 1 to 1879)\n",
      "  AUTHORS   Gebbia M, Ferrero GB, Pilia G, Bassi MT, Aylsworth A, Penman-Splitt\n",
      "            M, Bird LM, Bamforth JS, Burn J, Schlessinger D, Nelson DL and\n",
      "            Casey B.\n",
      "  TITLE     X-linked situs abnormalities result from mutations in ZIC3\n",
      "  JOURNAL   Nat Genet 17 (3), 305-308 (1997)\n",
      "   PUBMED   9354794\n",
      "REFERENCE   9  (bases 1 to 1879)\n",
      "  AUTHORS   Zhou X, Sasaki H, Lowe L, Hogan BL and Kuehn MR.\n",
      "  TITLE     Nodal is a novel TGF-beta-like gene expressed in the mouse node\n",
      "            during gastrulation\n",
      "  JOURNAL   Nature 361 (6412), 543-547 (1993)\n",
      "   PUBMED   8429908\n",
      "  REMARK    GeneRIF: Functional analysis of the mouse counterpart.\n",
      "REFERENCE   10 (bases 1 to 1879)\n",
      "  AUTHORS   Tekendo-Ngongang,C., Muenke,M. and Kruszka,P.\n",
      "  TITLE     Holoprosencephaly Overview\n",
      "  JOURNAL   (in) Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens\n",
      "            K and Amemiya A (Eds.);\n",
      "            GENEREVIEWS((R));\n",
      "            (1993)\n",
      "   PUBMED   20301702\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from AC022532.13, BC039861.1 and\n",
      "            AI670948.1.\n",
      "            On Aug 13, 2020 this sequence version replaced NM_001329906.1.\n",
      "            \n",
      "            Summary: This gene encodes a secreted ligand of the TGF-beta\n",
      "            (transforming growth factor-beta) superfamily of proteins. Ligands\n",
      "            of this family bind various TGF-beta receptors leading to\n",
      "            recruitment and activation of SMAD family transcription factors\n",
      "            that regulate gene expression. The encoded preproprotein is\n",
      "            proteolytically processed to generate the mature protein, which\n",
      "            regulates early embryonic development. This protein is required for\n",
      "            maintenance of human embryonic stem cell pluripotency and may play\n",
      "            a role in human placental development. Mutations in this gene are\n",
      "            associated with heterotaxy, a condition characterized by random\n",
      "            orientation of visceral organs with respect to the left-right axis.\n",
      "            [provided by RefSeq, Aug 2016].\n",
      "            \n",
      "            Transcript Variant: This variant (2) differs in the 5' UTR, lacks a\n",
      "            portion of the 5' coding region, and uses a downstream translation\n",
      "            start site compared to variant 1. The encoded isoform (2) has a\n",
      "            shorter N-terminus, and lacks the signal peptide and a portion of\n",
      "            the propeptide compared to isoform 1.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: BC039861.1, BM563257.1 [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-3                 AC022532.13        81444-81446\n",
      "            4-328               BC039861.1         1-325\n",
      "            329-329             AC022532.13        93712-93712\n",
      "            330-1352            BC039861.1         327-1349\n",
      "            1353-1600           AI670948.1         280-527             c\n",
      "            1601-1619           AC022532.13        97181-97199\n",
      "            1620-1879           AI670948.1         1-260               c\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..1879\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"10\"\n",
      "                     /map=\"10q22.1\"\n",
      "     gene            1..1879\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "                     /note=\"nodal growth differentiation factor\"\n",
      "                     /db_xref=\"GeneID:4838\"\n",
      "                     /db_xref=\"HGNC:HGNC:7865\"\n",
      "                     /db_xref=\"MIM:601265\"\n",
      "     exon            1..28\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            29..726\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     CDS             235..879\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "                     /note=\"isoform 2 is encoded by transcript variant 2; nodal\n",
      "                     homolog\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"nodal homolog isoform 2\"\n",
      "                     /protein_id=\"NP_001316835.1\"\n",
      "                     /db_xref=\"GeneID:4838\"\n",
      "                     /db_xref=\"HGNC:HGNC:7865\"\n",
      "                     /db_xref=\"MIM:601265\"\n",
      "                     /translation=\"MDLFTVTLSQVTFSLGSMVLEVTRPLSKWLKHPGALEKQMSRVA\n",
      "                     GECWPRPPTPPATNVLLMLYSNLSQEQRQLGGSTLLWEAESSWRAQEGQLSWEWGKRH\n",
      "                     RRHHLPDRSQLCRKVKFQVDFNLIGWGSWIIYPKQYNAYRCEGECPNPVGEEFHPTNH\n",
      "                     AYIQSLLKRYQPHRVPSTCCAPVKTKPLSMLYVDNGRVLLDHHKDMIVEECGCL\"\n",
      "     exon            727..1879\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      1345..1350\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "                     /note=\"hexamer: ATTAAA\"\n",
      "     polyA_site      1363\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "                     /note=\"major polyA site\"\n",
      "     regulatory      1481..1486\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      1500\n",
      "                     /gene=\"NODAL\"\n",
      "                     /gene_synonym=\"HTX5\"\n",
      "ORIGIN      \n",
      "        1 gaccaacagt tggggataca gactaaagat gtggcagtgg atgggcagaa ctggacgttt\n",
      "       61 gcttttgact tctccttcct gagccaacaa gaggatctgg catgggctga gctccggctg\n",
      "      121 cagctgtcca gccctgtgga cctccccact gagggctcac ttgccattga gattttccac\n",
      "      181 cagccaaagc ccgacacaga gcaggcttca gacagctgct tagagcggtt tcagatggac\n",
      "      241 ctattcactg tcactttgtc ccaggtcacc ttttccttgg gcagcatggt tttggaggtg\n",
      "      301 accaggcctc tctccaagtg gctgaagcac cctggggccc tggagaagca gatgtccagg\n",
      "      361 gtagctggag agtgctggcc gcggcccccc acaccgcctg ccaccaatgt gctccttatg\n",
      "      421 ctctactcca acctctcgca ggagcagagg cagctgggtg ggtccacctt gctgtgggaa\n",
      "      481 gccgagagct cctggcgggc ccaggaggga cagctgtcct gggagtgggg caagaggcac\n",
      "      541 cgtcgacatc acttgccaga cagaagtcaa ctgtgtcgga aggtcaagtt ccaggtggac\n",
      "      601 ttcaacctga tcggatgggg ctcctggatc atctacccca agcagtacaa cgcctatcgc\n",
      "      661 tgtgagggcg agtgtcctaa tcctgttggg gaggagtttc atccgaccaa ccatgcatac\n",
      "      721 atccagagtc tgctgaaacg ttaccagccc caccgagtcc cttccacttg ttgtgcccca\n",
      "      781 gtgaagacca agccgctgag catgctgtat gtggataatg gcagagtgct cctagatcac\n",
      "      841 cataaagaca tgatcgtgga agaatgtggg tgcctctgat gacatcctgg agggagactg\n",
      "      901 gatttgcctg cactctggaa ggctgggaaa ctcctggaag acatgataac catctaatcc\n",
      "      961 agtaaggaga aacagagagg ggcaaagttg ctctgcccac cagaactgaa gaggaggggc\n",
      "     1021 tgcccactct gtaaatgaag ggctcagtgg agtctggcca agcacagagg ctgctgtcag\n",
      "     1081 gaagagggag gaagaagcct gtgcaggggg ctggctggat gttctcttta ctgaaaagac\n",
      "     1141 agtggcaagg aaaagcacaa gtgcatgagt tctttactgg attttttaaa aacctgtgaa\n",
      "     1201 ccccccgaaa ctgtatgtga aagttgagac atatgtgcat gtattttgga ggtgggatga\n",
      "     1261 agtcacctat agctttcatg tattctccaa agtagtctgt gtgtgacctg tccccctccc\n",
      "     1321 caaagattaa ggatcactgt atagattaaa aagagtccgt caatctcatt gcctcaggct\n",
      "     1381 gggttggggg agccccacag ctttctggct ggccagtggc aatctactgg ccttgtccag\n",
      "     1441 aggctcactg gagtggttct ctgctaatga gctgtacaac aataaagcca ttgtctagtt\n",
      "     1501 ctcctgggcc agctggtgcc tgtgaaggca gaggcaggaa ctcatccaag aggaccggcc\n",
      "     1561 atgttgggtt acagaagaca tccctgcgtc agtctgcttc ggcagacaca gcctgagttt\n",
      "     1621 gttaaagttg gtgacaatcc acctcagtct ctcaatgtgt gctattaatg aggcctctga\n",
      "     1681 gcttcctatc cagcagtggt gaaggccttg ccctgggtgg caagatactt gctctatggt\n",
      "     1741 cacagctcag ccactggaag ctgtgcgacc tcaggtgagc aattcactgt ccagtctcca\n",
      "     1801 cttgtaaaag gaacgctggt gaatcctaat gcattcatat taaatgtctg ttgtcaggct\n",
      "     1861 cagaagagcc atgagcttt\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       NM_001164720            1508 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens coiled-coil and C2 domain containing 2A (CC2D2A),\n",
      "            transcript variant 3, mRNA.\n",
      "ACCESSION   NM_001164720 NR_024098\n",
      "VERSION     NM_001164720.3\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 1508)\n",
      "  AUTHORS   Lam Z, Albaba S, Study D and Balasubramanian M.\n",
      "  TITLE     Atypical, milder presentation in a child with CC2D2A and KIDINS220\n",
      "            variants\n",
      "  JOURNAL   Clin Dysmorphol 29 (1), 10-16 (2020)\n",
      "   PUBMED   31577543\n",
      "  REMARK    GeneRIF: Atypical, milder presentation in a child with CC2D2A and\n",
      "            KIDINS220 variants.\n",
      "REFERENCE   2  (bases 1 to 1508)\n",
      "  AUTHORS   Mejecase C, Hummel A, Mohand-Said S, Andrieu C, El Shamieh S,\n",
      "            Antonio A, Condroyer C, Boyard F, Foussard M, Blanchard S, Letexier\n",
      "            M, Saraiva JP, Sahel JA, Zeitz C and Audo I.\n",
      "  TITLE     Whole exome sequencing resolves complex phenotype and identifies\n",
      "            CC2D2A mutations underlying non-syndromic rod-cone dystrophy\n",
      "  JOURNAL   Clin Genet 95 (2), 329-333 (2019)\n",
      "   PUBMED   30267408\n",
      "  REMARK    GeneRIF: Our study identifies for the first time CC2D2A mutations\n",
      "            in isolated RCD and underlines the power of WES to decipher complex\n",
      "            phenotypes.\n",
      "REFERENCE   3  (bases 1 to 1508)\n",
      "  AUTHORS   Takenouchi T, Kuchikata T, Yoshihashi H, Fujiwara M, Uehara T,\n",
      "            Miyama S, Yamada S and Kosaki K.\n",
      "  TITLE     Diagnostic use of computational retrotransposon detection:\n",
      "            Successful definition of pathogenetic mechanism in a ciliopathy\n",
      "            phenotype\n",
      "  JOURNAL   Am J Med Genet A 173 (5), 1353-1357 (2017)\n",
      "   PUBMED   28374938\n",
      "  REMARK    GeneRIF: Using a dedicated bioinformatics algorithm for TE\n",
      "            detection, we identified an exonic retrotransposon insertion of L1\n",
      "            to the CC2D2A locus in a patient with Meckel-Gruber syndrome, the\n",
      "            most severe form of the ciliopathy phenotypes.\n",
      "REFERENCE   4  (bases 1 to 1508)\n",
      "  AUTHORS   Al-Hamed MH, Kurdi W, Alsahan N, Alabdullah Z, Abudraz R, Tulbah M,\n",
      "            Alnemer M, Khan R, Al-Jurayb H, Alahmed A, Tahir AI, Khalil D,\n",
      "            Edwards N, Al Abdulaziz B, Binhumaid FS, Majid S, Faquih T,\n",
      "            El-Kalioby M, Abouelhoda M, Altassan N, Monies D, Meyer B, Sayer JA\n",
      "            and Albaqumi M.\n",
      "  TITLE     Genetic spectrum of Saudi Arabian patients with antenatal cystic\n",
      "            kidney disease and ciliopathy phenotypes using a targeted renal\n",
      "            gene panel\n",
      "  JOURNAL   J Med Genet 53 (5), 338-347 (2016)\n",
      "   PUBMED   26862157\n",
      "  REMARK    GeneRIF: Mutations in CC2D2A were the most common cause of an\n",
      "            antenatal cystic kidney disease and a suspected ciliopathy in Saudi\n",
      "            Arabian cohort.\n",
      "REFERENCE   5  (bases 1 to 1508)\n",
      "  AUTHORS   Noor,A., Windpassinger,C., Patel,M., Stachowiak,B., Mikhailov,A.,\n",
      "            Azam,M., Irfan,M., Paterson,A.D., Lutufullah,M., Doherty,D.,\n",
      "            Vincent,J.B. and Ayub,M.\n",
      "  TITLE     Addendum. CC2D2A, encoding a coiled-coil and C2 domain protein,\n",
      "            causes autosomal-recessive mental retardation with retinitis\n",
      "            pigmentosa\n",
      "  JOURNAL   Am J Hum Genet 83 (5), 656 (2008)\n",
      "   PUBMED   19068953\n",
      "  REMARK    GeneRIF: CC2D2A causes autosomal-recessive mental retardation with\n",
      "            retinitis pigmentosa.\n",
      "REFERENCE   6  (bases 1 to 1508)\n",
      "  AUTHORS   Gorden NT, Arts HH, Parisi MA, Coene KL, Letteboer SJ, van Beersum\n",
      "            SE, Mans DA, Hikida A, Eckert M, Knutzen D, Alswaid AF, Ozyurek H,\n",
      "            Dibooglu S, Otto EA, Liu Y, Davis EE, Hutter CM, Bammler TK, Farin\n",
      "            FM, Dorschner M, Topcu M, Zackai EH, Rosenthal P, Owens KN,\n",
      "            Katsanis N, Vincent JB, Hildebrandt F, Rubel EW, Raible DW, Knoers\n",
      "            NV, Chance PF, Roepman R, Moens CB, Glass IA and Doherty D.\n",
      "  TITLE     CC2D2A is mutated in Joubert syndrome and interacts with the\n",
      "            ciliopathy-associated basal body protein CEP290\n",
      "  JOURNAL   Am J Hum Genet 83 (5), 559-571 (2008)\n",
      "   PUBMED   18950740\n",
      "  REMARK    GeneRIF: CC2D2A is mutated in Joubert syndrome and interacts with\n",
      "            the ciliopathy-associated basal body protein CEP290.\n",
      "REFERENCE   7  (bases 1 to 1508)\n",
      "  AUTHORS   Tallila J, Jakkula E, Peltonen L, Salonen R and Kestila M.\n",
      "  TITLE     Identification of CC2D2A as a Meckel syndrome gene adds an\n",
      "            important piece to the ciliopathy puzzle\n",
      "  JOURNAL   Am J Hum Genet 82 (6), 1361-1367 (2008)\n",
      "   PUBMED   18513680\n",
      "  REMARK    GeneRIF: CC2D2A appears to have at least two cilia-related\n",
      "            functions: the molecule seems to be a part of the basal body\n",
      "            complex where the cilium is assembled from and also seems to act as\n",
      "            a sensor for the intracellular calcium.\n",
      "REFERENCE   8  (bases 1 to 1508)\n",
      "  AUTHORS   Noor A, Windpassinger C, Patel M, Stachowiak B, Mikhailov A, Azam\n",
      "            M, Irfan M, Siddiqui ZK, Naeem F, Paterson AD, Lutfullah M, Vincent\n",
      "            JB and Ayub M.\n",
      "  TITLE     CC2D2A, encoding a coiled-coil and C2 domain protein, causes\n",
      "            autosomal-recessive mental retardation with retinitis pigmentosa\n",
      "  JOURNAL   Am J Hum Genet 82 (4), 1011-1018 (2008)\n",
      "   PUBMED   18387594\n",
      "  REMARK    GeneRIF: A homozygous splice-site mutation segregating in the\n",
      "            family with autosomal-recessive mental retardation, within a\n",
      "            coiled-coil and C2 domain-containing gene, CC2D2A was identified.\n",
      "REFERENCE   9  (bases 1 to 1508)\n",
      "  AUTHORS   Gunay-Aygun,M., Gahl,W.A. and Heller,T.\n",
      "  TITLE     Congenital Hepatic Fibrosis Overview horizontal line RETIRED\n",
      "            CHAPTER, FOR HISTORICAL REFERENCE ONLY\n",
      "  JOURNAL   (in) Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens\n",
      "            K and Amemiya A (Eds.);\n",
      "            GENEREVIEWS((R));\n",
      "            (1993)\n",
      "   PUBMED   20301743\n",
      "REFERENCE   10 (bases 1 to 1508)\n",
      "  AUTHORS   Parisi,M. and Glass,I.\n",
      "  TITLE     Joubert Syndrome\n",
      "  JOURNAL   (in) Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Stephens\n",
      "            K and Amemiya A (Eds.);\n",
      "            GENEREVIEWS((R));\n",
      "            (1993)\n",
      "   PUBMED   20301500\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from N35522.1, BC053865.1 and\n",
      "            AI376394.1.\n",
      "            On Aug 13, 2020 this sequence version replaced NM_001164720.2.\n",
      "            \n",
      "            Summary: This gene encodes a coiled-coil and calcium binding domain\n",
      "            protein that appears to play a critical role in cilia formation.\n",
      "            Mutations in this gene cause Meckel syndrome type 6, as well as\n",
      "            Joubert syndrome type 9. Alternative splicing results in multiple\n",
      "            transcript variants. [provided by RefSeq, Sep 2009].\n",
      "            \n",
      "            Transcript Variant: This variant (3) differs in the presence and\n",
      "            absence of exons at its 3' end, compared to variant 1. The encoded\n",
      "            isoform (c) has a distinct and significantly shorter C-terminus,\n",
      "            compared to isoform a.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: BC053865.1, SRR3476690.596220.1\n",
      "                                           [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-30                N35522.1           18-47\n",
      "            31-1504             BC053865.1         1-1474\n",
      "            1505-1508           AI376394.1         18-21               c\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..1508\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"4\"\n",
      "                     /map=\"4p15.32\"\n",
      "     gene            1..1508\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /note=\"coiled-coil and C2 domain containing 2A\"\n",
      "                     /db_xref=\"GeneID:57545\"\n",
      "                     /db_xref=\"HGNC:HGNC:29253\"\n",
      "                     /db_xref=\"MIM:612013\"\n",
      "     exon            1..176\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            177..233\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     CDS             195..530\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /note=\"isoform c is encoded by transcript variant 3;\n",
      "                     coiled-coil and C2 domain-containing protein 2A\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"coiled-coil and C2 domain-containing protein 2A\n",
      "                     isoform c\"\n",
      "                     /protein_id=\"NP_001158192.1\"\n",
      "                     /db_xref=\"CCDS:CCDS54744.1\"\n",
      "                     /db_xref=\"GeneID:57545\"\n",
      "                     /db_xref=\"HGNC:HGNC:29253\"\n",
      "                     /db_xref=\"MIM:612013\"\n",
      "                     /translation=\"MNPREEKVKIITEEFIENDEDADMGRQNKNSKVRRQPRKKQPPT\n",
      "                     AVPKEMVSEKSHLGNPQEPVQEEPKTRLLSMTVRRGPRRELVVKKSLGRPGTVTHVCN\n",
      "                     PSTLEGRGG\"\n",
      "     exon            234..317\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            318..441\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            442..1508\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      936..941\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      961\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /note=\"major polyA site\"\n",
      "     regulatory      1486..1491\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "                     /note=\"hexamer: AATAAA\"\n",
      "     polyA_site      1508\n",
      "                     /gene=\"CC2D2A\"\n",
      "                     /gene_synonym=\"COACH2; JBTS9; MKS6\"\n",
      "ORIGIN      \n",
      "        1 aagctggtgt ttttgctcca agacatggct gcagcttccc agggaggagc ccaggggcat\n",
      "       61 ctccaacact cagcctttcc ctccggaccc ttttaaaaga ttcaacagca ccgtctctct\n",
      "      121 ccacttccat cctagaatgc agaatctgca aagtgccttt tgtaaagttt cttaagggac\n",
      "      181 ccatcccagc caaaatgaat cccagggaag aaaaagtaaa aataattaca gaggagttca\n",
      "      241 ttgaaaatga tgaggatgca gacatgggaa gacagaataa gaactcaaag gttcgaagac\n",
      "      301 agccaagaaa gaaacagcca ccaactgctg tccccaagga aatggtgtcc gaaaaatccc\n",
      "      361 accttggcaa cccccaggag cctgtgcagg aggagcccaa gacccgcctc ctgagtatga\n",
      "      421 cagtccggag aggcccacgg agagagctgg ttgttaaaaa gagcctgggc aggccaggca\n",
      "      481 cggtgactca cgtctgtaat cccagcactt tggaaggccg aggcgggtaa atcacctgag\n",
      "      541 gttgggagtt caagaccagc ctgaccaaca tggagaaacc ccatctctac taaaaataca\n",
      "      601 aaaaaattag ccaggcgtgg tggcgcatgc ctgtaatccc agctacttgg gaggctgagg\n",
      "      661 catgagaatt gcttgaacct gggaggtgga ggttgcagtg agccgagatc ggagcactgc\n",
      "      721 actccagcct aggcaacaga gcaagactcc atctcagaaa acaaacaaac aaaaaagagc\n",
      "      781 ctggaacctc ctcccctctc tcttgctccc tctctcacca tgcgacacac ctgtttcccc\n",
      "      841 ttccccttct gacataagtg aaagcttcct aaggcctcac cagaggcaga tgctggcacc\n",
      "      901 atgcttattg cacagtctgc agaacagtga gccaaaataa acctcttttc tttataaatt\n",
      "      961 acccagtctc agatattcct ttatagcaat gcaaaatgga ctaacaatac ataagttagg\n",
      "     1021 gagtgttaat tagaatggaa attaaatatg cagaaactca aaggaagatg aaattgttag\n",
      "     1081 ggaagacata tgtgttaacc tagtgaggct tgaatcagat gtttggaaga aatgagggag\n",
      "     1141 gtttctgagg ctcctggtga ccttgccatt gaggacacat atacttctcc ccctacacac\n",
      "     1201 gccttttctt cttccccagt acaatgaaca ttaggaatta cctgctcatt gtagggaaaa\n",
      "     1261 tgaccaccct tcaggcattg cttttcatga aaagtggatt agacaagagg agaagtggtt\n",
      "     1321 acacttttta attgttacat aattgttaga ctgttttccc tatggcaaag ttgggaaata\n",
      "     1381 agtaactgtt ggaaactaga ggcctttgga agcaagtaga ctcttagatt ggaaaaatgt\n",
      "     1441 ggttctaatc taaaagcaat gatacaatga gagctgttat gaattaataa aaacagaacc\n",
      "     1501 aaaattta\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       NM_004194               2883 bp    mRNA    linear   PRI 03-DEC-2020\n",
      "DEFINITION  Homo sapiens ADAM metallopeptidase domain 22 (ADAM22), transcript\n",
      "            variant 4, mRNA.\n",
      "ACCESSION   NM_004194\n",
      "VERSION     NM_004194.5\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (bases 1 to 2883)\n",
      "  AUTHORS   Sundell GN, Arnold R, Ali M, Naksukpaiboon P, Orts J, Guntert P,\n",
      "            Chi CN and Ivarsson Y.\n",
      "  TITLE     Proteome-wide analysis of phospho-regulated PDZ domain interactions\n",
      "  JOURNAL   Mol Syst Biol 14 (8), e8129 (2018)\n",
      "   PUBMED   30126976\n",
      "  REMARK    Publication Status: Online-Only\n",
      "REFERENCE   2  (bases 1 to 2883)\n",
      "  AUTHORS   Yamagata A, Miyazaki Y, Yokoi N, Shigematsu H, Sato Y, Goto-Ito S,\n",
      "            Maeda A, Goto T, Sanbo M, Hirabayashi M, Shirouzu M, Fukata Y,\n",
      "            Fukata M and Fukai S.\n",
      "  TITLE     Structural basis of epilepsy-related ligand-receptor complex\n",
      "            LGI1-ADAM22\n",
      "  JOURNAL   Nat Commun 9 (1), 1546 (2018)\n",
      "   PUBMED   29670100\n",
      "  REMARK    GeneRIF: the LGI1-ADAM22 complex functions as the trans-synaptic\n",
      "            machinery for precise synaptic transmission\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   3  (bases 1 to 2883)\n",
      "  AUTHORS   Li J, Lu M, Jin J, Lu X, Xu T and Jin S.\n",
      "  TITLE     miR-449a Suppresses Tamoxifen Resistance in Human Breast Cancer\n",
      "            Cells by Targeting ADAM22\n",
      "  JOURNAL   Cell Physiol Biochem 50 (1), 136-149 (2018)\n",
      "   PUBMED   30278449\n",
      "  REMARK    GeneRIF: ADAM22 is critically involved in miR-449a-reduced\n",
      "            tamoxifen resistance of estrogen receptor-positive breast cancer\n",
      "            cells as a direct target of miR-449a.\n",
      "REFERENCE   4  (bases 1 to 2883)\n",
      "  AUTHORS   Dazzo E, Leonardi E, Belluzzi E, Malacrida S, Vitiello L, Greggio\n",
      "            E, Tosatto SC and Nobile C.\n",
      "  TITLE     Secretion-Positive LGI1 Mutations Linked to Lateral Temporal\n",
      "            Epilepsy Impair Binding to ADAM22 and ADAM23 Receptors\n",
      "  JOURNAL   PLoS Genet 12 (10), e1006376 (2016)\n",
      "   PUBMED   27760137\n",
      "  REMARK    GeneRIF: these results support the existence of a second mechanism,\n",
      "            alternative to inhibition of protein secretion, by which\n",
      "            ADLTE-causing LGI1 mutations exert their loss-of-function effect\n",
      "            extracellularly, and suggest that interactions of LGI1 with both\n",
      "            ADAM22 and ADAM23 play an important role in the molecular\n",
      "            mechanisms leading to utosomal dominant lateral temporal epilepsy\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   5  (bases 1 to 2883)\n",
      "  AUTHORS   Muona M, Fukata Y, Anttonen AK, Laari A, Palotie A, Pihko H,\n",
      "            Lonnqvist T, Valanne L, Somer M, Fukata M and Lehesjoki AE.\n",
      "  TITLE     Dysfunctional ADAM22 implicated in progressive encephalopathy with\n",
      "            cortical atrophy and epilepsy\n",
      "  JOURNAL   Neurol Genet 2 (1), e46 (2016)\n",
      "   PUBMED   27066583\n",
      "  REMARK    Publication Status: Online-Only\n",
      "REFERENCE   6  (bases 1 to 2883)\n",
      "  AUTHORS   Hillman RT, Green RE and Brenner SE.\n",
      "  TITLE     An unappreciated role for RNA surveillance\n",
      "  JOURNAL   Genome Biol 5 (2), R8 (2004)\n",
      "   PUBMED   14759258\n",
      "REFERENCE   7  (bases 1 to 2883)\n",
      "  AUTHORS   Zhu Pc, Sun Y, Xu R, Sang Y, Zhao J, Liu G, Cai L, Li C and Zhao S.\n",
      "  TITLE     The interaction between ADAM 22 and 14-3-3zeta: regulation of cell\n",
      "            adhesion and spreading\n",
      "  JOURNAL   Biochem Biophys Res Commun 301 (4), 991-999 (2003)\n",
      "   PUBMED   12589811\n",
      "  REMARK    GeneRIF: role for the 14-3-3zeta/ADAM 22 association in the\n",
      "            regulation of cell adhesion and related signaling events\n",
      "REFERENCE   8  (bases 1 to 2883)\n",
      "  AUTHORS   Harada T, Nishie A, Torigoe K, Ikezaki K, Shono T, Maehara Y,\n",
      "            Kuwano M and Wada M.\n",
      "  TITLE     The specific expression of three novel splice variant forms of\n",
      "            human metalloprotease-like disintegrin-like cysteine-rich protein 2\n",
      "            gene inBrain tissues and gliomas\n",
      "  JOURNAL   Jpn J Cancer Res 91 (10), 1001-1006 (2000)\n",
      "   PUBMED   11050470\n",
      "REFERENCE   9  (bases 1 to 2883)\n",
      "  AUTHORS   Poindexter K, Nelson N, DuBose RF, Black RA and Cerretti DP.\n",
      "  TITLE     The identification of seven metalloproteinase-disintegrin (ADAM)\n",
      "            genes from genomic libraries\n",
      "  JOURNAL   Gene 237 (1), 61-70 (1999)\n",
      "   PUBMED   10524237\n",
      "REFERENCE   10 (bases 1 to 2883)\n",
      "  AUTHORS   Sagane K, Ohya Y, Hasegawa Y and Tanaka I.\n",
      "  TITLE     Metalloproteinase-like, disintegrin-like, cysteine-rich proteins\n",
      "            MDC2 and MDC3: novel human cellular disintegrins highly expressed\n",
      "            in the brain\n",
      "  JOURNAL   Biochem J 334 (Pt 1), 93-98 (1998)\n",
      "   PUBMED   9693107\n",
      "COMMENT     REVIEWED REFSEQ: This record has been curated by NCBI staff. The\n",
      "            reference sequence was derived from DR000609.1, AB009671.1 and\n",
      "            AC005075.2.\n",
      "            On Aug 13, 2020 this sequence version replaced NM_004194.4.\n",
      "            \n",
      "            Summary: This gene encodes a member of the ADAM (a disintegrin and\n",
      "            metalloprotease domain) family. Members of this family are\n",
      "            membrane-anchored proteins structurally related to snake venom\n",
      "            disintegrins, and have been implicated in a variety of biological\n",
      "            processes involving cell-cell and cell-matrix interactions,\n",
      "            including fertilization, muscle development, and neurogenesis.\n",
      "            Unlike other members of the ADAM protein family, the protein\n",
      "            encoded by this gene lacks metalloprotease activity since it has no\n",
      "            zinc-binding motif. This gene is highly expressed in the brain and\n",
      "            may function as an integrin ligand in the brain. In mice, it has\n",
      "            been shown to be essential for correct myelination in the\n",
      "            peripheral nervous system. Alternative splicing results in several\n",
      "            transcript variants.[provided by RefSeq, Dec 2010].\n",
      "            \n",
      "            Transcript Variant: This variant (4), also known as beta, uses an\n",
      "            alternate splice site in the 3' coding region and lacks two\n",
      "            downstream exons, compared to variant 1. The resulting protein\n",
      "            (isoform 4) has a shorter and distinct C-terminus, compared to\n",
      "            isoform 1. The encoded isoform (4) may undergo proteolytic\n",
      "            processing.\n",
      "            \n",
      "            Publication Note:  This RefSeq record includes a subset of the\n",
      "            publications that are available for this gene. Please see the Gene\n",
      "            record to access additional publications.\n",
      "            \n",
      "            ##Evidence-Data-START##\n",
      "            Transcript exon combination :: AB009671.1, SRR1660809.167459.1\n",
      "                                           [ECO:0000332]\n",
      "            ##Evidence-Data-END##\n",
      "            COMPLETENESS: complete on the 3' end.\n",
      "PRIMARY     REFSEQ_SPAN         PRIMARY_IDENTIFIER PRIMARY_SPAN        COMP\n",
      "            1-136               DR000609.1         117-252\n",
      "            137-2880            AB009671.1         43-2786\n",
      "            2881-2883           AC005075.2         88176-88178\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..2883\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /mol_type=\"mRNA\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "                     /chromosome=\"7\"\n",
      "                     /map=\"7q21.12\"\n",
      "     gene            1..2883\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"ADAM metallopeptidase domain 22\"\n",
      "                     /db_xref=\"GeneID:53616\"\n",
      "                     /db_xref=\"HGNC:HGNC:201\"\n",
      "                     /db_xref=\"MIM:603709\"\n",
      "     exon            1..300\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     misc_feature    180..182\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"upstream in-frame stop codon\"\n",
      "     CDS             216..2795\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"isoform 4 precursor is encoded by transcript\n",
      "                     variant 4; a disintegrin and metalloproteinase domain 22;\n",
      "                     metalloproteinase-like, disintegrin-like, and\n",
      "                     cysteine-rich protein 2; metalloproteinase-disintegrin\n",
      "                     ADAM22-3; disintegrin and metalloproteinase\n",
      "                     domain-containing protein 22\"\n",
      "                     /codon_start=1\n",
      "                     /product=\"disintegrin and metalloproteinase\n",
      "                     domain-containing protein 22 isoform 4 precursor\"\n",
      "                     /protein_id=\"NP_004185.1\"\n",
      "                     /db_xref=\"CCDS:CCDS43610.1\"\n",
      "                     /db_xref=\"GeneID:53616\"\n",
      "                     /db_xref=\"HGNC:HGNC:201\"\n",
      "                     /db_xref=\"MIM:603709\"\n",
      "                     /translation=\"MQAAVAVSVPFLLLCVLGTCPPARCGQAGDASLMELEKRKENRF\n",
      "                     VERQSIVPLRLIYRSGGEDESRHDALDTRVRGDLGGPQLTHVDQASFQVDAFGTSFIL\n",
      "                     DVVLNHDLLSSEYIERHIEHGGKTVEVKGGEHCYYQGHIRGNPDSFVALSTCHGLHGM\n",
      "                     FYDGNHTYLIEPEENDTTQEDFHFHSVYKSRLFEFSLDDLPSEFQQVNITPSKFILKP\n",
      "                     RPKRSKRQLRRYPRNVEEETKYIELMIVNDHLMFKKHRLSVVHTNTYAKSVVNMADLI\n",
      "                     YKDQLKTRIVLVAMETWATDNKFAISENPLITLREFMKYRRDFIKEKSDAVHLFSGSQ\n",
      "                     FESSRSGAAYIGGICSLLKGGGVNEFGKTDLMAVTLAQSLAHNIGIISDKRKLASGEC\n",
      "                     KCEDTWSGCIMGDTGYYLPKKFTQCNIEEYHDFLNSGGGACLFNKPSKLLDPPECGNG\n",
      "                     FIETGEECDCGTPAECVLEGAECCKKCTLTQDSQCSDGLCCKKCKFQPMGTVCREAVN\n",
      "                     DCDIRETCSGNSSQCAPNIHKMDGYSCDGVQGICFGGRCKTRDRQCKYIWGQKVTASD\n",
      "                     KYCYEKLNIEGTEKGNCGKDKDTWIQCNKRDVLCGYLLCTNIGNIPRLGELDGEITST\n",
      "                     LVVQQGRTLNCSGGHVKLEEDVDLGYVEDGTPCGPQMMCLEHRCLPVASFNFSTCLSS\n",
      "                     KEGTICSGNGVCSNELKCVCNRHWIGSDCNTYFPHNDDAKTGITLSGNGVAGTNIIIG\n",
      "                     IIAGTILVLALILGITAWGYKNYREQRQLPQGDYVKKPGDGDSFYSDIPPGVSTNSAS\n",
      "                     SSKKRSNGLSHSWSERIPDTKHISDICENGRPRSNSWQGNLGGNKKKIRGKRFRPRSN\n",
      "                     STE\"\n",
      "     sig_peptide     216..290\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"COORDINATES: ab initio prediction:SignalP:4.0\"\n",
      "     misc_feature    738..740\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000255; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q9P0K1.1); glycosylation site\"\n",
      "     misc_feature    1770..1772\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000269|PubMed:19692335; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q9P0K1.1); glycosylation site\"\n",
      "     misc_feature    2115..2117\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000269|PubMed:19692335; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q9P0K1.1); glycosylation site\"\n",
      "     misc_feature    2238..2240\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"N-linked (GlcNAc...) asparagine.\n",
      "                     /evidence=ECO:0000269|PubMed:19692335; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q9P0K1.1); glycosylation site\"\n",
      "     misc_feature    2424..2486\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"propagated from UniProtKB/Swiss-Prot (Q9P0K1.1);\n",
      "                     transmembrane region\"\n",
      "     misc_feature    2643..2645\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"Phosphoserine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:Q9R1V6; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q9P0K1.1); phosphorylation site\"\n",
      "     misc_feature    2715..2717\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"Phosphoserine.\n",
      "                     /evidence=ECO:0000244|PubMed:21406692; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q9P0K1.1); phosphorylation site\"\n",
      "     misc_feature    2784..2786\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"Phosphoserine.\n",
      "                     /evidence=ECO:0000250|UniProtKB:Q9R1V6; propagated from\n",
      "                     UniProtKB/Swiss-Prot (Q9P0K1.1); phosphorylation site\"\n",
      "     exon            301..461\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            462..538\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            539..605\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            606..688\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            689..752\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            753..822\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            823..893\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            894..968\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            969..1040\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1041..1207\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1208..1292\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1293..1383\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1384..1435\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1436..1535\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1536..1607\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1608..1700\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1701..1781\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1782..1832\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1833..1896\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            1897..2002\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2003..2122\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2123..2291\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2292..2406\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2407..2497\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2498..2515\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2516..2623\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2624..2724\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     exon            2725..2883\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /inference=\"alignment:Splign:2.1.0\"\n",
      "     regulatory      2851..2856\n",
      "                     /regulatory_class=\"polyA_signal_sequence\"\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"hexamer: AGTAAA\"\n",
      "     polyA_site      2883\n",
      "                     /gene=\"ADAM22\"\n",
      "                     /gene_synonym=\"ADAM 22; DEE61; EIEE61; MDC2\"\n",
      "                     /note=\"major polyA site\"\n",
      "ORIGIN      \n",
      "        1 actcgctcgc tccccccgcc agcggaagcg tccgcgaagc acaatgcagc actgagccgc\n",
      "       61 ggtggaggtt gcagcgccac ggccgccgca gcaccggccg gggctgggtg gaggtggccg\n",
      "      121 cggggacccc gggggcgcgg agcgagggaa acggactcgg cggcgccggc atgaggagct\n",
      "      181 gagcgtctcg ggcgaggcgg gctgacggca gcaccatgca ggcggcagtg gctgtgtccg\n",
      "      241 tgcccttctt gctgctctgt gtcctgggga cctgccctcc ggcgcgctgc ggccaggcag\n",
      "      301 gagacgcctc attgatggag ctagagaaga ggaaggaaaa ccgcttcgtg gagcgccaga\n",
      "      361 gcatcgtgcc actgcgcctc atctaccgct cgggcggcga agacgaaagt cggcacgacg\n",
      "      421 cgctcgacac gcgggtgcgg ggcgacctcg gtggcccgca gttgactcat gttgaccaag\n",
      "      481 caagcttcca ggttgatgcc tttggaacgt cattcattct cgatgtcgtg ctaaatcatg\n",
      "      541 atttgctgtc ctctgaatac atagagagac acattgaaca tggaggcaag actgtggaag\n",
      "      601 ttaaaggagg agagcactgt tactaccagg gccatatccg aggaaaccct gactcatttg\n",
      "      661 ttgcattgtc aacatgccac ggacttcatg ggatgttcta tgacgggaac cacacatatc\n",
      "      721 tcattgagcc agaagaaaat gacactactc aagaggattt ccattttcat tcagtttaca\n",
      "      781 aatccagact gtttgaattt tccttggatg atcttccatc tgaatttcag caagtaaaca\n",
      "      841 ttactccatc aaaatttatt ttgaagccaa gaccaaaaag gagtaaacgg cagcttcgtc\n",
      "      901 gatatcctcg taatgtagaa gaagaaacca aatacattga actgatgatt gtgaatgatc\n",
      "      961 accttatgtt taaaaaacat cggctttccg ttgtacatac caatacctat gcgaaatctg\n",
      "     1021 tggtgaacat ggcagattta atatataaag accaacttaa gaccaggata gtattggttg\n",
      "     1081 ctatggaaac ctgggcgact gacaacaagt ttgccatatc tgaaaatcca ttgatcaccc\n",
      "     1141 tacgtgagtt tatgaaatac aggagggatt ttatcaaaga gaaaagtgat gcagttcacc\n",
      "     1201 ttttttcggg aagtcaattt gagagtagcc ggagcggggc agcttatatt ggtgggattt\n",
      "     1261 gctcgttgct gaaaggagga ggcgtgaatg aatttgggaa aactgattta atggctgtta\n",
      "     1321 cacttgccca gtcattagcc cataatattg gtattatctc agacaaaaga aagttagcaa\n",
      "     1381 gtggtgaatg taaatgcgag gacacgtggt ccgggtgcat aatgggagac actggctatt\n",
      "     1441 atcttcctaa aaagttcacc cagtgtaata ttgaagagta tcatgacttc ctgaatagtg\n",
      "     1501 gaggtggtgc ctgccttttc aacaaacctt ctaagcttct tgatcctcct gagtgtggca\n",
      "     1561 atggcttcat tgaaactgga gaggagtgtg attgtggaac cccggccgaa tgtgtccttg\n",
      "     1621 aaggagcaga gtgttgtaag aaatgcacct tgactcaaga ctctcaatgc agtgacggtc\n",
      "     1681 tttgctgtaa aaagtgcaag tttcagccta tgggcactgt gtgccgagaa gcagtaaatg\n",
      "     1741 attgtgatat tcgtgaaacg tgctcaggaa attcaagcca gtgtgcccct aatattcata\n",
      "     1801 aaatggatgg atattcatgt gatggtgttc agggaatttg ctttggagga agatgcaaaa\n",
      "     1861 ccagagatag acaatgcaaa tacatttggg ggcaaaaggt gacagcatca gacaaatatt\n",
      "     1921 gctatgagaa actgaatatt gaagggacgg agaagggtaa ctgtgggaaa gacaaagaca\n",
      "     1981 catggataca gtgcaacaaa cgggatgtgc tttgtggtta ccttttgtgt accaatattg\n",
      "     2041 gcaatatccc aaggcttgga gaactcgatg gtgaaatcac atctacttta gttgtgcagc\n",
      "     2101 aaggaagaac attaaactgc agtggtgggc atgttaagct tgaagaagat gtagatcttg\n",
      "     2161 gctatgtgga agatgggaca ccttgtggtc cccaaatgat gtgcttagaa cacaggtgtc\n",
      "     2221 ttcctgtggc ttctttcaac tttagtactt gcttgagcag taaagaaggc actatttgct\n",
      "     2281 caggaaatgg agtttgcagt aatgagctga agtgtgtgtg taacagacac tggataggtt\n",
      "     2341 ctgattgcaa cacttacttc cctcacaatg atgatgcaaa gactggtatc actctgtctg\n",
      "     2401 gcaatggtgt tgctggcacc aatatcataa taggcataat tgctggcacc attttagtgc\n",
      "     2461 tggccctcat attaggaata actgcgtggg gttataaaaa ctatcgagaa cagagacagt\n",
      "     2521 taccccaggg agattatgta aaaaagcctg gagatggtga ctctttttat agcgacattc\n",
      "     2581 ctcccggagt cagcacaaac tcagcatcta gttctaagaa gaggtcaaat gggctctctc\n",
      "     2641 attcttggag tgaaaggatt ccagacacaa aacatatttc agacatctgt gaaaatgggc\n",
      "     2701 gacctcgaag taactcttgg caaggtaacc tgggaggcaa caaaaagaaa atcagaggca\n",
      "     2761 aaagatttag acctcggtct aattcaactg agtaagtctg aacctctaat ggaaaaaggc\n",
      "     2821 actccaaggt cctgtgcaaa tagtgtcaaa agtaaattaa gcagatagtc aaagaactgt\n",
      "     2881 aaa\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       6XH3_D                    27 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  Co-crystal structure of HIV-1 TAR RNA in complex with lab-evolved\n",
      "            RRM TBP6.3.\n",
      "ACCESSION   6XH3_D\n",
      "VERSION     6XH3_D\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 27)\n",
      "  AUTHORS   Chavali,S.S., Mali,S.M., Jenkins,J.L., Fasan,R. and Wedekind,J.E.\n",
      "  TITLE     Co-crystal structures of HIV TAR RNA bound to lab-evolved proteins\n",
      "            show key roles for arginine relevant to the design of cyclic\n",
      "            peptide TAR inhibitors\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 27)\n",
      "  AUTHORS   Chavali,S.S., Jenkins,J.L. and Wedekind,J.E.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (18-JUN-2020)\n",
      "COMMENT     Co-crystal structure of HIV-1 TAR RNA in complex with lab-evolved\n",
      "            RRM TBP6.3.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..27\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 ggagatctga gcctgggagc tctctcc\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       6XH2_D                    27 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  Co-crystal structure of HIV-1 TAR RNA in complex with lab-evolved\n",
      "            RRM 6.6.\n",
      "ACCESSION   6XH2_D\n",
      "VERSION     6XH2_D\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 27)\n",
      "  AUTHORS   Chavali,S.S., Mali,S.M., Jenkins,J.L., Fasan,R. and Wedekind,J.E.\n",
      "  TITLE     Co-crystal structures of HIV TAR RNA bound to lab-evolved proteins\n",
      "            show key roles for arginine relevant to the design of cyclic\n",
      "            peptide TAR inhibitors\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 27)\n",
      "  AUTHORS   Chavali,S.S., Jenkins,J.L. and Wedekind,J.E.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (18-JUN-2020)\n",
      "COMMENT     Co-crystal structure of HIV-1 TAR RNA in complex with lab-evolved\n",
      "            RRM 6.6.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..27\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 gcagatctga gcctgggagc tctctgc\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       6XH1_D                    27 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  Co-crystal structure of HIV-1 TAR RNA in complex with lab-evolved\n",
      "            RRM TBP6.7 mutant.\n",
      "ACCESSION   6XH1_D\n",
      "VERSION     6XH1_D\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 27)\n",
      "  AUTHORS   Chavali,S.S., Mali,S.M., Jenkins,J.L., Fasan,R. and Wedekind,J.E.\n",
      "  TITLE     Co-crystal structures of HIV TAR RNA bound to lab-evolved proteins\n",
      "            show key roles for arginine relevant to the design of cyclic\n",
      "            peptide TAR inhibitors\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 27)\n",
      "  AUTHORS   Chavali,S.S., Jenkins,J.L. and Wedekind,J.E.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (18-JUN-2020)\n",
      "COMMENT     Co-crystal structure of HIV-1 TAR RNA in complex with lab-evolved\n",
      "            RRM TBP6.7 mutant.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..27\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 ggagatctga gcctgggagc tctctcc\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       6XH0_D                    27 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  Co-crystal structure of HIV-1 TAR RNA in complex with lab-evolved\n",
      "            RRM TBP6.9.\n",
      "ACCESSION   6XH0_D\n",
      "VERSION     6XH0_D\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 27)\n",
      "  AUTHORS   Chavali,S.S., Mali,S.M., Jenkins,J.L., Fasan,R. and Wedekind,J.E.\n",
      "  TITLE     Co-crystal structures of HIV TAR RNA bound to lab-evolved proteins\n",
      "            show key roles for arginine relevant to the design of cyclic\n",
      "            peptide TAR inhibitors\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 27)\n",
      "  AUTHORS   Chavali,S.S., Jenkins,J.L. and Wedekind,J.E.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (18-JUN-2020)\n",
      "COMMENT     Co-crystal structure of HIV-1 TAR RNA in complex with lab-evolved\n",
      "            RRM TBP6.9.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..27\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 gcagatctga gcctgggagc tctctgc\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       7JU1_X                    29 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  The FARFAR-NMR Ensemble of 29-mer HIV-1 Trans-activation Response\n",
      "            Element RNA (N=20).\n",
      "ACCESSION   7JU1_X\n",
      "VERSION     7JU1_X\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 29)\n",
      "  AUTHORS   Shi,H., Rangadurai,A., Abou-Assi,H., Roy,R., Case,D.A.,\n",
      "            Herschlag,D., Yesselman,J.D. and Al-Hashimi,H.M.\n",
      "  TITLE     Rapid and accurate determination of atomistic RNA dynamic ensemble\n",
      "            models using NMR and structure prediction\n",
      "  JOURNAL   Nat Commun (2020)\n",
      "REFERENCE   2  (bases 1 to 29)\n",
      "  AUTHORS   Shi,H., Rangadurai,A., Roy,R., Yesselman,J.D. and Al-Hashimi,H.M.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (18-AUG-2020)\n",
      "COMMENT     The FARFAR-NMR Ensemble of 29-mer HIV-1 Trans-activation Response\n",
      "            Element RNA (N=20).\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..29\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 ggcagatctg agcctgggag ctctctgcc\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       7K1Z_A                   103 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  Solution structure of the HIV-1 PBS-segment.\n",
      "ACCESSION   7K1Z_A\n",
      "VERSION     7K1Z_A\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 103)\n",
      "  AUTHORS   Song,Z. and Heng,X.\n",
      "  TITLE     Structure of the HIV-1 PBS-segment is conserved for viral\n",
      "            infectivity\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (bases 1 to 103)\n",
      "  AUTHORS   Heng,X. and Song,Z.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (08-SEP-2020)\n",
      "COMMENT     Solution structure of the HIV-1 PBS-segment.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..103\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 ggctctggta actagagatc cctcagaccc ttttagtcag tgtggaaaat ctctagcagt\n",
      "       61 ggcgcccgaa cagggacttg aaagcgaaag taaagccaga gcc\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       6WB2_C                   101 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  +3 extended HIV-1 reverse transcriptase initiation complex core\n",
      "            (displaced state).\n",
      "ACCESSION   6WB2_C\n",
      "VERSION     6WB2_C\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 101)\n",
      "  AUTHORS   Larsen,K.P., Choi,J., Jackson,L.N., Kappel,K., Zhang,J., Ha,B.,\n",
      "            Chen,D.H. and Puglisi,E.V.\n",
      "  TITLE     Distinct Conformational States Underlie Pausing during Initiation\n",
      "            of HIV-1 Reverse Transcription\n",
      "  JOURNAL   J Mol Biol 432 (16), 4499-4522 (2020)\n",
      "   PUBMED   32512005\n",
      "REFERENCE   2  (bases 1 to 101)\n",
      "  AUTHORS   Larsen,K.P., Jackson,L.N., Kappel,K., Zhang,J., Chen,D.H. and\n",
      "            Puglisi,E.V.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (26-MAR-2020)\n",
      "COMMENT     +3 extended HIV-1 reverse transcriptase initiation complex core\n",
      "            (displaced state).\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..101\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 gactctggta actagagatc cctcagaccc ttttagtcag tgtggaaaat ctctagcagt\n",
      "       61 ggcgcccgaa cagggacttg aaagcgaaag taaagccaga g\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       6WB1_C                   101 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  +3 extended HIV-1 reverse transcriptase initiation complex core\n",
      "            (intermediate state).\n",
      "ACCESSION   6WB1_C\n",
      "VERSION     6WB1_C\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 101)\n",
      "  AUTHORS   Larsen,K.P., Choi,J., Jackson,L.N., Kappel,K., Zhang,J., Ha,B.,\n",
      "            Chen,D.H. and Puglisi,E.V.\n",
      "  TITLE     Distinct Conformational States Underlie Pausing during Initiation\n",
      "            of HIV-1 Reverse Transcription\n",
      "  JOURNAL   J Mol Biol 432 (16), 4499-4522 (2020)\n",
      "   PUBMED   32512005\n",
      "REFERENCE   2  (bases 1 to 101)\n",
      "  AUTHORS   Larsen,K.P., Jackson,L.N., Kappel,K., Zhang,J., Chen,D.H. and\n",
      "            Puglisi,E.V.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (26-MAR-2020)\n",
      "COMMENT     +3 extended HIV-1 reverse transcriptase initiation complex core\n",
      "            (intermediate state).\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..101\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 gactctggta actagagatc cctcagaccc ttttagtcag tgtggaaaat ctctagcagt\n",
      "       61 ggcgcccgaa cagggacttg aaagcgaaag taaagccaga g\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       6WB0_C                   101 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  +3 extended HIV-1 reverse transcriptase initiation complex core\n",
      "            (pre-translocation state).\n",
      "ACCESSION   6WB0_C\n",
      "VERSION     6WB0_C\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 101)\n",
      "  AUTHORS   Larsen,K.P., Choi,J., Jackson,L.N., Kappel,K., Zhang,J., Ha,B.,\n",
      "            Chen,D.H. and Puglisi,E.V.\n",
      "  TITLE     Distinct Conformational States Underlie Pausing during Initiation\n",
      "            of HIV-1 Reverse Transcription\n",
      "  JOURNAL   J Mol Biol 432 (16), 4499-4522 (2020)\n",
      "   PUBMED   32512005\n",
      "REFERENCE   2  (bases 1 to 101)\n",
      "  AUTHORS   Larsen,K.P., Jackson,L.N., Kappel,K., Zhang,J. and Puglisi,E.V.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (26-MAR-2020)\n",
      "COMMENT     +3 extended HIV-1 reverse transcriptase initiation complex core\n",
      "            (pre-translocation state).\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..101\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 gactctggta actagagatc cctcagaccc ttttagtcag tgtggaaaat ctctagcagt\n",
      "       61 ggcgcccgaa cagggacttg aaagcgaaag taaagccaga g\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       6WAZ_C                   101 bp    RNA     linear   VRL 01-DEC-2020\n",
      "DEFINITION  +1 extended HIV-1 reverse transcriptase initiation complex core\n",
      "            (pre-translocation state).\n",
      "ACCESSION   6WAZ_C\n",
      "VERSION     6WAZ_C\n",
      "KEYWORDS    .\n",
      "SOURCE      Human immunodeficiency virus 1 (HIV-1)\n",
      "  ORGANISM  Human immunodeficiency virus 1\n",
      "            Viruses; Riboviria; Pararnavirae; Artverviricota; Revtraviricetes;\n",
      "            Ortervirales; Retroviridae; Orthoretrovirinae; Lentivirus.\n",
      "REFERENCE   1  (bases 1 to 101)\n",
      "  AUTHORS   Larsen,K.P., Choi,J., Jackson,L.N., Kappel,K., Zhang,J., Ha,B.,\n",
      "            Chen,D.H. and Puglisi,E.V.\n",
      "  TITLE     Distinct Conformational States Underlie Pausing during Initiation\n",
      "            of HIV-1 Reverse Transcription\n",
      "  JOURNAL   J Mol Biol 432 (16), 4499-4522 (2020)\n",
      "   PUBMED   32512005\n",
      "REFERENCE   2  (bases 1 to 101)\n",
      "  AUTHORS   Larsen,K.P., Jackson,L.N., Kappel,K., Zhang,J. and Puglisi,E.V.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (26-MAR-2020)\n",
      "COMMENT     +1 extended HIV-1 reverse transcriptase initiation complex core\n",
      "            (pre-translocation state).\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..101\n",
      "                     /organism=\"Human immunodeficiency virus 1\"\n",
      "                     /mol_type=\"other RNA\"\n",
      "                     /db_xref=\"taxon:11676\"\n",
      "ORIGIN      \n",
      "        1 gactctggta actagagatc cctcagaccc ttttagtcag tgtggaaaat ctctagcagt\n",
      "       61 ggcgcccgaa cagggacttg aaagcgaaag taaagccaga g\n",
      "//\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.api_key = \"92471a1f25f1f658f6711d0c7bc31ff80708\"\n",
    "Entrez.email = \"gree9242@uri.edu\"\n",
    "# Use a loop to go through the IDlist from previous section and fetch nucleotide databases\n",
    "for i in record1[\"IdList\"]:\n",
    "    handle = Entrez.efetch(db='nucleotide',id=i, rettype=\"gb\", retmode=\"text\",retmax=10)\n",
    "    print(handle.read())\n",
    "for i in record2[\"IdList\"]:\n",
    "    handle = Entrez.efetch(db='nucleotide',id=i, rettype=\"gb\", retmode=\"text\",retmax=10)\n",
    "    print(handle.read())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Retrieving publications\n",
    "\n",
    "Since we are interested in humans and their interaction with HIV we may want to find publications related to this field. Pubmed is a great resource and database to use to access free publications. We will be using similar coding structures with a few changes, especially when saving files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found 949 results\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.api_key = \"92471a1f25f1f658f6711d0c7bc31ff80708\"\n",
    "Entrez.email = \"gree9242@uri.edu\"\n",
    "search_results = Entrez.read(Entrez.esearch(db=\"pubmed\", term=hp, reldate=365, datetype=\"pdat\", usehistory=\"y\"))\n",
    "count = int(search_results[\"Count\"])\n",
    "print(\"Found %i results\" % count)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In case you were wondering that is 2.6 papers every day\n",
    "\n",
    "#### Saving abstracts\n",
    "\n",
    "Now that we know we have a lot of information to sift through maybe lets limit ourselves to saving the abstracts. We will be downloading these in batches. This helps NCBI admin to regulate traffic."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Going to download record 1 to 10\n",
      "Going to download record 11 to 20\n",
      "Going to download record 21 to 30\n",
      "Going to download record 31 to 40\n",
      "Going to download record 41 to 50\n",
      "Going to download record 51 to 60\n",
      "Going to download record 61 to 70\n",
      "Going to download record 71 to 80\n",
      "Going to download record 81 to 90\n",
      "Going to download record 91 to 100\n",
      "Going to download record 101 to 110\n",
      "Going to download record 111 to 120\n",
      "Going to download record 121 to 130\n",
      "Going to download record 131 to 140\n",
      "Going to download record 141 to 150\n",
      "Going to download record 151 to 160\n",
      "Going to download record 161 to 170\n",
      "Going to download record 171 to 180\n",
      "Going to download record 181 to 190\n",
      "Going to download record 191 to 200\n",
      "Going to download record 201 to 210\n",
      "Going to download record 211 to 220\n",
      "Going to download record 221 to 230\n",
      "Going to download record 231 to 240\n",
      "Going to download record 241 to 250\n",
      "Going to download record 251 to 260\n",
      "Going to download record 261 to 270\n",
      "Going to download record 271 to 280\n",
      "Going to download record 281 to 290\n",
      "Going to download record 291 to 300\n",
      "Going to download record 301 to 310\n",
      "Going to download record 311 to 320\n",
      "Going to download record 321 to 330\n",
      "Going to download record 331 to 340\n",
      "Going to download record 341 to 350\n",
      "Going to download record 351 to 360\n",
      "Going to download record 361 to 370\n",
      "Going to download record 371 to 380\n",
      "Going to download record 381 to 390\n",
      "Going to download record 391 to 400\n",
      "Going to download record 401 to 410\n",
      "Going to download record 411 to 420\n",
      "Going to download record 421 to 430\n",
      "Going to download record 431 to 440\n",
      "Going to download record 441 to 450\n",
      "Going to download record 451 to 460\n",
      "Going to download record 461 to 470\n",
      "Going to download record 471 to 480\n",
      "Going to download record 481 to 490\n",
      "Going to download record 491 to 500\n",
      "Going to download record 501 to 510\n",
      "Going to download record 511 to 520\n",
      "Going to download record 521 to 530\n",
      "Going to download record 531 to 540\n",
      "Going to download record 541 to 550\n",
      "Going to download record 551 to 560\n",
      "Going to download record 561 to 570\n",
      "Going to download record 571 to 580\n",
      "Going to download record 581 to 590\n",
      "Going to download record 591 to 600\n",
      "Going to download record 601 to 610\n",
      "Going to download record 611 to 620\n",
      "Going to download record 621 to 630\n",
      "Going to download record 631 to 640\n",
      "Going to download record 641 to 650\n",
      "Going to download record 651 to 660\n",
      "Going to download record 661 to 670\n",
      "Going to download record 671 to 680\n",
      "Going to download record 681 to 690\n",
      "Going to download record 691 to 700\n",
      "Going to download record 701 to 710\n",
      "Going to download record 711 to 720\n",
      "Going to download record 721 to 730\n",
      "Going to download record 731 to 740\n",
      "Going to download record 741 to 750\n",
      "Going to download record 751 to 760\n",
      "Going to download record 761 to 770\n",
      "Going to download record 771 to 780\n",
      "Going to download record 781 to 790\n",
      "Going to download record 791 to 800\n",
      "Going to download record 801 to 810\n",
      "Going to download record 811 to 820\n",
      "Going to download record 821 to 830\n",
      "Going to download record 831 to 840\n",
      "Going to download record 841 to 850\n",
      "Going to download record 851 to 860\n",
      "Going to download record 861 to 870\n",
      "Going to download record 871 to 880\n",
      "Going to download record 881 to 890\n",
      "Going to download record 891 to 900\n",
      "Going to download record 901 to 910\n",
      "Going to download record 911 to 920\n",
      "Going to download record 921 to 930\n",
      "Going to download record 931 to 940\n",
      "Going to download record 941 to 949\n"
     ]
    }
   ],
   "source": [
    "batch_size = 10\n",
    "out_handle = open(\"recent_human_hiv_papers.txt\", \"w\")\n",
    "for start in range(0, count, batch_size):\n",
    "    end = min(count, start + batch_size)\n",
    "    print(\"Going to download record %i to %i\" % (start + 1, end))\n",
    "    fetch_handle = Entrez.efetch(\n",
    "        db=\"pubmed\",\n",
    "        rettype=\"medline\",\n",
    "        retmode=\"text\",\n",
    "        retstart=start,\n",
    "        retmax=batch_size,\n",
    "        webenv=search_results[\"WebEnv\"],\n",
    "        query_key=search_results[\"QueryKey\"],\n",
    "    )\n",
    "    data = fetch_handle.read()\n",
    "    fetch_handle.close()\n",
    "    out_handle.write(data)\n",
    "out_handle.close()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Not bad for downloading 900 abstracts\n",
    "\n",
    "In terms of starting a research project, having access to this much relevant information is crucial to developing good hypothesis and designing experiments. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Downloading Sequences\n",
    "\n",
    "This next section is the most crucial aspect to our project. For the last two sections we have been on an information gathering hunt, which is impportant, but if we want to apply some of the genetic information contained within the NCBI database we need to access the volume of genetic information within. We will be taking a slightly more refined approach to this process becuase of how much genetic infromation there is. \n",
    "\n",
    "As an example, the human genome is 3.2 gb and takes up approximately 964.8 MB. That would be about 72,000 sheets of paper filled with ATCG. Since we want to alignment multiple different things we will ask a more specific question. Do any HIV proteins align with Human proteins? Our knowledge of how HIV invades the immune system help here.\n",
    "\n",
    "<img src=\"HIV_absorb_cd4_and_ccr5.png\" width=\"400\"/>\n",
    "\n",
    "What we know is that the major players in facilitating HIV invasion of T helper cells are the HIV gp120 and T Helper CD4 receptor. Changes in these receptors can inhibit invasion of the immune cell and keep people from developing AIDS. In this section we will download these two proteins to use in subsequent sections of this project."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found 30 results \n",
      "CD4 accession ids: ['NM_001382705.1', 'NM_001382714.1', 'NM_001382706.1', 'NM_001382707.1', 'NM_001195016.3', 'NM_001195015.3', 'NM_001195014.3', 'NM_001195017.3', 'NM_000616.5', 'NC_000012.12', 'NG_027688.1', 'MK170450.1', 'DQ895246.2', 'CM000263.1', 'CH471116.2', 'BC025782.1', 'U40625.1', 'U47924.1', 'AB590293.1', 'U01066.1']\n"
     ]
    }
   ],
   "source": [
    "# Download CD4 fasta\n",
    "from Bio import Entrez\n",
    "# Add email and key\n",
    "Entrez.api_key = \"92471a1f25f1f658f6711d0c7bc31ff80708\"\n",
    "Entrez.email = \"gree9242@uri.edu\"\n",
    "search_handle = Entrez.esearch(db=\"nuccore\",term=\"Homo sapiens[orgn] AND cd4 [gene]\", usehistory=\"y\", idtype=\"acc\")\n",
    "search_results = Entrez.read(search_handle)\n",
    "search_handle.close()\n",
    "#search_results.pop(\"NC_000012.12\")\n",
    "acc_list = search_results[\"IdList\"]\n",
    "count = int(search_results[\"Count\"])\n",
    "count == len(acc_list)\n",
    "webenv = search_results[\"WebEnv\"]\n",
    "query_key = search_results[\"QueryKey\"]\n",
    "print(\"Found %i results \" % count)\n",
    "print('CD4 accession ids: {}'.format(search_results[\"IdList\"]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Going to download record 1 to 3\n",
      "Going to download record 4 to 6\n",
      "Going to download record 7 to 9\n",
      "Going to download record 10 to 12\n",
      "Going to download record 13 to 15\n",
      "Going to download record 16 to 18\n",
      "Going to download record 19 to 21\n",
      "Going to download record 22 to 24\n",
      "Going to download record 25 to 27\n",
      "Going to download record 28 to 30\n"
     ]
    }
   ],
   "source": [
    "batch_size = 3\n",
    "out_handle = open(\"human_cd4.fasta\", \"w\")\n",
    "for start in range(0, count, batch_size):\n",
    "    end = min(count, start + batch_size)\n",
    "    print(\"Going to download record %i to %i\" % (start + 1, end))\n",
    "    fetch_handle = Entrez.efetch(\n",
    "        db=\"nucleotide\",\n",
    "        rettype=\"fasta\",\n",
    "        retmode=\"text\",\n",
    "        retstart=start,\n",
    "        retmax=batch_size,\n",
    "        webenv=webenv,\n",
    "        query_key=query_key,\n",
    "        idtype=\"acc\",\n",
    "    )\n",
    "    data = fetch_handle.read()\n",
    "    fetch_handle.close()\n",
    "    out_handle.write(data)\n",
    "out_handle.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found 1165 results \n",
      "CD4 accession ids: ['EU010360.1', 'EU010359.1', 'EU010358.1', 'EU010357.1', 'EU010356.1', 'EU010355.1', 'EU010354.1', 'EU010353.1', 'EU010352.1', 'EU010351.1', 'EU010350.1', 'EU010349.1', 'EU010348.1', 'EU010347.1', 'EU010346.1', 'EU010345.1', 'EU010344.1', 'EU010343.1', 'EU010342.1', 'EU010341.1']\n"
     ]
    }
   ],
   "source": [
    "# Download CD4 fasta\n",
    "from Bio import Entrez\n",
    "# Add email and key\n",
    "Entrez.api_key = \"92471a1f25f1f658f6711d0c7bc31ff80708\"\n",
    "Entrez.email = \"gree9242@uri.edu\"\n",
    "search_handle = Entrez.esearch(db=\"nuccore\",term=\"HIV1 [orgn] AND gp120 [gene]\", usehistory=\"y\", idtype=\"acc\")\n",
    "search_results = Entrez.read(search_handle)\n",
    "search_handle.close()\n",
    "#search_results.pop(\"NC_000012.12\")\n",
    "acc_list = search_results[\"IdList\"]\n",
    "count = int(search_results[\"Count\"])\n",
    "count == len(acc_list)\n",
    "webenv = search_results[\"WebEnv\"]\n",
    "query_key = search_results[\"QueryKey\"]\n",
    "print(\"Found %i results \" % count)\n",
    "print('CD4 accession ids: {}'.format(search_results[\"IdList\"]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Going to download record 1 to 3\n",
      "Going to download record 4 to 6\n",
      "Going to download record 7 to 9\n",
      "Going to download record 10 to 12\n",
      "Going to download record 13 to 15\n",
      "Going to download record 16 to 18\n",
      "Going to download record 19 to 21\n",
      "Going to download record 22 to 24\n",
      "Going to download record 25 to 27\n",
      "Going to download record 28 to 30\n",
      "Going to download record 31 to 33\n",
      "Going to download record 34 to 36\n",
      "Going to download record 37 to 39\n",
      "Going to download record 40 to 42\n",
      "Going to download record 43 to 45\n",
      "Going to download record 46 to 48\n",
      "Going to download record 49 to 51\n",
      "Going to download record 52 to 54\n",
      "Going to download record 55 to 57\n",
      "Going to download record 58 to 60\n",
      "Going to download record 61 to 63\n",
      "Going to download record 64 to 66\n",
      "Going to download record 67 to 69\n",
      "Going to download record 70 to 72\n",
      "Going to download record 73 to 75\n",
      "Going to download record 76 to 78\n",
      "Going to download record 79 to 81\n",
      "Going to download record 82 to 84\n",
      "Going to download record 85 to 87\n",
      "Going to download record 88 to 90\n",
      "Going to download record 91 to 93\n",
      "Going to download record 94 to 96\n",
      "Going to download record 97 to 99\n",
      "Going to download record 100 to 102\n",
      "Going to download record 103 to 105\n",
      "Going to download record 106 to 108\n",
      "Going to download record 109 to 111\n",
      "Going to download record 112 to 114\n",
      "Going to download record 115 to 117\n",
      "Going to download record 118 to 120\n",
      "Going to download record 121 to 123\n",
      "Going to download record 124 to 126\n",
      "Going to download record 127 to 129\n",
      "Going to download record 130 to 132\n",
      "Going to download record 133 to 135\n",
      "Going to download record 136 to 138\n",
      "Going to download record 139 to 141\n",
      "Going to download record 142 to 144\n",
      "Going to download record 145 to 147\n",
      "Going to download record 148 to 150\n",
      "Going to download record 151 to 153\n",
      "Going to download record 154 to 156\n",
      "Going to download record 157 to 159\n",
      "Going to download record 160 to 162\n",
      "Going to download record 163 to 165\n",
      "Going to download record 166 to 168\n",
      "Going to download record 169 to 171\n",
      "Going to download record 172 to 174\n",
      "Going to download record 175 to 177\n",
      "Going to download record 178 to 180\n",
      "Going to download record 181 to 183\n",
      "Going to download record 184 to 186\n",
      "Going to download record 187 to 189\n",
      "Going to download record 190 to 192\n",
      "Going to download record 193 to 195\n",
      "Going to download record 196 to 198\n",
      "Going to download record 199 to 201\n",
      "Going to download record 202 to 204\n",
      "Going to download record 205 to 207\n",
      "Going to download record 208 to 210\n",
      "Going to download record 211 to 213\n",
      "Going to download record 214 to 216\n",
      "Going to download record 217 to 219\n",
      "Going to download record 220 to 222\n",
      "Going to download record 223 to 225\n",
      "Going to download record 226 to 228\n",
      "Going to download record 229 to 231\n",
      "Going to download record 232 to 234\n",
      "Going to download record 235 to 237\n",
      "Going to download record 238 to 240\n",
      "Going to download record 241 to 243\n",
      "Going to download record 244 to 246\n",
      "Going to download record 247 to 249\n",
      "Going to download record 250 to 252\n",
      "Going to download record 253 to 255\n",
      "Going to download record 256 to 258\n",
      "Going to download record 259 to 261\n",
      "Going to download record 262 to 264\n",
      "Going to download record 265 to 267\n",
      "Going to download record 268 to 270\n",
      "Going to download record 271 to 273\n",
      "Going to download record 274 to 276\n",
      "Going to download record 277 to 279\n",
      "Going to download record 280 to 282\n",
      "Going to download record 283 to 285\n",
      "Going to download record 286 to 288\n",
      "Going to download record 289 to 291\n",
      "Going to download record 292 to 294\n",
      "Going to download record 295 to 297\n",
      "Going to download record 298 to 300\n",
      "Going to download record 301 to 303\n",
      "Going to download record 304 to 306\n",
      "Going to download record 307 to 309\n",
      "Going to download record 310 to 312\n",
      "Going to download record 313 to 315\n",
      "Going to download record 316 to 318\n",
      "Going to download record 319 to 321\n",
      "Going to download record 322 to 324\n",
      "Going to download record 325 to 327\n",
      "Going to download record 328 to 330\n",
      "Going to download record 331 to 333\n",
      "Going to download record 334 to 336\n",
      "Going to download record 337 to 339\n",
      "Going to download record 340 to 342\n",
      "Going to download record 343 to 345\n",
      "Going to download record 346 to 348\n",
      "Going to download record 349 to 351\n",
      "Going to download record 352 to 354\n",
      "Going to download record 355 to 357\n",
      "Going to download record 358 to 360\n",
      "Going to download record 361 to 363\n",
      "Going to download record 364 to 366\n",
      "Going to download record 367 to 369\n",
      "Going to download record 370 to 372\n",
      "Going to download record 373 to 375\n",
      "Going to download record 376 to 378\n",
      "Going to download record 379 to 381\n",
      "Going to download record 382 to 384\n",
      "Going to download record 385 to 387\n",
      "Going to download record 388 to 390\n",
      "Going to download record 391 to 393\n",
      "Going to download record 394 to 396\n",
      "Going to download record 397 to 399\n",
      "Going to download record 400 to 402\n",
      "Going to download record 403 to 405\n",
      "Going to download record 406 to 408\n",
      "Going to download record 409 to 411\n",
      "Going to download record 412 to 414\n",
      "Going to download record 415 to 417\n",
      "Going to download record 418 to 420\n",
      "Going to download record 421 to 423\n",
      "Going to download record 424 to 426\n",
      "Going to download record 427 to 429\n",
      "Going to download record 430 to 432\n",
      "Going to download record 433 to 435\n",
      "Going to download record 436 to 438\n",
      "Going to download record 439 to 441\n",
      "Going to download record 442 to 444\n",
      "Going to download record 445 to 447\n",
      "Going to download record 448 to 450\n",
      "Going to download record 451 to 453\n",
      "Going to download record 454 to 456\n",
      "Going to download record 457 to 459\n",
      "Going to download record 460 to 462\n",
      "Going to download record 463 to 465\n",
      "Going to download record 466 to 468\n",
      "Going to download record 469 to 471\n",
      "Going to download record 472 to 474\n",
      "Going to download record 475 to 477\n",
      "Going to download record 478 to 480\n",
      "Going to download record 481 to 483\n",
      "Going to download record 484 to 486\n",
      "Going to download record 487 to 489\n",
      "Going to download record 490 to 492\n",
      "Going to download record 493 to 495\n",
      "Going to download record 496 to 498\n",
      "Going to download record 499 to 501\n",
      "Going to download record 502 to 504\n",
      "Going to download record 505 to 507\n",
      "Going to download record 508 to 510\n",
      "Going to download record 511 to 513\n",
      "Going to download record 514 to 516\n",
      "Going to download record 517 to 519\n",
      "Going to download record 520 to 522\n",
      "Going to download record 523 to 525\n",
      "Going to download record 526 to 528\n",
      "Going to download record 529 to 531\n",
      "Going to download record 532 to 534\n",
      "Going to download record 535 to 537\n",
      "Going to download record 538 to 540\n",
      "Going to download record 541 to 543\n",
      "Going to download record 544 to 546\n",
      "Going to download record 547 to 549\n",
      "Going to download record 550 to 552\n",
      "Going to download record 553 to 555\n",
      "Going to download record 556 to 558\n",
      "Going to download record 559 to 561\n",
      "Going to download record 562 to 564\n",
      "Going to download record 565 to 567\n",
      "Going to download record 568 to 570\n",
      "Going to download record 571 to 573\n",
      "Going to download record 574 to 576\n",
      "Going to download record 577 to 579\n",
      "Going to download record 580 to 582\n",
      "Going to download record 583 to 585\n",
      "Going to download record 586 to 588\n",
      "Going to download record 589 to 591\n",
      "Going to download record 592 to 594\n",
      "Going to download record 595 to 597\n",
      "Going to download record 598 to 600\n",
      "Going to download record 601 to 603\n",
      "Going to download record 604 to 606\n",
      "Going to download record 607 to 609\n",
      "Going to download record 610 to 612\n",
      "Going to download record 613 to 615\n",
      "Going to download record 616 to 618\n",
      "Going to download record 619 to 621\n",
      "Going to download record 622 to 624\n",
      "Going to download record 625 to 627\n",
      "Going to download record 628 to 630\n",
      "Going to download record 631 to 633\n",
      "Going to download record 634 to 636\n",
      "Going to download record 637 to 639\n",
      "Going to download record 640 to 642\n",
      "Going to download record 643 to 645\n",
      "Going to download record 646 to 648\n",
      "Going to download record 649 to 651\n",
      "Going to download record 652 to 654\n",
      "Going to download record 655 to 657\n",
      "Going to download record 658 to 660\n",
      "Going to download record 661 to 663\n",
      "Going to download record 664 to 666\n",
      "Going to download record 667 to 669\n",
      "Going to download record 670 to 672\n",
      "Going to download record 673 to 675\n",
      "Going to download record 676 to 678\n",
      "Going to download record 679 to 681\n",
      "Going to download record 682 to 684\n",
      "Going to download record 685 to 687\n",
      "Going to download record 688 to 690\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Going to download record 691 to 693\n",
      "Going to download record 694 to 696\n",
      "Going to download record 697 to 699\n",
      "Going to download record 700 to 702\n",
      "Going to download record 703 to 705\n",
      "Going to download record 706 to 708\n",
      "Going to download record 709 to 711\n",
      "Going to download record 712 to 714\n",
      "Going to download record 715 to 717\n",
      "Going to download record 718 to 720\n",
      "Going to download record 721 to 723\n",
      "Going to download record 724 to 726\n",
      "Going to download record 727 to 729\n",
      "Going to download record 730 to 732\n",
      "Going to download record 733 to 735\n",
      "Going to download record 736 to 738\n",
      "Going to download record 739 to 741\n",
      "Going to download record 742 to 744\n",
      "Going to download record 745 to 747\n",
      "Going to download record 748 to 750\n",
      "Going to download record 751 to 753\n",
      "Going to download record 754 to 756\n",
      "Going to download record 757 to 759\n",
      "Going to download record 760 to 762\n",
      "Going to download record 763 to 765\n",
      "Going to download record 766 to 768\n",
      "Going to download record 769 to 771\n",
      "Going to download record 772 to 774\n",
      "Going to download record 775 to 777\n",
      "Going to download record 778 to 780\n",
      "Going to download record 781 to 783\n",
      "Going to download record 784 to 786\n",
      "Going to download record 787 to 789\n",
      "Going to download record 790 to 792\n",
      "Going to download record 793 to 795\n",
      "Going to download record 796 to 798\n",
      "Going to download record 799 to 801\n",
      "Going to download record 802 to 804\n",
      "Going to download record 805 to 807\n",
      "Going to download record 808 to 810\n",
      "Going to download record 811 to 813\n",
      "Going to download record 814 to 816\n",
      "Going to download record 817 to 819\n",
      "Going to download record 820 to 822\n",
      "Going to download record 823 to 825\n",
      "Going to download record 826 to 828\n",
      "Going to download record 829 to 831\n",
      "Going to download record 832 to 834\n",
      "Going to download record 835 to 837\n",
      "Going to download record 838 to 840\n",
      "Going to download record 841 to 843\n",
      "Going to download record 844 to 846\n",
      "Going to download record 847 to 849\n",
      "Going to download record 850 to 852\n",
      "Going to download record 853 to 855\n",
      "Going to download record 856 to 858\n",
      "Going to download record 859 to 861\n",
      "Going to download record 862 to 864\n",
      "Going to download record 865 to 867\n",
      "Going to download record 868 to 870\n",
      "Going to download record 871 to 873\n",
      "Going to download record 874 to 876\n",
      "Going to download record 877 to 879\n",
      "Going to download record 880 to 882\n",
      "Going to download record 883 to 885\n",
      "Going to download record 886 to 888\n",
      "Going to download record 889 to 891\n",
      "Going to download record 892 to 894\n",
      "Going to download record 895 to 897\n",
      "Going to download record 898 to 900\n",
      "Going to download record 901 to 903\n",
      "Going to download record 904 to 906\n",
      "Going to download record 907 to 909\n",
      "Going to download record 910 to 912\n",
      "Going to download record 913 to 915\n",
      "Going to download record 916 to 918\n",
      "Going to download record 919 to 921\n",
      "Going to download record 922 to 924\n",
      "Going to download record 925 to 927\n",
      "Going to download record 928 to 930\n",
      "Going to download record 931 to 933\n",
      "Going to download record 934 to 936\n",
      "Going to download record 937 to 939\n",
      "Going to download record 940 to 942\n",
      "Going to download record 943 to 945\n",
      "Going to download record 946 to 948\n",
      "Going to download record 949 to 951\n",
      "Going to download record 952 to 954\n",
      "Going to download record 955 to 957\n",
      "Going to download record 958 to 960\n",
      "Going to download record 961 to 963\n",
      "Going to download record 964 to 966\n",
      "Going to download record 967 to 969\n",
      "Going to download record 970 to 972\n",
      "Going to download record 973 to 975\n",
      "Going to download record 976 to 978\n",
      "Going to download record 979 to 981\n",
      "Going to download record 982 to 984\n",
      "Going to download record 985 to 987\n",
      "Going to download record 988 to 990\n",
      "Going to download record 991 to 993\n",
      "Going to download record 994 to 996\n",
      "Going to download record 997 to 999\n",
      "Going to download record 1000 to 1002\n",
      "Going to download record 1003 to 1005\n",
      "Going to download record 1006 to 1008\n",
      "Going to download record 1009 to 1011\n",
      "Going to download record 1012 to 1014\n",
      "Going to download record 1015 to 1017\n",
      "Going to download record 1018 to 1020\n",
      "Going to download record 1021 to 1023\n",
      "Going to download record 1024 to 1026\n",
      "Going to download record 1027 to 1029\n",
      "Going to download record 1030 to 1032\n",
      "Going to download record 1033 to 1035\n",
      "Going to download record 1036 to 1038\n",
      "Going to download record 1039 to 1041\n",
      "Going to download record 1042 to 1044\n",
      "Going to download record 1045 to 1047\n",
      "Going to download record 1048 to 1050\n",
      "Going to download record 1051 to 1053\n",
      "Going to download record 1054 to 1056\n",
      "Going to download record 1057 to 1059\n",
      "Going to download record 1060 to 1062\n",
      "Going to download record 1063 to 1065\n",
      "Going to download record 1066 to 1068\n",
      "Going to download record 1069 to 1071\n",
      "Going to download record 1072 to 1074\n",
      "Going to download record 1075 to 1077\n",
      "Going to download record 1078 to 1080\n",
      "Going to download record 1081 to 1083\n",
      "Going to download record 1084 to 1086\n",
      "Going to download record 1087 to 1089\n",
      "Going to download record 1090 to 1092\n",
      "Going to download record 1093 to 1095\n",
      "Going to download record 1096 to 1098\n",
      "Going to download record 1099 to 1101\n",
      "Going to download record 1102 to 1104\n",
      "Going to download record 1105 to 1107\n",
      "Going to download record 1108 to 1110\n",
      "Going to download record 1111 to 1113\n",
      "Going to download record 1114 to 1116\n",
      "Going to download record 1117 to 1119\n",
      "Going to download record 1120 to 1122\n",
      "Going to download record 1123 to 1125\n",
      "Going to download record 1126 to 1128\n",
      "Going to download record 1129 to 1131\n",
      "Going to download record 1132 to 1134\n",
      "Going to download record 1135 to 1137\n",
      "Going to download record 1138 to 1140\n",
      "Going to download record 1141 to 1143\n",
      "Going to download record 1144 to 1146\n",
      "Going to download record 1147 to 1149\n",
      "Going to download record 1150 to 1152\n",
      "Going to download record 1153 to 1155\n",
      "Going to download record 1156 to 1158\n",
      "Going to download record 1159 to 1161\n",
      "Going to download record 1162 to 1164\n",
      "Going to download record 1165 to 1165\n"
     ]
    }
   ],
   "source": [
    "batch_size = 3\n",
    "out_handle = open(\"HIV_gp120.fasta\", \"w\")\n",
    "for start in range(0, count, batch_size):\n",
    "    end = min(count, start + batch_size)\n",
    "    print(\"Going to download record %i to %i\" % (start + 1, end))\n",
    "    fetch_handle = Entrez.efetch(\n",
    "        db=\"nucleotide\",\n",
    "        rettype=\"fasta\",\n",
    "        retmode=\"text\",\n",
    "        retstart=start,\n",
    "        retmax=batch_size,\n",
    "        webenv=webenv,\n",
    "        query_key=idlist,\n",
    "        idtype=\"acc\",\n",
    "    )\n",
    "    data = fetch_handle.read()\n",
    "    fetch_handle.close()\n",
    "    out_handle.write(data)\n",
    "out_handle.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found 1165 results \n",
      "CD4 accession ids: ['EU010360.1', 'EU010359.1', 'EU010358.1', 'EU010357.1', 'EU010356.1', 'EU010355.1', 'EU010354.1', 'EU010353.1', 'EU010352.1']\n"
     ]
    }
   ],
   "source": [
    "#Editing version\n",
    "# Download CD4 fasta\n",
    "from Bio import Entrez\n",
    "# Add email and key\n",
    "Entrez.api_key = \"92471a1f25f1f658f6711d0c7bc31ff80708\"\n",
    "Entrez.email = \"gree9242@uri.edu\"\n",
    "search_handle = Entrez.esearch(db=\"nuccore\",term=\"HIV1 [orgn] AND gp120 [gene]\", usehistory=\"y\", idtype=\"acc\")\n",
    "search_results = Entrez.read(search_handle)\n",
    "search_handle.close()\n",
    "#search_results.pop(\"NC_000012.12\")\n",
    "acc_list = search_results[\"IdList\"]\n",
    "count = int(search_results[\"Count\"])\n",
    "count == len(acc_list)\n",
    "idlist = search_results[\"IdList\"][:9]\n",
    "webenv = search_results[\"WebEnv\"]\n",
    "query_key = search_results[\"QueryKey\"]\n",
    "print(\"Found %i results \" % count)\n",
    "print('CD4 accession ids: {}'.format(search_results[\"IdList\"][:9]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Going to download record 1 to 3\n"
     ]
    },
    {
     "ename": "HTTPError",
     "evalue": "HTTP Error 500: Internal Server Error",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mHTTPError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-105-3d1e50572e30>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      5\u001b[0m     \u001b[0mend\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmin\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mcount\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mstart\u001b[0m \u001b[1;33m+\u001b[0m \u001b[0mbatch_size\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      6\u001b[0m     \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"Going to download record %i to %i\"\u001b[0m \u001b[1;33m%\u001b[0m \u001b[1;33m(\u001b[0m\u001b[0mstart\u001b[0m \u001b[1;33m+\u001b[0m \u001b[1;36m1\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mend\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 7\u001b[1;33m     fetch_handle = Entrez.efetch(\n\u001b[0m\u001b[0;32m      8\u001b[0m         \u001b[0mdb\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;34m\"nucleotide\"\u001b[0m\u001b[1;33m,\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      9\u001b[0m         \u001b[0mrettype\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;34m\"fasta\"\u001b[0m\u001b[1;33m,\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\Bio\\Entrez\\__init__.py\u001b[0m in \u001b[0;36mefetch\u001b[1;34m(db, **keywords)\u001b[0m\n\u001b[0;32m    205\u001b[0m             \u001b[1;31m# more than about 200 IDs\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    206\u001b[0m             \u001b[0mpost\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;32mTrue\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 207\u001b[1;33m     \u001b[1;32mreturn\u001b[0m \u001b[0m_open\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mcgi\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mvariables\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mpost\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mpost\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    208\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    209\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\Bio\\Entrez\\__init__.py\u001b[0m in \u001b[0;36m_open\u001b[1;34m(cgi, params, post, ecitmatch)\u001b[0m\n\u001b[0;32m    604\u001b[0m                 \u001b[0mhandle\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0murlopen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mcgi\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mdata\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0moptions\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mencode\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"utf8\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    605\u001b[0m             \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 606\u001b[1;33m                 \u001b[0mhandle\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0murlopen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mcgi\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    607\u001b[0m         \u001b[1;32mexcept\u001b[0m \u001b[0mHTTPError\u001b[0m \u001b[1;32mas\u001b[0m \u001b[0mexception\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    608\u001b[0m             \u001b[1;31m# Reraise if the final try fails\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36murlopen\u001b[1;34m(url, data, timeout, cafile, capath, cadefault, context)\u001b[0m\n\u001b[0;32m    220\u001b[0m     \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    221\u001b[0m         \u001b[0mopener\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0m_opener\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 222\u001b[1;33m     \u001b[1;32mreturn\u001b[0m \u001b[0mopener\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mopen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0murl\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mdata\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    223\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    224\u001b[0m \u001b[1;32mdef\u001b[0m \u001b[0minstall_opener\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mopener\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36mopen\u001b[1;34m(self, fullurl, data, timeout)\u001b[0m\n\u001b[0;32m    529\u001b[0m         \u001b[1;32mfor\u001b[0m \u001b[0mprocessor\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mprocess_response\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mprotocol\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;33m[\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    530\u001b[0m             \u001b[0mmeth\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mgetattr\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mprocessor\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmeth_name\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 531\u001b[1;33m             \u001b[0mresponse\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmeth\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mreq\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mresponse\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    532\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    533\u001b[0m         \u001b[1;32mreturn\u001b[0m \u001b[0mresponse\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36mhttp_response\u001b[1;34m(self, request, response)\u001b[0m\n\u001b[0;32m    638\u001b[0m         \u001b[1;31m# request was successfully received, understood, and accepted.\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    639\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[1;32mnot\u001b[0m \u001b[1;33m(\u001b[0m\u001b[1;36m200\u001b[0m \u001b[1;33m<=\u001b[0m \u001b[0mcode\u001b[0m \u001b[1;33m<\u001b[0m \u001b[1;36m300\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 640\u001b[1;33m             response = self.parent.error(\n\u001b[0m\u001b[0;32m    641\u001b[0m                 'http', request, response, code, msg, hdrs)\n\u001b[0;32m    642\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36merror\u001b[1;34m(self, proto, *args)\u001b[0m\n\u001b[0;32m    567\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0mhttp_err\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    568\u001b[0m             \u001b[0margs\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;33m(\u001b[0m\u001b[0mdict\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'default'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'http_error_default'\u001b[0m\u001b[1;33m)\u001b[0m \u001b[1;33m+\u001b[0m \u001b[0morig_args\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 569\u001b[1;33m             \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_call_chain\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m*\u001b[0m\u001b[0margs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    570\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    571\u001b[0m \u001b[1;31m# XXX probably also want an abstract factory that knows when it makes\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36m_call_chain\u001b[1;34m(self, chain, kind, meth_name, *args)\u001b[0m\n\u001b[0;32m    500\u001b[0m         \u001b[1;32mfor\u001b[0m \u001b[0mhandler\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mhandlers\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    501\u001b[0m             \u001b[0mfunc\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mgetattr\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mhandler\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmeth_name\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 502\u001b[1;33m             \u001b[0mresult\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mfunc\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m*\u001b[0m\u001b[0margs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    503\u001b[0m             \u001b[1;32mif\u001b[0m \u001b[0mresult\u001b[0m \u001b[1;32mis\u001b[0m \u001b[1;32mnot\u001b[0m \u001b[1;32mNone\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    504\u001b[0m                 \u001b[1;32mreturn\u001b[0m \u001b[0mresult\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\urllib\\request.py\u001b[0m in \u001b[0;36mhttp_error_default\u001b[1;34m(self, req, fp, code, msg, hdrs)\u001b[0m\n\u001b[0;32m    647\u001b[0m \u001b[1;32mclass\u001b[0m \u001b[0mHTTPDefaultErrorHandler\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mBaseHandler\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    648\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0mhttp_error_default\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mreq\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mfp\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mcode\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmsg\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mhdrs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 649\u001b[1;33m         \u001b[1;32mraise\u001b[0m \u001b[0mHTTPError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mreq\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mfull_url\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mcode\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmsg\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mhdrs\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mfp\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    650\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    651\u001b[0m \u001b[1;32mclass\u001b[0m \u001b[0mHTTPRedirectHandler\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mBaseHandler\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mHTTPError\u001b[0m: HTTP Error 500: Internal Server Error"
     ]
    }
   ],
   "source": [
    "# Editing version\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "handle = Entrez.efetch(db=\"nucleotide\", id=idlist, retmode=\"xml\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Viewing files\n",
    "\n",
    "Luckily Jupyter notebooks is fairly straight forward in veiwing these files (as long as they are UTF-8 encoded, which most NCBI files are). The abstract file and fasta file are both in the local directory from where this notebook is being used. Make sure to go back into these files and view some of the diversity of genetic information that is obtained. One issue with data mining resources like NCBI is that our specificity in refininig searches is extremely important, as downloading certain elements that have little to no relevance is very common. In this case view the last entry of the human CD4.fasta and you will find that we downloaded a whole chromosome of the human genome reference. While it is amazing that we can do that, and it may provided a good reference point, there are some analyses that we would not want to included that entry on."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Onward to alignment\n",
    "\n",
    "In the next tutorial we will be aligning our pathogen sequences in the fasta file that we downloaded. Make sure you have the notebook and fasta file locally available by running this script. See you over there!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
